# FunGen-xQTL Data Catalog

|     |     |     |     |     |     |     |     |     |
|:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |
| [#](#0) 	| [A](#a) 	| [B](#b) 	| [C](#c) 	| [D](#d) 	| [E](#e) 	| [F](#f) 	| [G](#g) 	| [H](#h) 	|
| [I](#i) 	| [J](#j) 	| [K](#k) 	| [L](#l) 	| [M](#m) 	| [N](#n) 	| [O](#o) 	| [P](#p) 	| [Q](#q) 	|
| [R](#r) 	| [S](#s) 	| [T](#t) 	| [U](#u) 	| [V](#v) 	| [W](#w) 	| [X](#x) 	| [Y](#y) 	| [Z](#z)  	|
## A
* [ADGC GWAS imputation protocol](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/gwas/ADGC_GWAS_imputation_protocol.md). Cited from [ADGC GWAS Data QC Protocol](https://bitbucket.org/wanpinglee_penn/gwas_qc/src/master/).
	* Lead analysts: **Note that results from TOPMed imputation server are all against **HG38.**.**
* [Alzheimer's Disease GWAS Summary Data (Bellenguez)](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/gwas/Bellenguez_AD.md). The SNP-level association testing summary statistics for Alzheimer's disease from Bellenguez et al 2022 Nature Genetics.
	* Lead analysts: **Oluwatosin Olayinka, Hao Sun and Rui Dong.**
* [Alzheimer's Disease GWAS Summary Data (Jansen)](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/gwas/Jansen_AD.md). The SNP-level association testing summary statistics for Alzheimer's disease from Jansen et al 2021 Nature Genetics.
	* Lead analysts: **Oluwatosin Olayinka.**
* [Alzheimer's Disease GWAS Summary Data (Kunkle)](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/gwas/Kunkle_AD.md). The SNP-level association testing summary statistics for Alzheimer's disease from Kunkle et al 2019 Nature Genetics.
	* Lead analysts: **Oluwatosin Olayinka.**
* [Alzheimer's Disease GWAS Summary Data (Wightman)](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/gwas/Wightman_AD.md). The SNP-level association testing summary statistics for Alzheimer's disease from Wightman et al 2021 Nature Genetics.
	* Lead analysts: **Oluwatosin Olayinka.**

## D
* [DIAN blood derived methylation](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/dian_blood_methylation.md).
	* Lead analysts: **TBD, Alan Renton and Carlos Cruchaga.**
* [DIAN blood derived methylation QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/dian_blood_methylation_qtl.md).
	* Lead analysts: **TBD, Alan Renton and Carlos Cruchaga.**

## E
* [EFIGA CSF proteomics](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/EFIGA_CSF_proteomics.md).
	* Lead analysts: **Zining Qi.**
* [EFIGA CSF proteomics QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/EFIGA_CSF_proteomics_qtl.md).
	* Lead analysts: **Zining Qi.**
* [EFIGA genotype](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/WHICAP_genotype.md).
	* Lead analysts: **Zining Qi.**
* [EFIGA plasma metabolomics](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/EFIGA_plasma_metabolomics.md).
	* Lead analysts: **Zining Qi.**
* [EFIGA plasma metabolomics QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/EFIGA_plasma_metabolomics_qtl.md).
	* Lead analysts: **Zining Qi.**

## F
* [FunGen-xQTL protocol data](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/FunGen_xQTL_protocol_data.md). A toy data-set consisting of 49 de-identified samples from ROSMAP project, used to illustrates the computational protocols we developed for the detection and analysis of molecular QTLs (xQTLs).
	* Lead analysts: **Hao Sun.**

## H
* [HIHG African American blood alternative splicing](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MAGENTA_AA_blood_splicing.md).
	* Lead analysts: **Makaela Mews.**
* [HIHG genotype data](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MAGENTA_genotype.md).
	* Lead analysts: **Makaela Mews.**
* [HIHG Non-Hispanic White blood alternative splicing QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/MAGENTA_NHW_blood_splicing_qtl.md).
	* Lead analysts: **Makaela Mews.**

## I
* [Integrative multi-trait colocalization](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/integration/integration_colocalization.md). This document outlines the multi-trait colocalization analysis performed using FunGen-xQTL resources, with existing and some of the new method developed by the AWG.
	* Lead analysts: **Xuewei Cao (coloc-boost), Kushal Dey (coloc-boost), Chengran Yang (HyPrColoc).**
* [Integrative multivariate analysis](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/integration/integration_multivariate.md). This document outlines various schemes we perform multi-variate analysis across different xQTL using FunGen-xQTL resources, with innovative use of fine-mapping, colocalization and multivariate adaptive shrinkage (MASH) results.
	* Lead analysts: **Ru Feng.**
* [Integrative simple xQTL meta-analysis](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/integration/integration_metal.md). This document outlines the simple meta-analysis performed in the FunGen-xQTL study in contexts as it fits, using fixed and random effects meta-analysis (METAL software).
	* Lead analysts: **Xuanhe Chen.**

## K
* [Knight ADRC brain gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/Knight_ADRC_brain_expression.md).
	* Lead analysts: **Chunming Liu.**
* [Knight ADRC brain gene expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/Knight_ADRC_brain_expression_qtl.md).
	* Lead analysts: **Chunming Liu.**
* [Knight ADRC brain metabolomics](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/Knight_ADRC_brain_metabolomics.md).
	* Lead analysts: **Zining Qi.**
* [Knight ADRC brain metabolomics QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/Knight_ADRC_brain_metabolomics_qtl.md).
	* Lead analysts: **Zining Qi.**
* [Knight ADRC brain methylation](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/Knight_ADRC_brain_methylation.md).
	* Lead analysts: **Alexandre Pelletier.**
* [Knight ADRC brain methylation QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/Knight_ADRC_brain_methylation_qtl.md).
	* Lead analysts: **Alexandre Pelletier.**
* [Knight ADRC brain proteomics](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/Knight_ADRC_brain_proteomics.md). Charles F And Joanne Knight Alzheimer's Disease Research Center (Knight-ADRC).
	* Lead analysts: **Zining Qi.**
* [Knight ADRC brain proteomics QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/Knight_ADRC_brain_proteomics_qtl.md). Charles F.
	* Lead analysts: **Zining Qi(zq2209@cumc.columbia.edu).**
* [Knight ADRC brain splicing](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/Knight_ADRC_brain_splicing.md).
	* Lead analysts: **Xuanhe Chen.**
* [Knight ADRC brain splicing QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/Knight_ADRC_brain_splicing_qtl.md).
	* Lead analysts: **Xuanhe Chen.**
* [Knight ADRC CSF metabolomics](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/Knight_ADRC_CSF_metabolomics.md).
	* Lead analysts: **Zining Qi.**
* [Knight ADRC CSF metabolomics QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/Knight_ADRC_CSF_metabolomics_qtl.md).
	* Lead analysts: **Zining Qi.**
* [Knight ADRC CSF proteomics](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/Knight_ADRC_CSF_proteomics.md).
	* Lead analysts: **Zining Qi.**
* [Knight ADRC CSF proteomics QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/Knight_ADRC_CSF_proteomics_qtl.md).
	* Lead analysts: **Zining Qi.**
* [Knight ADRC genotype data](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/Knight_ADRC_genotype.md).
	* Lead analysts: **Zining Qi.**

## M
* [MAGENTA African American blood alternative splicing QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/MAGENTA_AA_blood_splicing_qtl.md).
	* Lead analysts: **Makaela Mews.**
* [MAGENTA African American blood gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MAGENTA_AA_blood_expression.md). Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P.
	* Lead analysts: **Makaela Mews (analyst); Dr.**
* [MAGENTA African American Blood Gene Expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/MAGENTA_AA_blood_expression_qtl.md). Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P.
	* Lead analysts: **Makaela Mews (analyst; mxm1368@case.edu);  Dr.**
* [MAGENTA Non-Hispanic White blood alternative splicing](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MAGENTA_NHW_blood_splicing.md).
	* Lead analysts: **Makaela Mews.**
* [MAGENTA Non-Hispanic White blood gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MAGENTA_NHW_blood_expression.md). Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P.
	* Lead analysts: **Makaela Mews (analyst); Dr.**
* [MAGENTA Non-Hispanic White Blood Gene Expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/MAGENTA_NHW_blood_expression_qtl.md). Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P.
	* Lead analysts: **Makaela Mews (analyst; mxm1368@case.edu);  Dr.**
* [MiGA genotype data](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MiGA_genotype.md). Microglia Genomic Atlas from the Netherlands Brain Bank (NBB) and the Neuropathology Brain Bank and Research CoRE at Mount Sinai Hospital.
	* Lead analysts: **Travyse Edwards.**
* [MiGA multi-brain region eQTL integration](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/integration/MiGA_integration.md). MiGA eQTL multivariate adaptive shrinkage and multivariate fine-mapping.
	* Lead analysts: **Travyse Edwards.**
* [MiGA multi-brain region gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MiGA_brain_expression.md). A genetic and transcriptomic resource comprised of 255 primary human microglia samples isolated ex vivo from four different brain regions of 100 human subjects with neurodegenerative, neurological, or neuropsychiatric disorders, as well as unaffected controls.
	* Lead analysts: **Travyse Edwards.**
* [MiGA multi-brain region gene expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/MiGA_brain_expression_qtl.md). A genetic and transcriptomic resource comprised of 255 primary human microglia samples isolated ex vivo from four different brain regions of 100 human subjects with neurodegenerative, neurological, or neuropsychiatric disorders, as well as unaffected controls.
	* Lead analysts: **Travyse Edwards.**
* [MSBB brain alternative splicing](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MSBB_brain_splicing.md).
	* Lead analysts: **Minghui Wang.**
* [MSBB brain alternative splicing QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/MSBB_brain_splicing_qtl.md).
	* Lead analysts: **Minghui Wang.**
* [MSBB brain gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MSBB_brain_expression.md).
	* Lead analysts: **Minghui Wang.**
* [MSBB brain gene expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/MSBB_brain_expression_qtl.md).
	* Lead analysts: **Minghui Wang.**
* [MSBB brain methylation](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MSBB_brain_methylation.md).
	* Lead analysts: **Alexandre Pelletier.**
* [MSBB brain methylation QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/MSBB_brain_methylation_qtl.md).
	* Lead analysts: **Alexandre Pelletier.**
* [MSBB brain proteomics](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MSBB_proteomics.md).
	* Lead analysts: **Minghui Wang.**
* [MSBB brain proteomics QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/MSBB_proteomics_qtl.md).
	* Lead analysts: **Minghui Wang.**
* [MSBB WGS data](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/MSBB_WGS.md). MSBB whole-genome sequence data.
	* Lead analysts: **Minghui Wang and Julia TCW.**

## P
* [PART working group brain methylation](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/PART_brain_methylation.md).
	* Lead analysts: **Alexandre Pelletier, Corey McMillan.**
* [PART working group brain methylation QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/PART_brain_methylation_qtl.md).
	* Lead analysts: **Alexandre Pelletier, Corey McMillan.**

## R
* [ROSMAP AC alternative splicing](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_AC_splicing.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) AC alternative splicing.
	* Lead analysts: **Frank Grenn.**
* [ROSMAP AC alternative splicing QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_AC_splicing_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) AC alternative splicing.
	* Lead analysts: **Frank Grenn.**
* [ROSMAP AC gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_AC_expression.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) AC gene expression.
	* Lead analysts: **Frank Grenn.**
* [ROSMAP AC gene expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_AC_expression_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) AC gene expression.
	* Lead analysts: **Frank Grenn.**
* [ROSMAP Covariates data](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_covariates.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) covariates data.
	* Lead analysts: **There are couple of versions of covariates files been used for QTL calling, which may lead to inconsistency of data production and downstream integration analysis.**
* [ROSMAP DLPFC alternative splicing](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_DLPFC_splicing.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC alternative splicing.
	* Lead analysts: **Xuanhe Chen; Shrishtee Kandoi.**
* [ROSMAP DLPFC alternative splicing QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_DLPFC_splicing_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC alternative splicing.
	* Lead analysts: **Shrishtee Kandoi and Xuanhe Chen (xuanhechenxhc@163.com).**
* [ROSMAP DLPFC gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_DLPFC_expression.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC gene expression.
	* Lead analysts: **Xuanhe Chen (xuanhechenxhc@163.com).**
* [ROSMAP DLPFC gene expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_DLPFC_expression_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC gene expression.
	* Lead analysts: **Shrishtee Kandoi; Frank Grenn.**
* [ROSMAP DLPFC H3K9ac](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_DLPFC_ChIPSeq_H3K9ac.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) h3k9ac data-set.
	* Lead analysts: **Xuanhe Chen; Hao Sun; Hans Klein.**
* [ROSMAP DLPFC H3K9ac QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_DLPFC_ChIPSeq_H3K9ac_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) h3k9ac QTL analysis using the FGC xQTL pipeline.
	* Lead analysts: **Xuanhe Chen (xuanhechenxhc@163.com).**
* [ROSMAP DLPFC methylation](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_DLPFC_methylation.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC methylation data-set.
	* Lead analysts: **Alexandre Pelletier, Jiajun Tao.**
* [ROSMAP DLPFC methylation QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_DLPFC_methylation_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC methylation data-set.
	* Lead analysts: **Alexandre Pelletier.**
* [ROSMAP DLPFC protein expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_DLPFC_proteomics.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC protein expression.
	* Lead analysts: **Zining Qi.**
* [ROSMAP DLPFC protein expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_DLPFC_proteomics_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC protein expression.
	* Lead analysts: **Zining Qi(zq2209@cumc.columbia.edu).**
* [ROSMAP multi-trait QTL integration analysis](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/integration/ROSMAP_integration.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) multi-trait QTL integration.
	* Lead analysts: **Ru Feng.**
* [ROSMAP PCC alternative splicing](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_PCC_splicing.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) PCC alternative splicing.
	* Lead analysts: **Frank Grenn.**
* [ROSMAP PCC alternative splicing QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_PCC_splicing_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) PCC alternative splicing.
	* Lead analysts: **Frank Grenn.**
* [ROSMAP PCC Expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_PCC_expression_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) PCC alternative splicing.
	* Lead analysts: **Frank Grenn.**
* [ROSMAP PCC gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_PCC_expression.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) PCC alternative splicing.
	* Lead analysts: **Frank Grenn.**
* [ROSMAP RNA-seq microglia gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_microglia_expression.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) microglia data-set.
	* Lead analysts: **Travyse Edwards.**
* [ROSMAP RNA-seq microglia gene expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_microglia_expression_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) microglia data-set.
	* Lead analysts: **Travyse Edwards.**
* [ROSMAP RNA-seq monocyte gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_monocyte_expression.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) monocyte data-set.
	* Lead analysts: **Travyse Edwards.**
* [ROSMAP RNA-seq monocyte QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_monocyte_expression_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) monocyte data-set.
	* Lead analysts: **Travyse Edwards.**
* [ROSMAP snRNA-seq pseudo-bulk gene expression](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_snRNAseq_pseudo_bulk.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) snRNA-seq from different cells in Dorsolateral Prefrontal Cortex (DLPFC).
	* Lead analysts: **Hao Sun and Masashi Fujita.**
* [ROSMAP snRNA-seq pseudo-bulk gene expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/ROSMAP_snRNAseq_pseudo_bulk_qtl.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) snRNA-seq from different cells in Dorsolateral Prefrontal Cortex (DLPFC).
	* Lead analysts: **Hao Sun (eQTL), Masashi Fujita (eQTL), Haochen Sun (fine-mapping), Jiajun Tao (replication).**
* [ROSMAP WGS data](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/ROSMAP_WGS.md). Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) whole-genome sequence data.
	* Lead analysts: **Hao Sun and Xuanhe Chen.**

## S
* [STARNET genotype data](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/STARNET_genotype.md). STARNET is an RNA expression study of various disease-relevant tissues obtained from living patients with cardiovascular disease (CVD).
	* Lead analysts: **Travyse Edwards.**
* [STARNET macrophage gene expression QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/STARNET_macrophage_qtl.md). STARNET is an RNA expression study of various disease-relevant tissues obtained from living patients with cardiovascular disease (CVD).
	* Lead analysts: **- Contact Name: Travyse Edwards.**

## W
* [WHICAP (pilot) plasma metabolomics](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/omics/WHICAP_plasma_metabolomics.md).
	* Lead analysts: **Zining Qi.**
* [WHICAP (pilot) plasma metabolomics QTL](https://github.com/cumc/xqtl-analysis/tree/main/descriptor/qtl/WHICAP_plasma_metabolomics_qtl.md).
	* Lead analysts: **Zining Qi.**
