<!DOCTYPE html>
<html lang="en-us" dir="ltr">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="ROSMAP DLPFC protein expression QTL #  Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC protein expression. Please refer to this document for an overview of the ROSMAP project.
Contact #  Zining Qi( zq2209@cumc.columbia.edu)
Study Overview #   Study name : ROSMAP DLPFC proteomics QTL Study Description : Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) proteomics QTL analysis summary statistics using the FGC xQTL pipeline.">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ffffff">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#343a40">
<meta name="color-scheme" content="light dark"><meta property="og:title" content="ROSMAP DLPFC protein expression QTL" />
<meta property="og:description" content="ROSMAP DLPFC protein expression QTL #  Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC protein expression. Please refer to this document for an overview of the ROSMAP project.
Contact #  Zining Qi( zq2209@cumc.columbia.edu)
Study Overview #   Study name : ROSMAP DLPFC proteomics QTL Study Description : Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) proteomics QTL analysis summary statistics using the FGC xQTL pipeline." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://cumc.github.io/xqtl-data/r/rosmap-dlpfc-protein-expression-qtl/" /><meta property="article:section" content="r" />



<title>ROSMAP DLPFC protein expression QTL | FunGen-xQTL Data Repository</title>
<link rel="manifest" href="/xqtl-data/manifest.json">
<link rel="icon" href="/xqtl-data/favicon.png" >
<link rel="canonical" href="https://cumc.github.io/xqtl-data/r/rosmap-dlpfc-protein-expression-qtl/">
<link rel="stylesheet" href="/xqtl-data/book.min.33a48f5432973b8ff9a82679d9e45d67f2c15d4399bd2829269455cfe390b5e8.css" integrity="sha256-M6SPVDKXO4/5qCZ52eRdZ/LBXUOZvSgpJpRVz&#43;OQteg=" crossorigin="anonymous">
  <script defer src="/xqtl-data/flexsearch.min.js"></script>
  <script defer src="/xqtl-data/en.search.min.72cac54aadb23f4cf5bda8ed7c97d23840ac2f5e40787e1309feebe9c16daaa3.js" integrity="sha256-csrFSq2yP0z1vajtfJfSOECsL15AeH4TCf7r6cFtqqM=" crossorigin="anonymous"></script>

  <script defer src="/xqtl-data/sw.min.5a8d9ccf5ea73dcbbb1f34b183236a28dd36cc6d6540bb5b46431b4e00517252.js" integrity="sha256-Wo2cz16nPcu7HzSxgyNqKN02zG1lQLtbRkMbTgBRclI=" crossorigin="anonymous"></script>
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->
  
</head>
<body dir="ltr">
  <input type="checkbox" class="hidden toggle" id="menu-control" />
  <input type="checkbox" class="hidden toggle" id="toc-control" />
  <main class="container flex">
    <aside class="book-menu">
      <div class="book-menu-content">
        
  <nav>
<h2 class="book-brand">
  <a class="flex align-center" href="/xqtl-data/"><span>FunGen-xQTL Data Repository</span>
  </a>
</h2>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>












  <ul>
<li>Alphabetic List of Data Available
<ul>
<li>
  <a href="/xqtl-data/a/">A</a></li>
<li>
  <a href="/xqtl-data/d/">D</a></li>
<li>
  <a href="/xqtl-data/e/">E</a></li>
<li>
  <a href="/xqtl-data/f/">F</a></li>
<li>
  <a href="/xqtl-data/h/">H</a></li>
<li>
  <a href="/xqtl-data/i/">I</a></li>
<li>
  <a href="/xqtl-data/k/">K</a></li>
<li>
  <a href="/xqtl-data/m/">M</a></li>
<li>
  <a href="/xqtl-data/n/">N</a></li>
<li>
  <a href="/xqtl-data/p/">P</a></li>
<li>
  <a href="/xqtl-data/r/">R</a></li>
<li>
  <a href="/xqtl-data/s/">S</a></li>
<li>
  <a href="/xqtl-data/t/">T</a></li>
<li>
  <a href="/xqtl-data/w/">W</a></li>
</ul>
</li>
</ul>










</nav>




  <script>(function(){var a=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(b){localStorage.setItem("menu.scrollTop",a.scrollTop)}),a.scrollTop=localStorage.getItem("menu.scrollTop")})()</script>


 
      </div>
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/xqtl-data/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>ROSMAP DLPFC protein expression QTL</strong>

  <label for="toc-control">
    
    <img src="/xqtl-data/svg/toc.svg" class="book-icon" alt="Table of Contents" />
    
  </label>
</div>


  
  <aside class="hidden clearfix">
    
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#analysis-status">Analysis Status</a></li>
    <li><a href="#links-to-qtl-analysis-notebooks">Links to QTL analysis notebooks</a>
      <ul>
        <li><a href="#association-data-preprocessing">Association data preprocessing</a></li>
        <li><a href="#association-scan-using-tensorqtl-and-summary-statistics-standardization">Association scan using TensorQTL and summary statistics standardization</a></li>
      </ul>
    </li>
    <li><a href="#analysis-details">Analysis Details</a>
      <ul>
        <li><a href="#sample-size-tracking-through-the-analysis-workflow">Sample size tracking through the analysis workflow</a></li>
        <li><a href="#qc-and-normalization-details-for-phenotype-data">QC and Normalization Details for Phenotype Data</a></li>
        <li><a href="#genotype-parameters">Genotype parameters</a></li>
        <li><a href="#covariate-parameters">Covariate parameters</a></li>
        <li><a href="#tensorqtl-association-scan-parameters">TensorQTL association scan parameters</a></li>
      </ul>
    </li>
    <li><a href="#dataset-description">Dataset Description</a>
      <ul>
        <li><a href="#paths-to-genotype-matrix">Path(s) to genotype matrix</a></li>
        <li><a href="#paths-to-omics-data-matrix">Path(s) to omics-data matrix</a></li>
        <li><a href="#paths-to-covariate-data-matrix">Path(s) to covariate data matrix</a></li>
        <li><a href="#paths-to-qtl-results">Path(s) to QTL results</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-model">Path(s) to fine-mapping with SuSiE model</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-rss-model">Path(s) to fine-mapping with SuSiE RSS model</a></li>
      </ul>
    </li>
  </ul>
</nav>



  </aside>
  
 
      </header>

      
      
  <article class="markdown book-article"><h1 id="rosmap-dlpfc-protein-expression-qtl">
  ROSMAP DLPFC protein expression QTL
  <a class="anchor" href="#rosmap-dlpfc-protein-expression-qtl">#</a>
</h1>
<p>Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC protein expression.
Please refer to 
  <a href="../study_info/ROSMAP.md">this document</a> for an overview of the ROSMAP project.</p>
<h2 id="contact">
  Contact
  <a class="anchor" href="#contact">#</a>
</h2>
<p>Zining Qi(
  <a href="mailto:zq2209@cumc.columbia.edu">zq2209@cumc.columbia.edu</a>)</p>
<h2 id="study-overview">
  Study Overview
  <a class="anchor" href="#study-overview">#</a>
</h2>
<ul>
<li>Study name : ROSMAP DLPFC proteomics QTL</li>
<li>Study Description : Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) proteomics QTL analysis summary statistics using the FGC xQTL pipeline.
Samples' phenotype information (sex, age, race etc.) can be found in ROSMAP metadata, see &ldquo;Other information&rdquo; section in 
  <a href="../study_info/ROSMAP.md">this document</a></li>
</ul>
<h2 id="analysis-status">
  Analysis Status
  <a class="anchor" href="#analysis-status">#</a>
</h2>
<p>TransQTL association: Finished.</p>
<h2 id="links-to-qtl-analysis-notebooks">
  Links to QTL analysis notebooks
  <a class="anchor" href="#links-to-qtl-analysis-notebooks">#</a>
</h2>
<p>See notebooks in:</p>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Wang_Columbia/pqtl">https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Wang_Columbia/pqtl</a>
The notebooks in this folder contain the commands and data wrangling codes for analysis of the proteomics data in ROSMAP. (data wrangling exist because not all data are processed using the xqtl-pipeline from the beginning and need to be reformatted to fit one intermediate step of the pipeline).</li>
</ul>
<h3 id="association-data-preprocessing">
  Association data preprocessing
  <a class="anchor" href="#association-data-preprocessing">#</a>
</h3>
<p>phenotype preprocessing can be run in parallel but covariate preprocessing need to be done last since it requires both phenotype data and PCs from PCA in genotype preprocessing.</p>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/ROSMAP/genotype/genotype_preprocessing.ipynb">genotype_preprocessing.ipynb</a> shows the commands used for genotype processing and preparation steps, including QC and formatting for genotype data.</li>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/ROSMAP/pqtl/genotype_pca.ipynb">genotype_pca.ipynb</a> shows the commands used for computing genotype PC for population structure inference.</li>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/ROSMAP/pqtl/phenotype_preprocessing.ipynb">phenotype_preprocessing.ipynb</a> shows the commands used for the phenotype data formatting steps. QC is not included (should be performed already elsewhere).</li>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/ROSMAP/pqtl/covariates_preprocessing.ipynb">covariate_preprocessing.ipynb</a> shows the commands used for the covariate data processing and preparation steps, including icomputing molecular phenotype PCA for hidden factor analysis and merging all covariates together.</li>
</ul>
<h3 id="association-scan-using-tensorqtl-and-summary-statistics-standardization">
  Association scan using TensorQTL and summary statistics standardization
  <a class="anchor" href="#association-scan-using-tensorqtl-and-summary-statistics-standardization">#</a>
</h3>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/ROSMAP/pqtl/cis_association.ipynb">cis_association.ipynb</a> provides information and result of cis-QTL analysis.</li>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/ROSMAP/pqtl/trans_association.ipynb">trans_association.ipynb</a> provides information and result of AD genes trans-QTL analysis.</li>
</ul>
<h2 id="analysis-details">
  Analysis Details
  <a class="anchor" href="#analysis-details">#</a>
</h2>
<h3 id="sample-size-tracking-through-the-analysis-workflow">
  Sample size tracking through the analysis workflow
  <a class="anchor" href="#sample-size-tracking-through-the-analysis-workflow">#</a>
</h3>
<table>
<thead>
<tr>
<th>Step</th>
<th>Sample Size</th>
</tr>
</thead>
<tbody>
<tr>
<td>Raw data</td>
<td>596</td>
</tr>
<tr>
<td>After overlapping with ROSMAP WGS</td>
<td>512</td>
</tr>
<tr>
<td>After overlapping with ROSMAP joint call</td>
<td>416</td>
</tr>
</tbody>
</table>
<h3 id="qc-and-normalization-details-for-phenotype-data">
  QC and Normalization Details for Phenotype Data
  <a class="anchor" href="#qc-and-normalization-details-for-phenotype-data">#</a>
</h3>
<p>Used proteomics data directly without further QC. For QCs performed on data please refer to <code>descriptor/omics/ROSMAP_DLPFC_proteomics.md</code></p>
<h3 id="genotype-parameters">
  Genotype parameters
  <a class="anchor" href="#genotype-parameters">#</a>
</h3>
<ul>
<li>Kinship coefficient for related individuals: 0.0625</li>
<li>Minor allele count cutoff for single variant analysis: 0</li>
<li>Variant level missingness threshold: 0.1</li>
<li>Sample level missingness threshold: 0.1</li>
<li>HWE cutoof: 1E-08</li>
</ul>
<h3 id="covariate-parameters">
  Covariate parameters
  <a class="anchor" href="#covariate-parameters">#</a>
</h3>
<p>PCA analysis:</p>
<ul>
<li>Minor allele count cutoff for PCA: 5</li>
<li>LD pruning via PLINK for PCA analysis: window 50, shift 10, r2 0.1</li>
<li>Tolerance of missingness in covariates: 0.3</li>
<li>number of pc that captured more than 70% PVE were kept
Hidden Factor analysis - PCA</li>
<li>Use Marchenko-Pastur limit to choose PCs
Final covariates: Sex, Age at death, PMI, PCs(12), hidden factors(41)</li>
</ul>
<h3 id="tensorqtl-association-scan-parameters">
  TensorQTL association scan parameters
  <a class="anchor" href="#tensorqtl-association-scan-parameters">#</a>
</h3>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/resource/extended_cis_before_winsorize.tsv">Extened TADB windows</a> for cis analysis.</li>
<li>Trans window for the up and downstream radius to analyze around the region of interest, in units of bp: 1000000</li>
<li>Minor allele count cutoff: 5</li>
<li>Minor allele frequency cutoff: 5/(2.0*416) = 0.00422297</li>
</ul>
<h2 id="dataset-description">
  Dataset Description
  <a class="anchor" href="#dataset-description">#</a>
</h2>
<h3 id="paths-to-genotype-matrix">
  Path(s) to genotype matrix
  <a class="anchor" href="#paths-to-genotype-matrix">#</a>
</h3>
<ul>
<li>Wang Lab, <code>/mnt/vast/hpc/csg/FunGen_xQTL/ROSMAP/Genotype</code> Whole Genotype</li>
</ul>
<pre tabindex="0"><code>$ ls -lh *.{bim,bed,fam}
-rw-r--r-- 1 zq2209 zq2209  27G Aug 14 19:48 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.bed
-rw-r--r-- 1 zq2209 zq2209 3.6G Aug 14 19:47 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 19:47 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.fam
</code></pre><ul>
<li>Wang Lab, <code>/mnt/vast/hpc/csg/FunGen_xQTL/ROSMAP/Genotype/plink_by_chr</code> Genotype by chromosome</li>
</ul>
<pre tabindex="0"><code>$ ls -lh *.{bim,bed,fam}
-rw-r--r-- 1 zq2209 zq2209 1.3G Aug 14 22:16 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.10.bed
-rw-r--r-- 1 zq2209 zq2209 188M Aug 14 22:16 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.10.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 22:16 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.10.fam
-rw-r--r-- 1 zq2209 zq2209 1.3G Aug 14 22:16 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.11.bed
-rw-r--r-- 1 zq2209 zq2209 183M Aug 14 22:16 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.11.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 22:16 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.11.fam
-rw-r--r-- 1 zq2209 zq2209 1.3G Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.12.bed
-rw-r--r-- 1 zq2209 zq2209 179M Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.12.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.12.fam
-rw-r--r-- 1 zq2209 zq2209 941M Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.13.bed
-rw-r--r-- 1 zq2209 zq2209 133M Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.13.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.13.fam
-rw-r--r-- 1 zq2209 zq2209 870M Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.14.bed
-rw-r--r-- 1 zq2209 zq2209 121M Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.14.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.14.fam
-rw-r--r-- 1 zq2209 zq2209 797M Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.15.bed
-rw-r--r-- 1 zq2209 zq2209 111M Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.15.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.15.fam
-rw-r--r-- 1 zq2209 zq2209 902M Aug 14 22:28 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.16.bed
-rw-r--r-- 1 zq2209 zq2209 126M Aug 14 22:28 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.16.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 22:28 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.16.fam
-rw-r--r-- 1 zq2209 zq2209 796M Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.17.bed
-rw-r--r-- 1 zq2209 zq2209 113M Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.17.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.17.fam
-rw-r--r-- 1 zq2209 zq2209 747M Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.18.bed
-rw-r--r-- 1 zq2209 zq2209 104M Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.18.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.18.fam
-rw-r--r-- 1 zq2209 zq2209 639M Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.19.bed
-rw-r--r-- 1 zq2209 zq2209  91M Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.19.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.19.fam
-rw-r--r-- 1 zq2209 zq2209 2.2G Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.1.bed
-rw-r--r-- 1 zq2209 zq2209 293M Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.1.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.1.fam
-rw-r--r-- 1 zq2209 zq2209 628M Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.20.bed
-rw-r--r-- 1 zq2209 zq2209  89M Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.20.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.20.fam
-rw-r--r-- 1 zq2209 zq2209 367M Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.21.bed
-rw-r--r-- 1 zq2209 zq2209  53M Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.21.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:55 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.21.fam
-rw-r--r-- 1 zq2209 zq2209 397M Aug 14 22:22 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.22.bed
-rw-r--r-- 1 zq2209 zq2209  58M Aug 14 22:22 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.22.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 22:22 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.22.fam
-rw-r--r-- 1 zq2209 zq2209 2.3G Aug 14 22:29 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.2.bed
-rw-r--r-- 1 zq2209 zq2209 315M Aug 14 22:29 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.2.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 22:29 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.2.fam
-rw-r--r-- 1 zq2209 zq2209 1.9G Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.3.bed
-rw-r--r-- 1 zq2209 zq2209 254M Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.3.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.3.fam
-rw-r--r-- 1 zq2209 zq2209 1.9G Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.4.bed
-rw-r--r-- 1 zq2209 zq2209 248M Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.4.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.4.fam
-rw-r--r-- 1 zq2209 zq2209 1.7G Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.5.bed
-rw-r--r-- 1 zq2209 zq2209 231M Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.5.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.5.fam
-rw-r--r-- 1 zq2209 zq2209 1.7G Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.6.bed
-rw-r--r-- 1 zq2209 zq2209 220M Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.6.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.6.fam
-rw-r--r-- 1 zq2209 zq2209 1.6G Aug 14 22:11 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.7.bed
-rw-r--r-- 1 zq2209 zq2209 214M Aug 14 22:11 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.7.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 22:11 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.7.fam
-rw-r--r-- 1 zq2209 zq2209 1.5G Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.8.bed
-rw-r--r-- 1 zq2209 zq2209 199M Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.8.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:53 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.8.fam
-rw-r--r-- 1 zq2209 zq2209 1.2G Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.9.bed
-rw-r--r-- 1 zq2209 zq2209 160M Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.9.bim
-rw-r--r-- 1 zq2209 zq2209  23K Aug 14 21:54 ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.9.fam
</code></pre><h3 id="paths-to-omics-data-matrix">
  Path(s) to omics-data matrix
  <a class="anchor" href="#paths-to-omics-data-matrix">#</a>
</h3>
<ul>
<li>Wang Lab, <code>/mnt/vast/hpc/csg/zq2209/data_production/proteomics/rosmap/pheno</code></li>
</ul>
<pre tabindex="0"><code>$ ls -lh *.{gz}
-rw-r--r-- 1 zq2209 zq2209   35M Jun  1 13:09  rosmap_pheno.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.5M Jun  1 13:09  rosmap_pheno_chr10.bed.gz
-rw-r--r-- 1 zq2209 zq2209  2.2M Jun  1 13:09  rosmap_pheno_chr11.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.9M Jun  1 13:09  rosmap_pheno_chr12.bed.gz
-rw-r--r-- 1 zq2209 zq2209  655K Jun  1 13:09  rosmap_pheno_chr13.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.3M Jun  1 13:09  rosmap_pheno_chr14.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.1M Jun  1 13:09  rosmap_pheno_chr15.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.5M Jun  1 13:09  rosmap_pheno_chr16.bed.gz
-rw-r--r-- 1 zq2209 zq2209  2.1M Jun  1 13:09  rosmap_pheno_chr17.bed.gz
-rw-r--r-- 1 zq2209 zq2209  509K Jun  1 13:09  rosmap_pheno_chr18.bed.gz
-rw-r--r-- 1 zq2209 zq2209  2.1M Jun  1 13:09  rosmap_pheno_chr19.bed.gz
-rw-r--r-- 1 zq2209 zq2209  3.5M Jun  1 13:09  rosmap_pheno_chr1.bed.gz
-rw-r--r-- 1 zq2209 zq2209 1007K Jun  1 13:09  rosmap_pheno_chr20.bed.gz
-rw-r--r-- 1 zq2209 zq2209  299K Jun  1 13:09  rosmap_pheno_chr21.bed.gz
-rw-r--r-- 1 zq2209 zq2209  901K Jun  1 13:09  rosmap_pheno_chr22.bed.gz
-rw-r--r-- 1 zq2209 zq2209  2.5M Jun  1 13:09  rosmap_pheno_chr2.bed.gz
-rw-r--r-- 1 zq2209 zq2209  2.1M Jun  1 13:09  rosmap_pheno_chr3.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.4M Jun  1 13:09  rosmap_pheno_chr4.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.7M Jun  1 13:09  rosmap_pheno_chr5.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.8M Jun  1 13:09  rosmap_pheno_chr6.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.6M Jun  1 13:09  rosmap_pheno_chr7.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.2M Jun  1 13:09  rosmap_pheno_chr8.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.4M Jun  1 13:09  rosmap_pheno_chr9.bed.gz
-rw-r--r-- 1 zq2209 zq2209  1.3M Jun  1 13:09  rosmap_pheno_chrX.bed.gz
-rw-r--r-- 1 zq2209 zq2209  9.3K Jun  1 13:09  rosmap_pheno_chrY.bed.gz
</code></pre><h3 id="paths-to-covariate-data-matrix">
  Path(s) to covariate data matrix
  <a class="anchor" href="#paths-to-covariate-data-matrix">#</a>
</h3>
<ul>
<li>Wang Lab, <code>/mnt/vast/hpc/csg/zq2209/data_production/proteomics/rosmap/cov</code></li>
</ul>
<pre tabindex="0"><code>$ ls -lh *.gz
-rw-r--r-- 1 zq2209 zq2209 201K Jun  1 13:13 rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.gz
</code></pre><h3 id="paths-to-qtl-results">
  Path(s) to QTL results
  <a class="anchor" href="#paths-to-qtl-results">#</a>
</h3>
<ul>
<li>Wang lab: <code>/mnt/vast/hpc/csg/molecular_phenotype_calling/pQTL_cis/rosmap</code></li>
</ul>
<pre tabindex="0"><code>ls -lh *.txt
-rw-r--r-- 1 zq2209 zq2209  76K May 22 13:50 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.10.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 487M May 22 13:47 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.10.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 116K May 22 13:53 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.11.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 678M May 22 13:49 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.11.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 103K May 22 13:53 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.12.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 629M May 22 13:50 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.12.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  36K May 22 13:46 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.13.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 233M May 22 13:45 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.13.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  66K May 22 13:48 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.14.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 424M May 22 13:46 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.14.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  58K May 22 13:52 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.15.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 317M May 22 13:51 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.15.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  83K May 22 13:54 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.16.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 514M May 22 13:51 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.16.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 114K May 22 13:59 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.17.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 693M May 22 13:55 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.17.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  27K May 22 13:55 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.18.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 169M May 22 13:54 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.18.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 112K May 22 13:59 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.19.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 831M May 22 13:55 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.19.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 187K May 22 13:42 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.1.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 1.1G May 22 13:36 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.1.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  54K May 22 13:58 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.20.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 334M May 22 13:56 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.20.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  16K May 22 13:56 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.21.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 102M May 22 13:56 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.21.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  50K May 22 13:55 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.22.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 331M May 22 13:53 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.22.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 131K May 22 13:44 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.2.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 755M May 22 13:40 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.2.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 110K May 22 13:44 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.3.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 637M May 22 13:40 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.3.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  71K May 22 13:43 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.4.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 450M May 22 13:41 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.4.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  87K May 22 13:39 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.5.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 521M May 22 13:36 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.5.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  95K May 22 13:48 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.6.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 803M May 22 13:44 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.6.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  85K May 22 13:38 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.7.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 545M May 22 13:35 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.7.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  64K May 22 13:38 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.8.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 387M May 22 13:36 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.8.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  75K May 22 13:48 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.9.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 475M May 22 13:46 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.9.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 2.4M May 22 13:59 pheno_recipe_rosmap_pheno.rosmap_cov.ROSMAP_NIA_WGS.leftnorm.filtered.filtered.prune.pca.resid.Marchenko_pc.emprical.cis_sumstats.txt
</code></pre><ul>
<li>FTP:</li>
</ul>
<h3 id="paths-to-fine-mapping-with-susie-model">
  Path(s) to fine-mapping with SuSiE model
  <a class="anchor" href="#paths-to-fine-mapping-with-susie-model">#</a>
</h3>
<h3 id="paths-to-fine-mapping-with-susie-rss-model">
  Path(s) to fine-mapping with SuSiE RSS model
  <a class="anchor" href="#paths-to-fine-mapping-with-susie-rss-model">#</a>
</h3>
</article>
 
      

      <footer class="book-footer">
        
  <div class="flex flex-wrap justify-between">





</div>



  <script>(function(){function a(c){const a=window.getSelection(),b=document.createRange();b.selectNodeContents(c),a.removeAllRanges(),a.addRange(b)}document.querySelectorAll("pre code").forEach(b=>{b.addEventListener("click",function(c){if(window.getSelection().toString())return;a(b.parentElement),navigator.clipboard&&navigator.clipboard.writeText(b.parentElement.textContent)})})})()</script>


 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      <div class="book-toc-content">
        
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#analysis-status">Analysis Status</a></li>
    <li><a href="#links-to-qtl-analysis-notebooks">Links to QTL analysis notebooks</a>
      <ul>
        <li><a href="#association-data-preprocessing">Association data preprocessing</a></li>
        <li><a href="#association-scan-using-tensorqtl-and-summary-statistics-standardization">Association scan using TensorQTL and summary statistics standardization</a></li>
      </ul>
    </li>
    <li><a href="#analysis-details">Analysis Details</a>
      <ul>
        <li><a href="#sample-size-tracking-through-the-analysis-workflow">Sample size tracking through the analysis workflow</a></li>
        <li><a href="#qc-and-normalization-details-for-phenotype-data">QC and Normalization Details for Phenotype Data</a></li>
        <li><a href="#genotype-parameters">Genotype parameters</a></li>
        <li><a href="#covariate-parameters">Covariate parameters</a></li>
        <li><a href="#tensorqtl-association-scan-parameters">TensorQTL association scan parameters</a></li>
      </ul>
    </li>
    <li><a href="#dataset-description">Dataset Description</a>
      <ul>
        <li><a href="#paths-to-genotype-matrix">Path(s) to genotype matrix</a></li>
        <li><a href="#paths-to-omics-data-matrix">Path(s) to omics-data matrix</a></li>
        <li><a href="#paths-to-covariate-data-matrix">Path(s) to covariate data matrix</a></li>
        <li><a href="#paths-to-qtl-results">Path(s) to QTL results</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-model">Path(s) to fine-mapping with SuSiE model</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-rss-model">Path(s) to fine-mapping with SuSiE RSS model</a></li>
      </ul>
    </li>
  </ul>
</nav>


 
      </div>
    </aside>
    
  </main>

  
</body>
</html>












