<!DOCTYPE html>
<html lang="en-us" dir="ltr">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="Knight ADRC genotype data #  Contact #  Zining Qi
Data Descriptions #  Raw genotype files #  MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.bed, MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.bim (N= 10,641,345), MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.fam (N= 441)were the plink set created by extracting the Washu samples using geno = 0.1 and Maf = 0.0005 filters, Which then was checked for the array duplication. If there are duplicate samples (by array) in the data they will be filtered basing on their missingness and array frequency information.">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ffffff">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#343a40">
<meta name="color-scheme" content="light dark"><meta property="og:title" content="Knight ADRC genotype data" />
<meta property="og:description" content="Knight ADRC genotype data #  Contact #  Zining Qi
Data Descriptions #  Raw genotype files #  MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.bed, MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.bim (N= 10,641,345), MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.fam (N= 441)were the plink set created by extracting the Washu samples using geno = 0.1 and Maf = 0.0005 filters, Which then was checked for the array duplication. If there are duplicate samples (by array) in the data they will be filtered basing on their missingness and array frequency information." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://cumc.github.io/xqtl-data/k/knight-adrc-genotype-data/" /><meta property="article:section" content="k" />



<title>Knight ADRC genotype data | FunGen-xQTL Data Repository</title>
<link rel="manifest" href="/xqtl-data/manifest.json">
<link rel="icon" href="/xqtl-data/favicon.png" >
<link rel="canonical" href="https://cumc.github.io/xqtl-data/k/knight-adrc-genotype-data/">
<link rel="stylesheet" href="/xqtl-data/book.min.33a48f5432973b8ff9a82679d9e45d67f2c15d4399bd2829269455cfe390b5e8.css" integrity="sha256-M6SPVDKXO4/5qCZ52eRdZ/LBXUOZvSgpJpRVz&#43;OQteg=" crossorigin="anonymous">
  <script defer src="/xqtl-data/flexsearch.min.js"></script>
  <script defer src="/xqtl-data/en.search.min.26ec5af939b7ea920bd0a2734b7344c1e2076b7ed37b22088900c60cf519ff8e.js" integrity="sha256-Juxa&#43;Tm36pIL0KJzS3NEweIHa37TeyIIiQDGDPUZ/44=" crossorigin="anonymous"></script>

  <script defer src="/xqtl-data/sw.min.5a8d9ccf5ea73dcbbb1f34b183236a28dd36cc6d6540bb5b46431b4e00517252.js" integrity="sha256-Wo2cz16nPcu7HzSxgyNqKN02zG1lQLtbRkMbTgBRclI=" crossorigin="anonymous"></script>
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->
  
</head>
<body dir="ltr">
  <input type="checkbox" class="hidden toggle" id="menu-control" />
  <input type="checkbox" class="hidden toggle" id="toc-control" />
  <main class="container flex">
    <aside class="book-menu">
      <div class="book-menu-content">
        
  <nav>
<h2 class="book-brand">
  <a class="flex align-center" href="/xqtl-data/"><span>FunGen-xQTL Data Repository</span>
  </a>
</h2>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>












  <ul>
<li>Alphabetic List of Data Available
<ul>
<li>
  <a href="/xqtl-data/a/">A</a></li>
<li>
  <a href="/xqtl-data/d/">D</a></li>
<li>
  <a href="/xqtl-data/e/">E</a></li>
<li>
  <a href="/xqtl-data/f/">F</a></li>
<li>
  <a href="/xqtl-data/h/">H</a></li>
<li>
  <a href="/xqtl-data/k/">K</a></li>
<li>
  <a href="/xqtl-data/m/">M</a></li>
<li>
  <a href="/xqtl-data/n/">N</a></li>
<li>
  <a href="/xqtl-data/p/">P</a></li>
<li>
  <a href="/xqtl-data/r/">R</a></li>
<li>
  <a href="/xqtl-data/s/">S</a></li>
<li>
  <a href="/xqtl-data/t/">T</a></li>
<li>
  <a href="/xqtl-data/w/">W</a></li>
</ul>
</li>
</ul>










</nav>




  <script>(function(){var a=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(b){localStorage.setItem("menu.scrollTop",a.scrollTop)}),a.scrollTop=localStorage.getItem("menu.scrollTop")})()</script>


 
      </div>
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/xqtl-data/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>Knight ADRC genotype data</strong>

  <label for="toc-control">
    
    <img src="/xqtl-data/svg/toc.svg" class="book-icon" alt="Table of Contents" />
    
  </label>
</div>


  
  <aside class="hidden clearfix">
    
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#data-descriptions">Data Descriptions</a>
      <ul>
        <li><a href="#raw-genotype-files">Raw genotype files</a></li>
        <li><a href="#genotype-processed-via-xqtl-pipeline">Genotype processed via xQTL pipeline:</a></li>
      </ul>
    </li>
  </ul>
</nav>



  </aside>
  
 
      </header>

      
      
  <article class="markdown book-article"><h1 id="knight-adrc-genotype-data">
  Knight ADRC genotype data
  <a class="anchor" href="#knight-adrc-genotype-data">#</a>
</h1>
<h2 id="contact">
  Contact
  <a class="anchor" href="#contact">#</a>
</h2>
<p>Zining Qi</p>
<h2 id="data-descriptions">
  Data Descriptions
  <a class="anchor" href="#data-descriptions">#</a>
</h2>
<h3 id="raw-genotype-files">
  Raw genotype files
  <a class="anchor" href="#raw-genotype-files">#</a>
</h3>
<p>MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.bed, MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.bim (N= 10,641,345), MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.fam (N= 441)were the plink set created by extracting the Washu samples using geno = 0.1 and Maf = 0.0005 filters, Which then was checked for the array duplication.
If there are duplicate samples (by array) in the data they will be filtered basing on their missingness and array frequency information.
Data provided is on GRCH38 genome build. We used TOPMED reference panel for imputation. The IDs in the fam were in the PA_DB_UID format .
Bim file follows Chr:SNP:Ref:Alt format for snp name and the Ref is set to A2 allele in the plink file.
Array information of all the samples was enclosed in the file: ID_Array_info.txt
Detailed QC steps used to process the data can be viewed in file:
Imputation QC protocol.pdf</p>
<ul>
<li>Data Location on CU cluster:</li>
</ul>
<pre tabindex="0"><code>/mnt/vast/hpc/csg/ftp_lisanwanglab_sync/ftp_fgc_xqtl/projects/SNParrayGeno/knightadrc-washu$ ls -lh
-rw-r--r-- 1 xc2610 root 184K Nov 29 13:35 'Imputation QC protocol.pdf'
-rw-r--r-- 1 xc2610 root 1.2G Nov 29 13:36  MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.bed
-rw-r--r-- 1 xc2610 root 372M Nov 29 13:36  MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.bim
-rw-r--r-- 1 xc2610 root  14K Nov 29 13:36  MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.fam
-rw-r--r-- 1 xc2610 root 1.1K Nov 29 13:36  MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.log
-rw-r--r-- 1 xc2610 root  959 Nov 29 13:36  README.txt
</code></pre><ul>
<li>Data Location on FTP:
<code>ftp_fgc_xqtl/projects/SNParrayGeno/knightadrc-washu/</code></li>
</ul>
<h3 id="genotype-processed-via-xqtl-pipeline">
  Genotype processed via xQTL pipeline:
  <a class="anchor" href="#genotype-processed-via-xqtl-pipeline">#</a>
</h3>
<p>For details how the phenotype data is processed please check 
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/knight/pqtl/genotype_preprocessing.ipynb">genotype_preprocessing</a></p>
<ul>
<li>Data Location on CU cluster: <code>/mnt/mfs/hgrcgrid/homes/zq2209/proteomics/knight/MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.plink_files_list.txt</code></li>
</ul>
</article>
 
      

      <footer class="book-footer">
        
  <div class="flex flex-wrap justify-between">





</div>



  <script>(function(){function a(c){const a=window.getSelection(),b=document.createRange();b.selectNodeContents(c),a.removeAllRanges(),a.addRange(b)}document.querySelectorAll("pre code").forEach(b=>{b.addEventListener("click",function(c){if(window.getSelection().toString())return;a(b.parentElement),navigator.clipboard&&navigator.clipboard.writeText(b.parentElement.textContent)})})})()</script>


 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      <div class="book-toc-content">
        
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#data-descriptions">Data Descriptions</a>
      <ul>
        <li><a href="#raw-genotype-files">Raw genotype files</a></li>
        <li><a href="#genotype-processed-via-xqtl-pipeline">Genotype processed via xQTL pipeline:</a></li>
      </ul>
    </li>
  </ul>
</nav>


 
      </div>
    </aside>
    
  </main>

  
</body>
</html>












