<!DOCTYPE html>
<html lang="en-us" dir="ltr">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="ROSMAP DLPFC alternative splicing QTL #  Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC alternative splicing. Please refer to this document for an overview of the ROSMAP project.
Contact #  Shrishtee Kandoi and Xuanhe Chen ( xuanhechenxhc@163.com)
Study Overview #   Study name : ROSMAP DLPFC sQTL Study Description : Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) sQTL analysis summary statistics using the FGC xQTL pipeline.">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ffffff">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#343a40">
<meta name="color-scheme" content="light dark"><meta property="og:title" content="ROSMAP DLPFC alternative splicing QTL" />
<meta property="og:description" content="ROSMAP DLPFC alternative splicing QTL #  Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC alternative splicing. Please refer to this document for an overview of the ROSMAP project.
Contact #  Shrishtee Kandoi and Xuanhe Chen ( xuanhechenxhc@163.com)
Study Overview #   Study name : ROSMAP DLPFC sQTL Study Description : Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) sQTL analysis summary statistics using the FGC xQTL pipeline." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://cumc.github.io/xqtl-data/r/rosmap-dlpfc-alternative-splicing-qtl/" /><meta property="article:section" content="r" />



<title>ROSMAP DLPFC alternative splicing QTL | FunGen-xQTL Data Repository</title>
<link rel="manifest" href="/xqtl-data/manifest.json">
<link rel="icon" href="/xqtl-data/favicon.png" >
<link rel="canonical" href="https://cumc.github.io/xqtl-data/r/rosmap-dlpfc-alternative-splicing-qtl/">
<link rel="stylesheet" href="/xqtl-data/book.min.33a48f5432973b8ff9a82679d9e45d67f2c15d4399bd2829269455cfe390b5e8.css" integrity="sha256-M6SPVDKXO4/5qCZ52eRdZ/LBXUOZvSgpJpRVz&#43;OQteg=" crossorigin="anonymous">
  <script defer src="/xqtl-data/flexsearch.min.js"></script>
  <script defer src="/xqtl-data/en.search.min.fd7011d93cabe050c70980ac5fda5d47a1920abc6a0f225733faa3ae96df1be4.js" integrity="sha256-/XAR2Tyr4FDHCYCsX9pdR6GSCrxqDyJXM/qjrpbfG&#43;Q=" crossorigin="anonymous"></script>

  <script defer src="/xqtl-data/sw.min.5a8d9ccf5ea73dcbbb1f34b183236a28dd36cc6d6540bb5b46431b4e00517252.js" integrity="sha256-Wo2cz16nPcu7HzSxgyNqKN02zG1lQLtbRkMbTgBRclI=" crossorigin="anonymous"></script>
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->
  
</head>
<body dir="ltr">
  <input type="checkbox" class="hidden toggle" id="menu-control" />
  <input type="checkbox" class="hidden toggle" id="toc-control" />
  <main class="container flex">
    <aside class="book-menu">
      <div class="book-menu-content">
        
  <nav>
<h2 class="book-brand">
  <a class="flex align-center" href="/xqtl-data/"><span>FunGen-xQTL Data Repository</span>
  </a>
</h2>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>












  <ul>
<li>Alphabetic List of Data Available
<ul>
<li>
  <a href="/xqtl-data/a/">A</a></li>
<li>
  <a href="/xqtl-data/d/">D</a></li>
<li>
  <a href="/xqtl-data/e/">E</a></li>
<li>
  <a href="/xqtl-data/f/">F</a></li>
<li>
  <a href="/xqtl-data/h/">H</a></li>
<li>
  <a href="/xqtl-data/i/">I</a></li>
<li>
  <a href="/xqtl-data/k/">K</a></li>
<li>
  <a href="/xqtl-data/m/">M</a></li>
<li>
  <a href="/xqtl-data/n/">N</a></li>
<li>
  <a href="/xqtl-data/p/">P</a></li>
<li>
  <a href="/xqtl-data/r/">R</a></li>
<li>
  <a href="/xqtl-data/s/">S</a></li>
<li>
  <a href="/xqtl-data/t/">T</a></li>
<li>
  <a href="/xqtl-data/w/">W</a></li>
</ul>
</li>
</ul>










</nav>




  <script>(function(){var a=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(b){localStorage.setItem("menu.scrollTop",a.scrollTop)}),a.scrollTop=localStorage.getItem("menu.scrollTop")})()</script>


 
      </div>
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/xqtl-data/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>ROSMAP DLPFC alternative splicing QTL</strong>

  <label for="toc-control">
    
    <img src="/xqtl-data/svg/toc.svg" class="book-icon" alt="Table of Contents" />
    
  </label>
</div>


  
  <aside class="hidden clearfix">
    
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#analysis-status">Analysis Status</a></li>
    <li><a href="#analysis-details">Analysis Details</a>
      <ul>
        <li><a href="#sample-size-tracking-through-the-analysis-workflow">Sample size tracking through the analysis workflow</a></li>
        <li><a href="#qc-and-normalization-details-for-phenotype-data">QC and Normalization Details for Phenotype Data</a></li>
        <li><a href="#genotype-parameters">Genotype parameters</a></li>
        <li><a href="#covariate-parameters">Covariate parameters</a></li>
        <li><a href="#tensorqtl-association-scan-parameters">TensorQTL association scan parameters</a></li>
      </ul>
    </li>
    <li><a href="#links-to-qtl-analysis-notebooks">Links to QTL analysis notebooks</a>
      <ul>
        <li><a href="#association-data-preprocessing">Association data preprocessing</a></li>
        <li><a href="#association-scan-using-tensorqtl-and-summary-statistics-standardization">Association scan using TensorQTL and summary statistics standardization</a></li>
        <li><a href="#susie-univariate-fine-mapping">SuSiE univariate fine mapping</a></li>
      </ul>
    </li>
    <li><a href="#dataset-details">Dataset Details</a>
      <ul>
        <li><a href="#paths-to-genotype-matrix">Path(s) to genotype matrix</a></li>
        <li><a href="#paths-to-omics-data-matrix">Path(s) to omics-data matrix</a></li>
        <li><a href="#paths-to-covariate-data-matrix">Path(s) to covariate data matrix</a></li>
        <li><a href="#paths-to-qtl-results">Path(s) to QTL results</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-model">Path(s) to fine-mapping with SuSiE model</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-rss-model">Path(s) to fine-mapping with SuSiE RSS model</a></li>
      </ul>
    </li>
  </ul>
</nav>



  </aside>
  
 
      </header>

      
      
  <article class="markdown book-article"><h1 id="rosmap-dlpfc-alternative-splicing-qtl">
  ROSMAP DLPFC alternative splicing QTL
  <a class="anchor" href="#rosmap-dlpfc-alternative-splicing-qtl">#</a>
</h1>
<p>Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) DLPFC alternative splicing.
Please refer to 
  <a href="../study_info/ROSMAP.md">this document</a> for an overview of the ROSMAP project.</p>
<h2 id="contact">
  Contact
  <a class="anchor" href="#contact">#</a>
</h2>
<p>Shrishtee Kandoi and Xuanhe Chen (
  <a href="mailto:xuanhechenxhc@163.com">xuanhechenxhc@163.com</a>)</p>
<h2 id="study-overview">
  Study Overview
  <a class="anchor" href="#study-overview">#</a>
</h2>
<ul>
<li>Study name : ROSMAP DLPFC sQTL</li>
<li>Study Description : Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) sQTL analysis summary statistics using the FGC xQTL pipeline.
Samples' phenotype information (sex, age, race etc.) can be found in ROSMAP metadata, see &ldquo;Other information&rdquo; section in 
  <a href="../study_info/ROSMAP.md">this document</a></li>
</ul>
<h2 id="analysis-status">
  Analysis Status
  <a class="anchor" href="#analysis-status">#</a>
</h2>
<p>TransQTL association: Need to be performed.</p>
<h2 id="analysis-details">
  Analysis Details
  <a class="anchor" href="#analysis-details">#</a>
</h2>
<h3 id="sample-size-tracking-through-the-analysis-workflow">
  Sample size tracking through the analysis workflow
  <a class="anchor" href="#sample-size-tracking-through-the-analysis-workflow">#</a>
</h3>
<table>
<thead>
<tr>
<th>Step</th>
<th>Sample Size</th>
</tr>
</thead>
<tbody>
<tr>
<td>Raw data</td>
<td>1141</td>
</tr>
<tr>
<td>After overlapping with ROSMAP WGS</td>
<td>839</td>
</tr>
</tbody>
</table>
<h3 id="qc-and-normalization-details-for-phenotype-data">
  QC and Normalization Details for Phenotype Data
  <a class="anchor" href="#qc-and-normalization-details-for-phenotype-data">#</a>
</h3>
<p>In the case of leafcutter data, any features with over 40% missing values across samples were removed. The remaining missing values were replaced with the mean of existing values, and introns with less than 0.005 variation were eliminated. A scaled normalization was performed on each intron cluster (representing a row of leafcutter data), followed by Quantile-Quantile Normalization on each sample (representing a column of leafcutter data). In contrast, psichomics phenotype data underwent a different imputation approach; zero-imputation was performed to retain &ldquo;no event detected&rdquo; information, and only scaled normalization was applied to each event (row).</p>
<h3 id="genotype-parameters">
  Genotype parameters
  <a class="anchor" href="#genotype-parameters">#</a>
</h3>
<ul>
<li>Kinship coefficient for related individuals: 0.0625</li>
<li>Minor allele count cutoff for single variant analysis: 5</li>
<li>Variant level missingness threshold: 0.1</li>
<li>Sample level missingness threshold: 0.1</li>
<li>HWE filter: 1e-06</li>
</ul>
<h3 id="covariate-parameters">
  Covariate parameters
  <a class="anchor" href="#covariate-parameters">#</a>
</h3>
<p>PCA analysis:</p>
<ul>
<li>MAF for PCA: 0.01</li>
<li>LD pruning via PLINK for PCA analysis: window 50, shift 10, r2 0.1</li>
<li>Tolerance of missingness in covariates: 0.3</li>
<li>number of pc that captured more than 70% PVE were kept
PEER analysis (Default values from PEER software):</li>
<li>The number of max iteration: 1000</li>
<li>Prior parameter Alpha_a: 0.001</li>
<li>Prior parameter Alpha_b: 0.1</li>
<li>Prior parameter Eps_a: 0.1</li>
<li>Prior parameter Eps_b: 10.0</li>
<li>Tolerance  parameter tol: 0.001</li>
<li>Tolerance parameter var_tol: 0.00001</li>
<li>minimum variance explained criteria to drop factors while training r2_tol: False</li>
<li>Convergence mode: &ldquo;fast&rdquo;
Final covariates: Sex, Age at death, PMI, PCs(13), PEER factors(60)</li>
</ul>
<h3 id="tensorqtl-association-scan-parameters">
  TensorQTL association scan parameters
  <a class="anchor" href="#tensorqtl-association-scan-parameters">#</a>
</h3>
<ul>
<li>cis window for the up and downstream radius to analyze around the region of interest, in units of bp: 1000000</li>
<li>Minor allele count cutoff: 5</li>
<li>Minor allele frequency cutoff: 5/(2.0*592) = 0.00422297</li>
</ul>
<h2 id="links-to-qtl-analysis-notebooks">
  Links to QTL analysis notebooks
  <a class="anchor" href="#links-to-qtl-analysis-notebooks">#</a>
</h2>
<p>See notebooks in:</p>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Zhang_BU/ROSMAP_DLPFC/">https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Zhang_BU/ROSMAP_DLPFC/</a>
The notebooks in this folder contain the commands and data wrangling codes for analysis of the eQTL and sQTL data in ROSMAP. (data wrangling exist because not all data are processed using the xqtl-pipeline from the beginning and need to be reformatted to fit one intermediate step of the pipeline). Since the sQTL phenotype pre-processing used eQTL STAR alignment output, some of the early stage code are shared.</li>
</ul>
<h3 id="association-data-preprocessing">
  Association data preprocessing
  <a class="anchor" href="#association-data-preprocessing">#</a>
</h3>
<p>phenotype preprocessing and genotype preprocessing can be run in parallel but covariate preprocessing needs to be done last since it requires both phenotype data and PCs from PCA in genotype preprocessing.</p>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Zhang_BU/ROSMAP_DLPFC/genotype/genotype_qc.ipynb">genotype_qc.ipynb</a> shows the commands used for genotype processing and preparation steps, including why we decided not to remove any outliers.</li>
<li>Also includes QC, PCA, and PCA result figures, some are inside the notebook and some are seperated PDF files. For example 
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Zhang_BU/ROSMAP_DLPFC/sQTL/leafcutter_PEER.diag.pdf">leafcutter_PEER.diag.pdf</a> is the view of PCA result of the sQTL analysis.</li>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Zhang_BU/ROSMAP_DLPFC/sQTL/phenotype_preprocessing_sQTL.ipynb">phenotype_preprocessing_sqtl.ipynb</a> shows the commands used for the phenotype data processing and preparation steps.</li>
<li>This notebook strated from some eQTL intermidiate output and convert them to bed format inputs taken by tensorQTL.</li>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Zhang_BU/ROSMAP_DLPFC/sQTL/covariate_preprocessing_sQTL.ipynb">covariate_preprocessing_sQTL.ipynb</a> shows the commands used for the covariate data processing and preparation steps.</li>
<li>This includes PEER factor analysis steps and merging the PEER factors to covariates in raw data and PCA.</li>
</ul>
<h3 id="association-scan-using-tensorqtl-and-summary-statistics-standardization">
  Association scan using TensorQTL and summary statistics standardization
  <a class="anchor" href="#association-scan-using-tensorqtl-and-summary-statistics-standardization">#</a>
</h3>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Zhang_BU/ROSMAP_DLPFC/sQTL/association_scan_sQTL.ipynb">association_scan_sQTL.ipynb</a> provides information about the TensorQTL cis association scan.</li>
</ul>
<h3 id="susie-univariate-fine-mapping">
  SuSiE univariate fine mapping
  <a class="anchor" href="#susie-univariate-fine-mapping">#</a>
</h3>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Zhang_BU/ROSMAP_DLPFC/sQTL/fine_mapping.ipynb">fine_mapping.ipynb</a> contains the workflow of sQTL univariate fine mapping using filterd psichomics data.</li>
</ul>
<h2 id="dataset-details">
  Dataset Details
  <a class="anchor" href="#dataset-details">#</a>
</h2>
<ul>
<li>Number of cases :</li>
<li>Number of controls :</li>
<li>Annotation information :</li>
<li>Genotype level imputation :</li>
<li>Imputation Panel :</li>
</ul>
<h3 id="paths-to-genotype-matrix">
  Path(s) to genotype matrix
  <a class="anchor" href="#paths-to-genotype-matrix">#</a>
</h3>
<p><strong>output of genotype_qc.ipynb</strong></p>
<ul>
<li>BU cluster, <code>/restricted/projectnb/casa/skandoi/ROSMAP_DLPFC/genotype_partition/ROSMAP_NIA_WGS.leftnorm.filtered.filtered.plink_files_list.txt</code></li>
<li>Wang Lab, <code>/mnt/mfs/hgrcgrid/homes/zq2209/ROSMAP_NIA_WGS.leftnorm.filtered.filtered.plink_files_list.txt</code></li>
</ul>
<pre tabindex="0"><code>$ cd /mnt/vast/hpc/csg/molecular_phenotype_calling/genotype_arch/
$ ls -lh *.{bim,bed,fam}
-rw-r--r-- 1 hs3163 hs3163 220M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.10.bed
-rw-r--r-- 1 hs3163 hs3163  33M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.10.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.10.fam
-rw-r--r-- 1 hs3163 hs3163 207M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.11.bed
-rw-r--r-- 1 hs3163 hs3163  31M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.11.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.11.fam
-rw-r--r-- 1 hs3163 hs3163 207M Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.12.bed
-rw-r--r-- 1 hs3163 hs3163  31M Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.12.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.12.fam
-rw-r--r-- 1 hs3163 hs3163 156M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.13.bed
-rw-r--r-- 1 hs3163 hs3163  23M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.13.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.13.fam
-rw-r--r-- 1 hs3163 hs3163 141M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.14.bed
-rw-r--r-- 1 hs3163 hs3163  21M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.14.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.14.fam
-rw-r--r-- 1 hs3163 hs3163 123M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.15.bed
-rw-r--r-- 1 hs3163 hs3163  18M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.15.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.15.fam
-rw-r--r-- 1 hs3163 hs3163 137M Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.16.bed
-rw-r--r-- 1 hs3163 hs3163  21M Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.16.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.16.fam
-rw-r--r-- 1 hs3163 hs3163 125M Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.17.bed
-rw-r--r-- 1 hs3163 hs3163  19M Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.17.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.17.fam
-rw-r--r-- 1 hs3163 hs3163 121M Sep 29 15:11 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.18.bed
-rw-r--r-- 1 hs3163 hs3163  18M Sep 29 15:11 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.18.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:11 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.18.fam
-rw-r--r-- 1 hs3163 hs3163 106M Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.19.bed
-rw-r--r-- 1 hs3163 hs3163  16M Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.19.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:10 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.19.fam
-rw-r--r-- 1 hs3163 hs3163 345M Sep 29 15:08 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.1.bed
-rw-r--r-- 1 hs3163 hs3163  50M Sep 29 15:08 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.1.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:08 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.1.fam
-rw-r--r-- 1 hs3163 hs3163 101M Sep 29 15:11 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.20.bed
-rw-r--r-- 1 hs3163 hs3163  15M Sep 29 15:11 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.20.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:11 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.20.fam
-rw-r--r-- 1 hs3163 hs3163  60M Sep 29 15:11 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.21.bed
-rw-r--r-- 1 hs3163 hs3163 9.1M Sep 29 15:11 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.21.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:11 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.21.fam
-rw-r--r-- 1 hs3163 hs3163  66M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.22.bed
-rw-r--r-- 1 hs3163 hs3163  11M Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.22.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:09 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.22.fam
-rw-r--r-- 1 hs3163 hs3163 359M Sep 29 15:05 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.2.bed
-rw-r--r-- 1 hs3163 hs3163  52M Sep 29 15:05 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.2.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:05 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.2.fam
-rw-r--r-- 1 hs3163 hs3163 302M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.3.bed
-rw-r--r-- 1 hs3163 hs3163  42M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.3.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.3.fam
-rw-r--r-- 1 hs3163 hs3163 302M Sep 29 15:08 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.4.bed
-rw-r--r-- 1 hs3163 hs3163  43M Sep 29 15:08 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.4.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:08 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.4.fam
-rw-r--r-- 1 hs3163 hs3163 274M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.5.bed
-rw-r--r-- 1 hs3163 hs3163  39M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.5.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.5.fam
-rw-r--r-- 1 hs3163 hs3163 280M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.6.bed
-rw-r--r-- 1 hs3163 hs3163  40M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.6.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.6.fam
-rw-r--r-- 1 hs3163 hs3163 255M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.7.bed
-rw-r--r-- 1 hs3163 hs3163  37M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.7.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.7.fam
-rw-r--r-- 1 hs3163 hs3163 232M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.8.bed
-rw-r--r-- 1 hs3163 hs3163  33M Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.8.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:07 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.8.fam
-rw-r--r-- 1 hs3163 hs3163 190M Sep 29 15:08 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.9.bed
-rw-r--r-- 1 hs3163 hs3163  27M Sep 29 15:08 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.9.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 29 15:08 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.9.fam
-rw-r--r-- 1 hs3163 hs3163 4.2G Sep 28 20:02 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.bed
-rw-r--r-- 1 hs3163 hs3163 619M Sep 28 20:02 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.bim
-rw-r--r-- 1 hs3163 hs3163  23K Sep 28 20:02 ROSMAP_NIA_WGS.leftnorm.filtered.filtered.fam
</code></pre><h3 id="paths-to-omics-data-matrix">
  Path(s) to omics-data matrix
  <a class="anchor" href="#paths-to-omics-data-matrix">#</a>
</h3>
<p><strong>output of phenotype_preprocessing_sqtl.ipynb</strong>
leafCutter:</p>
<ul>
<li>FTP server: <code>/ftp_fgc_xqtl/projects/rna-seq/BU/ROSMAP_DLPFC/sQTL/phenotype_preprocessing/</code>
psichomics:</li>
<li>Wang Lab, <code>/mnt/vast/hpc/csg/molecular_phenotype_calling/psichomics_sQTL/pheno</code></li>
</ul>
<pre tabindex="0"><code>$ ls -lh *.{txt,bed}
-rw-r--r-- 1 xc2610 xc2610 2.7K Apr  3 14:41 ROSMAP_psichomics_bed_recipe.txt
-rw-r--r-- 1 xc2610 xc2610  65M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr10.bed
-rw-r--r-- 1 xc2610 xc2610 113M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr11.bed
-rw-r--r-- 1 xc2610 xc2610 102M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr12.bed
-rw-r--r-- 1 xc2610 xc2610  28M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr13.bed
-rw-r--r-- 1 xc2610 xc2610  69M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr14.bed
-rw-r--r-- 1 xc2610 xc2610  58M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr15.bed
-rw-r--r-- 1 xc2610 xc2610  88M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr16.bed
-rw-r--r-- 1 xc2610 xc2610 125M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr17.bed
-rw-r--r-- 1 xc2610 xc2610  26M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr18.bed
-rw-r--r-- 1 xc2610 xc2610 117M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr19.bed
-rw-r--r-- 1 xc2610 xc2610 166M Apr  5 14:34 ROSMAP.psichomics.qqnorm.revised.formated_chr1.bed
-rw-r--r-- 1 xc2610 xc2610  52M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr20.bed
-rw-r--r-- 1 xc2610 xc2610  18M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr21.bed
-rw-r--r-- 1 xc2610 xc2610  50M Apr 14 17:01 ROSMAP.psichomics.qqnorm.revised.formated_chr22.bed
-rw-r--r-- 1 xc2610 xc2610 122M Apr  5 16:55 ROSMAP.psichomics.qqnorm.revised.formated_chr2.bed
-rw-r--r-- 1 xc2610 xc2610 108M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr3.bed
-rw-r--r-- 1 xc2610 xc2610  60M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr4.bed
-rw-r--r-- 1 xc2610 xc2610  75M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr5.bed
-rw-r--r-- 1 xc2610 xc2610  80M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr6.bed
-rw-r--r-- 1 xc2610 xc2610  84M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr7.bed
-rw-r--r-- 1 xc2610 xc2610  62M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr8.bed
-rw-r--r-- 1 xc2610 xc2610  66M Apr 14 17:00 ROSMAP.psichomics.qqnorm.revised.formated_chr9.bed
-rw-r--r-- 1 xc2610 xc2610 426K Feb 13 01:15 ROSMAP.psichomics.qqnorm.revised.formated_chrMT.bed
-rw-r--r-- 1 xc2610 xc2610  47M Feb 13 01:15 ROSMAP.psichomics.qqnorm.revised.formated_chrX.bed
-rw-r--r-- 1 xc2610 xc2610 529K Feb 13 01:15 ROSMAP.psichomics.qqnorm.revised.formated_chrY.bed
</code></pre><h3 id="paths-to-covariate-data-matrix">
  Path(s) to covariate data matrix
  <a class="anchor" href="#paths-to-covariate-data-matrix">#</a>
</h3>
<p><strong>output of covariate_preprocessing.ipynb</strong>
leafCutter:</p>
<ul>
<li>FTP server: <code>/ftp_fgc_xqtl/projects/rna-seq/BU/ROSMAP_DLPFC/sQTL/covariate_preprocessing/leafcutter.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.gz</code>
psichomics:</li>
<li>Wang Lab, <code>/mnt/vast/hpc/csg/molecular_phenotype_calling/psichomics_sQTL/cov</code></li>
</ul>
<pre tabindex="0"><code>$ ls -lh *.gz
-rw-r--r-- 1 xc2610 xc2610 546K Feb  9 13:06 psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.gz
</code></pre><h3 id="paths-to-qtl-results">
  Path(s) to QTL results
  <a class="anchor" href="#paths-to-qtl-results">#</a>
</h3>
<p><strong>output of association_scan_sQTL.ipynb</strong>
leafCutter:</p>
<ul>
<li>FTP server: <code>/ftp_fgc_xqtl/projects/rna-seq/BU/ROSMAP_DLPFC/sQTL/association_scan</code>
psichomics:</li>
<li>Wang lab: <code>/mnt/vast/hpc/csg/molecular_phenotype_calling/psichomics_sQTL/output</code></li>
</ul>
<pre tabindex="0"><code>$ls -lh *.txt
-rw-r--r-- 1 xc2610 xc2610 928K Feb 13 02:34 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.10.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 7.4G Feb 13 01:45 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.10.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.6M Feb 13 03:08 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.11.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610  12G Feb 13 01:51 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.11.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.5M Feb 13 03:26 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.12.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610  11G Feb 13 01:53 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.12.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 389K Feb 13 02:57 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.13.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 3.2G Feb 13 02:36 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.13.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 973K Feb 13 03:35 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.14.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 7.9G Feb 13 02:44 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.14.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 813K Feb 13 03:24 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.15.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 5.7G Feb 13 02:46 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.15.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.2M Feb 13 03:53 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.16.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 9.8G Feb 13 02:50 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.16.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.8M Feb 13 04:39 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.17.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610  14G Feb 13 03:01 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.17.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 353K Feb 13 03:18 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.18.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 2.8G Feb 13 03:01 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.18.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.6M Feb 13 05:01 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.19.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610  16G Feb 13 03:25 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.19.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 2.3M Feb 13 04:16 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.1.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610  17G Feb 13 02:03 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.1.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 729K Feb 13 03:36 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.20.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 5.7G Feb 13 03:00 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.20.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 254K Feb 13 03:25 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.21.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 2.1G Feb 13 03:12 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.21.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 712K Feb 13 03:58 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.22.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 5.9G Feb 13 03:19 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.22.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.7M Feb 13 03:16 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.2.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610  13G Feb 13 01:53 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.2.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.5M Feb 13 03:11 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.3.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610  11G Feb 13 01:51 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.3.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 839K Feb 13 02:30 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.4.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 6.7G Feb 13 01:44 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.4.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.1M Feb 13 02:37 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.5.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 7.9G Feb 13 01:45 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.5.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.1M Feb 13 03:02 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.6.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610  12G Feb 13 01:51 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.6.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 1.2M Feb 13 02:51 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.7.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 9.5G Feb 13 01:49 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.7.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 871K Feb 13 02:31 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.8.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 6.8G Feb 13 01:45 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.8.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 939K Feb 13 02:39 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.9.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 7.5G Feb 13 01:46 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.9.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610  27M Feb 13 05:02 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.emprical.cis_sumstats.txt
</code></pre><p>psichomics (grouped by each gene - event_type pair):</p>
<ul>
<li>Wang lab: <code>/mnt/vast/hpc/csg/molecular_phenotype_calling/psichomics_sQTL/event_sep/output</code></li>
</ul>
<pre tabindex="0"><code>$ ls -lh *.txt
-rw-r--r-- 1 xc2610 xc2610 229K Apr  3 15:30 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.10.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 1.8G Apr  3 14:49 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.10.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 414K Apr  3 15:53 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.11.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 2.9G Apr  3 14:52 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.11.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 373K Apr  3 16:12 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.12.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 2.7G Apr  3 15:10 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.12.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610  99K Apr  3 15:46 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.13.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 785M Apr  3 15:29 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.13.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 249K Apr  3 16:16 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.14.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 2.0G Apr  3 15:33 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.14.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 213K Apr  3 16:18 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.15.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 1.5G Apr  3 15:47 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.15.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 332K Apr  3 16:30 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.16.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 2.6G Apr  3 15:38 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.16.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 453K Apr  3 17:19 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.17.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 3.4G Apr  3 16:01 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.17.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610  91K Apr  3 16:04 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.18.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 699M Apr  3 15:49 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.18.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 471K Apr  3 17:18 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.19.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 4.3G Apr  3 15:55 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.19.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 617K Apr  3 16:33 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.1.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 4.3G Apr  3 14:55 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.1.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 182K Apr  3 16:05 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.20.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 1.4G Apr  3 15:34 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.20.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610  73K Apr  3 16:06 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.21.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 577M Apr  3 15:55 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.21.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 194K Apr  3 16:30 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.22.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 1.6G Apr  3 15:58 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.22.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 434K Apr  3 16:01 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.2.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 3.1G Apr  3 14:52 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.2.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 394K Apr  3 15:53 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.3.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 2.7G Apr  3 14:51 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.3.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 220K Apr  3 15:25 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.4.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 1.7G Apr  3 14:48 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.4.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 286K Apr  3 15:42 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.5.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 2.1G Apr  3 14:50 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.5.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 319K Apr  3 15:52 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.6.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 3.4G Apr  3 14:51 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.6.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 331K Apr  3 15:43 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.7.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 2.6G Apr  3 14:50 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.7.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 234K Apr  3 15:26 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.8.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 1.8G Apr  3 14:48 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.8.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 243K Apr  3 15:29 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.9.emprical.cis_sumstats.txt
-rw-r--r-- 1 xc2610 xc2610 1.9G Apr  3 14:49 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.9.norminal.cis_long_table.txt
-rw-r--r-- 1 xc2610 xc2610 7.4M Apr  3 17:21 ROSMAP_psichomics_bed_recipe_psichomics.dlpfc_batch_all.ROSMAP_covariates.ROSMAP_NIA_WGS.pca.PEER.txt.emprical.cis_sumstats.txt
</code></pre><h3 id="paths-to-fine-mapping-with-susie-model">
  Path(s) to fine-mapping with SuSiE model
  <a class="anchor" href="#paths-to-fine-mapping-with-susie-model">#</a>
</h3>
<p><strong>output of fine_mapping.ipynb</strong></p>
<ul>
<li>Wang lab: <code>/mnt/vast/hpc/csg/molecular_phenotype_calling/sQTL_finemapping/ROSMAP_psichomics</code>
The results are seperated in psi_A3SS, psi_A5SS, psi_AFE, psi_ALE, psi_MXE, psi_SE sub directories according to their event type. Due to the massive amount of files for splicing data they are not listed here.</li>
</ul>
<h3 id="paths-to-fine-mapping-with-susie-rss-model">
  Path(s) to fine-mapping with SuSiE RSS model
  <a class="anchor" href="#paths-to-fine-mapping-with-susie-rss-model">#</a>
</h3>
</article>
 
      

      <footer class="book-footer">
        
  <div class="flex flex-wrap justify-between">





</div>



  <script>(function(){function a(c){const a=window.getSelection(),b=document.createRange();b.selectNodeContents(c),a.removeAllRanges(),a.addRange(b)}document.querySelectorAll("pre code").forEach(b=>{b.addEventListener("click",function(c){if(window.getSelection().toString())return;a(b.parentElement),navigator.clipboard&&navigator.clipboard.writeText(b.parentElement.textContent)})})})()</script>


 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      <div class="book-toc-content">
        
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#analysis-status">Analysis Status</a></li>
    <li><a href="#analysis-details">Analysis Details</a>
      <ul>
        <li><a href="#sample-size-tracking-through-the-analysis-workflow">Sample size tracking through the analysis workflow</a></li>
        <li><a href="#qc-and-normalization-details-for-phenotype-data">QC and Normalization Details for Phenotype Data</a></li>
        <li><a href="#genotype-parameters">Genotype parameters</a></li>
        <li><a href="#covariate-parameters">Covariate parameters</a></li>
        <li><a href="#tensorqtl-association-scan-parameters">TensorQTL association scan parameters</a></li>
      </ul>
    </li>
    <li><a href="#links-to-qtl-analysis-notebooks">Links to QTL analysis notebooks</a>
      <ul>
        <li><a href="#association-data-preprocessing">Association data preprocessing</a></li>
        <li><a href="#association-scan-using-tensorqtl-and-summary-statistics-standardization">Association scan using TensorQTL and summary statistics standardization</a></li>
        <li><a href="#susie-univariate-fine-mapping">SuSiE univariate fine mapping</a></li>
      </ul>
    </li>
    <li><a href="#dataset-details">Dataset Details</a>
      <ul>
        <li><a href="#paths-to-genotype-matrix">Path(s) to genotype matrix</a></li>
        <li><a href="#paths-to-omics-data-matrix">Path(s) to omics-data matrix</a></li>
        <li><a href="#paths-to-covariate-data-matrix">Path(s) to covariate data matrix</a></li>
        <li><a href="#paths-to-qtl-results">Path(s) to QTL results</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-model">Path(s) to fine-mapping with SuSiE model</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-rss-model">Path(s) to fine-mapping with SuSiE RSS model</a></li>
      </ul>
    </li>
  </ul>
</nav>


 
      </div>
    </aside>
    
  </main>

  
</body>
</html>












