<!DOCTYPE html>
<html lang="en-us" dir="ltr">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="MAGENTA Non-Hispanic White Blood Gene Expression QTL #  Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P. Hussman Institute for Human Genomics (HIHG) at the University of Miami Miller School of Medicine (Miami, FL), North Carolina A&amp;T State University (Greensboro, NC), and Case Western Reserve University (Cleveland, OH).">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ffffff">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#343a40">
<meta name="color-scheme" content="light dark"><meta property="og:title" content="MAGENTA Non-Hispanic White Blood Gene Expression QTL" />
<meta property="og:description" content="MAGENTA Non-Hispanic White Blood Gene Expression QTL #  Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P. Hussman Institute for Human Genomics (HIHG) at the University of Miami Miller School of Medicine (Miami, FL), North Carolina A&amp;T State University (Greensboro, NC), and Case Western Reserve University (Cleveland, OH)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://cumc.github.io/xqtl-data/m/magenta-non-hispanic-white-blood-gene-expression-qtl/" /><meta property="article:section" content="m" />



<title>MAGENTA Non-Hispanic White Blood Gene Expression QTL | FunGen-xQTL Data Repository</title>
<link rel="manifest" href="/xqtl-data/manifest.json">
<link rel="icon" href="/xqtl-data/favicon.png" >
<link rel="canonical" href="https://cumc.github.io/xqtl-data/m/magenta-non-hispanic-white-blood-gene-expression-qtl/">
<link rel="stylesheet" href="/xqtl-data/book.min.33a48f5432973b8ff9a82679d9e45d67f2c15d4399bd2829269455cfe390b5e8.css" integrity="sha256-M6SPVDKXO4/5qCZ52eRdZ/LBXUOZvSgpJpRVz&#43;OQteg=" crossorigin="anonymous">
  <script defer src="/xqtl-data/flexsearch.min.js"></script>
  <script defer src="/xqtl-data/en.search.min.7d8c7087392e9b20178e66069a2711358324dd1c2231aa1919452888d5fa6faa.js" integrity="sha256-fYxwhzkumyAXjmYGmicRNYMk3RwiMaoZGUUoiNX6b6o=" crossorigin="anonymous"></script>

  <script defer src="/xqtl-data/sw.min.5a8d9ccf5ea73dcbbb1f34b183236a28dd36cc6d6540bb5b46431b4e00517252.js" integrity="sha256-Wo2cz16nPcu7HzSxgyNqKN02zG1lQLtbRkMbTgBRclI=" crossorigin="anonymous"></script>
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->
  
</head>
<body dir="ltr">
  <input type="checkbox" class="hidden toggle" id="menu-control" />
  <input type="checkbox" class="hidden toggle" id="toc-control" />
  <main class="container flex">
    <aside class="book-menu">
      <div class="book-menu-content">
        
  <nav>
<h2 class="book-brand">
  <a class="flex align-center" href="/xqtl-data/"><span>FunGen-xQTL Data Repository</span>
  </a>
</h2>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>












  <ul>
<li>Alphabetic List of Data Available
<ul>
<li>
  <a href="/xqtl-data/a/">A</a></li>
<li>
  <a href="/xqtl-data/d/">D</a></li>
<li>
  <a href="/xqtl-data/e/">E</a></li>
<li>
  <a href="/xqtl-data/f/">F</a></li>
<li>
  <a href="/xqtl-data/h/">H</a></li>
<li>
  <a href="/xqtl-data/k/">K</a></li>
<li>
  <a href="/xqtl-data/m/">M</a></li>
<li>
  <a href="/xqtl-data/n/">N</a></li>
<li>
  <a href="/xqtl-data/p/">P</a></li>
<li>
  <a href="/xqtl-data/r/">R</a></li>
<li>
  <a href="/xqtl-data/s/">S</a></li>
<li>
  <a href="/xqtl-data/t/">T</a></li>
<li>
  <a href="/xqtl-data/w/">W</a></li>
</ul>
</li>
</ul>










</nav>




  <script>(function(){var a=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(b){localStorage.setItem("menu.scrollTop",a.scrollTop)}),a.scrollTop=localStorage.getItem("menu.scrollTop")})()</script>


 
      </div>
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/xqtl-data/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>MAGENTA Non-Hispanic White Blood Gene Expression QTL</strong>

  <label for="toc-control">
    
    <img src="/xqtl-data/svg/toc.svg" class="book-icon" alt="Table of Contents" />
    
  </label>
</div>


  
  <aside class="hidden clearfix">
    
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#analysis-status">Analysis Status</a></li>
    <li><a href="#dataset-details">Dataset Details</a>
      <ul>
        <li><a href="#paths-to-genotype-matrix">Path(s) to genotype matrix</a></li>
        <li><a href="#paths-to-omics-data-matrix">Path(s) to omics-data matrix</a></li>
        <li><a href="#paths-to-covariate-data-matrix">Path(s) to covariate data matrix</a></li>
        <li><a href="#paths-to-qtl-results">Path(s) to QTL results</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-model">Path(s) to fine-mapping with SuSiE model</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-rss-model">Path(s) to fine-mapping with SuSiE RSS model</a></li>
        <li><a href="#paths-to-colocalization-with-susie-coloc">Path(s) to colocalization with SuSiE-coloc</a></li>
      </ul>
    </li>
    <li><a href="#links-to-qtl-analysis-notebooks">Links to QTL analysis notebooks</a></li>
  </ul>
</nav>



  </aside>
  
 
      </header>

      
      
  <article class="markdown book-article"><h1 id="magenta-non-hispanic-white-blood-gene-expression-qtl">
  MAGENTA Non-Hispanic White Blood Gene Expression QTL
  <a class="anchor" href="#magenta-non-hispanic-white-blood-gene-expression-qtl">#</a>
</h1>
<p>Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P. Hussman Institute for Human Genomics (HIHG) at the University of Miami Miller School of Medicine (Miami, FL), North Carolina A&amp;T State University (Greensboro, NC), and Case Western Reserve University (Cleveland, OH).  Participants were ascertained as part of the ADSP Follow-up Study and included both cases (&gt;65 years of age of onset) and controls (&gt;65 years of age at age of exam).  All participants were adjudicated by a clinical panel with expertise in AD related disorders and classified as AD according to standard criteria developed by the National Institute of Aging and the Alzheimer’s Association.</p>
<h2 id="contact">
  Contact
  <a class="anchor" href="#contact">#</a>
</h2>
<p>Makaela Mews (analyst; 
  <a href="mailto:mxm1368@case.edu">mxm1368@case.edu</a>);  Dr. William S. Bush (
  <a href="mailto:wsb36@case.edu">wsb36@case.edu</a>);  Dr. Anthony J. Griswold (Griswold, Anthony J 
  <a href="mailto:agriswold@miami.edu">agriswold@miami.edu</a>)</p>
<h2 id="study-overview">
  Study Overview
  <a class="anchor" href="#study-overview">#</a>
</h2>
<ul>
<li>Sample information: <code>MAGENTA/MAGENTA_NHW_eqtl_sample_attributes.csv</code>.</li>
<li>Lab protocol: <code>MAGENTA/MAGENTA_NHW_eqtl_lab_protocol.csv</code>.</li>
<li>Computational protocol: <code>MAGENTA/MAGENTA_NHW_eqtl_computational_protocol.csv</code>.</li>
<li>QTL summary statistics output: <code>MAGENTA/MAGENTA_NHW_expression.qtl_results.csv</code>.</li>
<li>Fine-mapping results individual level data model: <code>####/####.susie.csv</code>.</li>
<li>Fine-mapping results summary statistics model: <code>####/####.susie_rss.csv</code>.</li>
<li>Colocalization with AD GWAS: <code>####/####.susie_coloc.csv</code>.
where &ldquo;Sample information&rdquo; is a CSV file containing sample attributes / complete set of covariates for the exact samples used in the QTL study;
&ldquo;Lab protocol&rdquo; is a CSV file containing the experimental protocol information to process the samples;
&ldquo;Computational protocol&rdquo; is a CSV file containing the computational protocol information to process the samples;
&ldquo;QTL summary statistics output&rdquo; is the list of QTL summary statistics results;
&ldquo;Fine-mapping results individual level data model&rdquo; is the list of fine-mapped QTL using individual level phenotype and genotype data input for SuSiE model. &ldquo;Fine-mapping results summary statistics model&rdquo; uses QTL summary statistics for SuSiE RSS model.
&ldquo;Colocalization with AD GWAS&rdquo; is the list of colocalization analysis with different AD GWAS publications and data-sets using coloc model on SuSiE results.</li>
</ul>
<h2 id="analysis-status">
  Analysis Status
  <a class="anchor" href="#analysis-status">#</a>
</h2>
<p>TransQTL association: Need to be updated with new covariate files.</p>
<h2 id="dataset-details">
  Dataset Details
  <a class="anchor" href="#dataset-details">#</a>
</h2>
<h3 id="paths-to-genotype-matrix">
  Path(s) to genotype matrix
  <a class="anchor" href="#paths-to-genotype-matrix">#</a>
</h3>
<p>Here please document the exact genotype matrix used for QTL calling.</p>
<ul>
<li>Path(s) to the data on your local cluster where you analyzed the data.</li>
<li>TensorQTL - cis:  Bush Lab, server:  rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/nhw_239/geno/by_chr/nhw_chr_1_22.no_rel.filtered.plink_files_list.txt</code></li>
<li>TensorQTL - trans:  Bush Lab, server:  rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/nhw_239/geno/geno_final/rerun/nhw_chr_1_22.no_rel.filtered.bed</code></li>
<li>NIAGADS FTP, <code>/ftp_fgc_xqtl/projects/...</code>
A summary of the genotype data including size used for TensorQTL - cis:</li>
</ul>
<pre tabindex="0"><code>$ ls -lh *{bed,bim,fam,txt}
-rw-r--r-- 1 mxm1368 wsb36  29M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.10.bed
-rw-r--r-- 1 mxm1368 wsb36  19M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.10.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.10.fam
-rw-r--r-- 1 mxm1368 wsb36  28M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.11.bed
-rw-r--r-- 1 mxm1368 wsb36  18M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.11.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.11.fam
-rw-r--r-- 1 mxm1368 wsb36  28M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.12.bed
-rw-r--r-- 1 mxm1368 wsb36  18M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.12.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.12.fam
-rw-r--r-- 1 mxm1368 wsb36  21M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.13.bed
-rw-r--r-- 1 mxm1368 wsb36  13M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.13.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.13.fam
-rw-r--r-- 1 mxm1368 wsb36  19M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.14.bed
-rw-r--r-- 1 mxm1368 wsb36  12M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.14.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.14.fam
-rw-r--r-- 1 mxm1368 wsb36  16M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.15.bed
-rw-r--r-- 1 mxm1368 wsb36  10M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.15.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.15.fam
-rw-r--r-- 1 mxm1368 wsb36  18M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.16.bed
-rw-r--r-- 1 mxm1368 wsb36  11M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.16.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.16.fam
-rw-r--r-- 1 mxm1368 wsb36  16M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.17.bed
-rw-r--r-- 1 mxm1368 wsb36 9.3M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.17.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.17.fam
-rw-r--r-- 1 mxm1368 wsb36  17M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.18.bed
-rw-r--r-- 1 mxm1368 wsb36  11M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.18.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.18.fam
-rw-r--r-- 1 mxm1368 wsb36  13M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.19.bed
-rw-r--r-- 1 mxm1368 wsb36 7.9M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.19.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.19.fam
-rw-r--r-- 1 mxm1368 wsb36  44M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.1.bed
-rw-r--r-- 1 mxm1368 wsb36  27M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.1.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.1.fam
-rw-r--r-- 1 mxm1368 wsb36  13M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.20.bed
-rw-r--r-- 1 mxm1368 wsb36 7.8M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.20.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.20.fam
-rw-r--r-- 1 mxm1368 wsb36 7.8M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.21.bed
-rw-r--r-- 1 mxm1368 wsb36 4.9M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.21.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.21.fam
-rw-r--r-- 1 mxm1368 wsb36 7.9M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.22.bed
-rw-r--r-- 1 mxm1368 wsb36 4.9M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.22.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.22.fam
-rw-r--r-- 1 mxm1368 wsb36  47M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.2.bed
-rw-r--r-- 1 mxm1368 wsb36  29M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.2.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.2.fam
-rw-r--r-- 1 mxm1368 wsb36  41M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.3.bed
-rw-r--r-- 1 mxm1368 wsb36  25M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.3.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.3.fam
-rw-r--r-- 1 mxm1368 wsb36  41M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.4.bed
-rw-r--r-- 1 mxm1368 wsb36  25M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.4.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.4.fam
-rw-r--r-- 1 mxm1368 wsb36  37M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.5.bed
-rw-r--r-- 1 mxm1368 wsb36  22M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.5.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.5.fam
-rw-r--r-- 1 mxm1368 wsb36  38M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.6.bed
-rw-r--r-- 1 mxm1368 wsb36  23M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.6.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.6.fam
-rw-r--r-- 1 mxm1368 wsb36  34M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.7.bed
-rw-r--r-- 1 mxm1368 wsb36  20M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.7.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.7.fam
-rw-r--r-- 1 mxm1368 wsb36  32M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.8.bed
-rw-r--r-- 1 mxm1368 wsb36  19M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.8.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.8.fam
-rw-r--r-- 1 mxm1368 wsb36  25M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.9.bed
-rw-r--r-- 1 mxm1368 wsb36  15M Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.9.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 23 14:26 nhw_chr_1_22.no_rel.filtered.9.fam
-rw-r--r-- 1 mxm1368 wsb36 2.3K Mar 23 14:27 nhw_chr_1_22.no_rel.filtered.plink_files_list.txt
</code></pre><p>A summary of the genotype data including size used for TensorQTL - trans:</p>
<pre tabindex="0"><code>$ ls -lh *.no_rel.filtered.{bim,bed,fam}
-rw-r--r-- 1 mxm1368 wsb36 563M Mar 22 13:30 nhw_chr_1_22.no_rel.filtered.bed
-rw-r--r-- 1 mxm1368 wsb36 344M Mar 22 13:30 nhw_chr_1_22.no_rel.filtered.bim
-rw-r--r-- 1 mxm1368 wsb36 5.9K Mar 22 13:30 nhw_chr_1_22.no_rel.filtered.fam
</code></pre><h3 id="paths-to-omics-data-matrix">
  Path(s) to omics-data matrix
  <a class="anchor" href="#paths-to-omics-data-matrix">#</a>
</h3>
<p>Here please document the exact omics-data matrix used for QTL calling.</p>
<ul>
<li>Path(s) to the data on your local cluster where you analyzed the data.</li>
<li>TensorQTL - cis:  Bush Lab, server: rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/nhw_239/mol_phe/tmm_per_chr</code></li>
<li>TensorQTL - trans:  Bush Lab, server: rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/nhw_239/mol_phe</code></li>
<li>NIAGADS FTP, <code>/ftp_fgc_xqtl/projects/...</code>
A summary of the phenotype data including size used for TensorQTL - cis:</li>
</ul>
<pre tabindex="0"><code>$ pwd
/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/nhw_239/mol_phe/tmm_per_chr
$ ls -lh *.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  489K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr10.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  718K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr11.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  722K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr12.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  224K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr13.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  508K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr14.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  436K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr15.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  692K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr16.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  870K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr17.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  196K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr18.bed.gz
-rw-r--r-- 1 mxm1368 wsb36 1003K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr19.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  1.4M Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr1.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  347K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr20.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  143K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr21.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  388K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr22.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  873K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr2.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  700K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr3.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  442K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr4.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  576K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr5.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  680K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr6.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  688K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr7.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  436K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr8.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  506K Mar  2 19:06 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chr9.bed.gz
-rw-r--r-- 1 mxm1368 wsb36  417K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chrX.bed.gz
-rw-r--r-- 1 mxm1368 wsb36   14K Mar  2 19:05 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.chrY.bed.gz
</code></pre><p>A summary of the phenotype data including size used for TensorQTL - trans:</p>
<pre tabindex="0"><code>$ pwd
/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/nhw_239/mol_phe
$ ls -lh *tmm.expression.bed.gz
-rw-r--r-- 1 mxm1368 wsb36 14M Feb 27 18:02 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.bed.gz
</code></pre><h3 id="paths-to-covariate-data-matrix">
  Path(s) to covariate data matrix
  <a class="anchor" href="#paths-to-covariate-data-matrix">#</a>
</h3>
<p>Here please document the exact covariate matrix used for QTL calling, <strong>including selected covariates from sample attribute file, genotype PC and phenotype hidden confounding factors computed in your previous analysis</strong></p>
<ul>
<li>Path(s) to the data on your local cluster where you analyzed the data.</li>
<li>Bush Lab, server:  rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/nhw_239/cov</code></li>
<li>NIAGADS FTP, <code>/ftp_fgc_xqtl/projects/...</code>
A summary of the covariate data including size used for TensorQTL - cis &amp; trans:</li>
</ul>
<pre tabindex="0"><code>$ ls -lh *tmm.expression.cov_pca.resid.PEER.cov.gz
-rw-r--r-- 1 mxm1368 wsb36 91K Mar  2 19:01 nhw_no_rel.rnaseqc.low_expression_filtered.outlier_removed.tmm.expression.cov_pca.resid.PEER.cov.gz
</code></pre><h3 id="paths-to-qtl-results">
  Path(s) to QTL results
  <a class="anchor" href="#paths-to-qtl-results">#</a>
</h3>
<ul>
<li>TensorQTL - cis: Bush Lab, server:  rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/nhw_239/tensorqtl/cis/plink_mac5</code></li>
<li>TensorQTL - trans: Bush Lab, server:  rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/nhw_239/tensorqtl/trans/plink_mac5</code>
A summary of the TensorQTL - cis data including size:</li>
</ul>
<pre tabindex="0"><code>$ ls -lh
total 30G
-rw-r--r-- 1 mxm1368 wsb36 144M Mar 23 14:48 nhw_no_rel.10.cis_qtl_pairs.10.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 15:40 nhw_no_rel.10.cis_qtl_pairs.10.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  75K Mar 23 15:40 nhw_no_rel.10.cis_qtl_pairs.10.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 169K Mar 23 15:40 nhw_no_rel.10.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 801M Mar 23 14:51 nhw_no_rel.10.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 193M Mar 23 15:59 nhw_no_rel.11.cis_qtl_pairs.11.parquet
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 23 17:08 nhw_no_rel.11.cis_qtl_pairs.11.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36 113K Mar 23 17:08 nhw_no_rel.11.cis_qtl_pairs.11.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 246K Mar 23 17:08 nhw_no_rel.11.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.1G Mar 23 16:05 nhw_no_rel.11.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 201M Mar 23 16:58 nhw_no_rel.12.cis_qtl_pairs.12.parquet
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 23 17:25 nhw_no_rel.12.cis_qtl_pairs.12.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36 112K Mar 23 17:25 nhw_no_rel.12.cis_qtl_pairs.12.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 248K Mar 23 17:25 nhw_no_rel.12.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.2G Mar 23 17:02 nhw_no_rel.12.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36  69M Mar 23 16:01 nhw_no_rel.13.cis_qtl_pairs.13.parquet
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 23 16:24 nhw_no_rel.13.cis_qtl_pairs.13.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  38K Mar 23 16:24 nhw_no_rel.13.cis_qtl_pairs.13.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36  77K Mar 23 16:24 nhw_no_rel.13.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 378M Mar 23 16:02 nhw_no_rel.13.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 147M Mar 23 16:01 nhw_no_rel.14.cis_qtl_pairs.14.parquet
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 23 16:55 nhw_no_rel.14.cis_qtl_pairs.14.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  82K Mar 23 16:55 nhw_no_rel.14.cis_qtl_pairs.14.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 175K Mar 23 16:55 nhw_no_rel.14.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 845M Mar 23 16:05 nhw_no_rel.14.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 121M Mar 23 14:46 nhw_no_rel.15.cis_qtl_pairs.15.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 15:30 nhw_no_rel.15.cis_qtl_pairs.15.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  67K Mar 23 15:30 nhw_no_rel.15.cis_qtl_pairs.15.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 151K Mar 23 15:30 nhw_no_rel.15.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 654M Mar 23 14:49 nhw_no_rel.15.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 191M Mar 23 15:49 nhw_no_rel.16.cis_qtl_pairs.16.parquet
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 23 16:57 nhw_no_rel.16.cis_qtl_pairs.16.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36 100K Mar 23 16:57 nhw_no_rel.16.cis_qtl_pairs.16.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 238K Mar 23 16:57 nhw_no_rel.16.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.1G Mar 23 15:54 nhw_no_rel.16.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 232M Mar 23 16:12 nhw_no_rel.17.cis_qtl_pairs.17.parquet
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 23 17:20 nhw_no_rel.17.cis_qtl_pairs.17.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36 125K Mar 23 17:20 nhw_no_rel.17.cis_qtl_pairs.17.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 300K Mar 23 17:20 nhw_no_rel.17.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.3G Mar 23 16:19 nhw_no_rel.17.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36  63M Mar 23 14:41 nhw_no_rel.18.cis_qtl_pairs.18.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 14:59 nhw_no_rel.18.cis_qtl_pairs.18.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  31K Mar 23 14:59 nhw_no_rel.18.cis_qtl_pairs.18.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36  68K Mar 23 14:59 nhw_no_rel.18.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 339M Mar 23 14:42 nhw_no_rel.18.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 325M Mar 23 15:00 nhw_no_rel.19.cis_qtl_pairs.19.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 16:53 nhw_no_rel.19.cis_qtl_pairs.19.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36 144K Mar 23 16:53 nhw_no_rel.19.cis_qtl_pairs.19.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 344K Mar 23 16:53 nhw_no_rel.19.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.8G Mar 23 15:08 nhw_no_rel.19.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 345M Mar 23 15:07 nhw_no_rel.1.cis_qtl_pairs.1.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 17:12 nhw_no_rel.1.cis_qtl_pairs.1.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36 189K Mar 23 17:12 nhw_no_rel.1.cis_qtl_pairs.1.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 458K Mar 23 17:12 nhw_no_rel.1.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.9G Mar 23 15:17 nhw_no_rel.1.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 102M Mar 23 15:09 nhw_no_rel.20.cis_qtl_pairs.20.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 15:47 nhw_no_rel.20.cis_qtl_pairs.20.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  54K Mar 23 15:47 nhw_no_rel.20.cis_qtl_pairs.20.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 119K Mar 23 15:47 nhw_no_rel.20.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 555M Mar 23 15:11 nhw_no_rel.20.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36  48M Mar 23 15:40 nhw_no_rel.21.cis_qtl_pairs.21.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 15:54 nhw_no_rel.21.cis_qtl_pairs.21.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  25K Mar 23 15:54 nhw_no_rel.21.cis_qtl_pairs.21.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36  48K Mar 23 15:54 nhw_no_rel.21.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 241M Mar 23 15:41 nhw_no_rel.21.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 112M Mar 23 16:35 nhw_no_rel.22.cis_qtl_pairs.22.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 17:10 nhw_no_rel.22.cis_qtl_pairs.22.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  50K Mar 23 17:10 nhw_no_rel.22.cis_qtl_pairs.22.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 134K Mar 23 17:10 nhw_no_rel.22.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 647M Mar 23 16:38 nhw_no_rel.22.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 234M Mar 23 16:28 nhw_no_rel.2.cis_qtl_pairs.2.parquet
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 23 17:24 nhw_no_rel.2.cis_qtl_pairs.2.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36 124K Mar 23 17:24 nhw_no_rel.2.cis_qtl_pairs.2.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 300K Mar 23 17:24 nhw_no_rel.2.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.3G Mar 23 16:34 nhw_no_rel.2.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 188M Mar 23 16:40 nhw_no_rel.3.cis_qtl_pairs.3.parquet
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 23 17:23 nhw_no_rel.3.cis_qtl_pairs.3.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36 102K Mar 23 17:23 nhw_no_rel.3.cis_qtl_pairs.3.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 240K Mar 23 17:23 nhw_no_rel.3.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.1G Mar 23 16:45 nhw_no_rel.3.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 133M Mar 23 14:47 nhw_no_rel.4.cis_qtl_pairs.4.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 15:35 nhw_no_rel.4.cis_qtl_pairs.4.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  69K Mar 23 15:35 nhw_no_rel.4.cis_qtl_pairs.4.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 153K Mar 23 15:35 nhw_no_rel.4.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 752M Mar 23 14:49 nhw_no_rel.4.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 163M Mar 23 14:49 nhw_no_rel.5.cis_qtl_pairs.5.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 15:50 nhw_no_rel.5.cis_qtl_pairs.5.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  81K Mar 23 15:50 nhw_no_rel.5.cis_qtl_pairs.5.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 198K Mar 23 15:50 nhw_no_rel.5.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 886M Mar 23 14:53 nhw_no_rel.5.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 248M Mar 23 14:53 nhw_no_rel.6.cis_qtl_pairs.6.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 16:21 nhw_no_rel.6.cis_qtl_pairs.6.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  85K Mar 23 16:21 nhw_no_rel.6.cis_qtl_pairs.6.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 235K Mar 23 16:21 nhw_no_rel.6.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.5G Mar 23 15:00 nhw_no_rel.6.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 196M Mar 23 14:52 nhw_no_rel.7.cis_qtl_pairs.7.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 16:04 nhw_no_rel.7.cis_qtl_pairs.7.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  98K Mar 23 16:04 nhw_no_rel.7.cis_qtl_pairs.7.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 237K Mar 23 16:04 nhw_no_rel.7.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 1.1G Mar 23 14:57 nhw_no_rel.7.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 132M Mar 23 14:47 nhw_no_rel.8.cis_qtl_pairs.8.parquet
-rw-r--r-- 1 mxm1368 wsb36  285 Mar 23 15:33 nhw_no_rel.8.cis_qtl_pairs.8.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  66K Mar 23 15:33 nhw_no_rel.8.cis_qtl_pairs.8.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 150K Mar 23 15:33 nhw_no_rel.8.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 712M Mar 23 14:49 nhw_no_rel.8.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36 150M Mar 23 15:47 nhw_no_rel.9.cis_qtl_pairs.9.parquet
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 23 16:39 nhw_no_rel.9.cis_qtl_pairs.9.parquet.stderr
-rw-r--r-- 1 mxm1368 wsb36  73K Mar 23 16:39 nhw_no_rel.9.cis_qtl_pairs.9.parquet.stdout
-rw-r--r-- 1 mxm1368 wsb36 174K Mar 23 16:39 nhw_no_rel.9.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36 793M Mar 23 15:51 nhw_no_rel.9.norminal.cis_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36  106 Mar 23 17:26 nhw_no_rel.emprical.cis_sumstats.summary
-rw-r--r-- 1 mxm1368 wsb36 5.9M Mar 23 17:26 nhw_no_rel.emprical.cis_sumstats.txt
-rw-r--r-- 1 mxm1368 wsb36  311 Mar 23 17:25 TensorQTL.cis._column_info.txt
-rw-r--r-- 1 mxm1368 wsb36 5.0K Mar 23 17:25 TensorQTL.cis._recipe.tsv
-rw-r--r-- 1 mxm1368 wsb36  18K Mar 23 17:26 TensorQTL.cis._recipe.tsv.stderr
-rw-r--r-- 1 mxm1368 wsb36   46 Mar 23 17:25 TensorQTL.cis._recipe.tsv.stdout
</code></pre><p>A summary of the TensorQTL - trans data including size:</p>
<pre tabindex="0"><code>$ ls -lh
total 6.5M
-rw-r--r-- 1 mxm1368 wsb36 4.9M Mar 24 09:42 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.norminal.trans_long_table.txt
-rw-r--r-- 1 mxm1368 wsb36  360 Mar 24 09:42 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.norminal.trans_long_table.txt.stderr
-rw-r--r-- 1 mxm1368 wsb36  56K Mar 24 09:42 nhw_gene_tpm_prep_v2.low_expression_filtered.outlier_removed.tmm.expression.norminal.trans_long_table.txt.stdout
-rw-r--r-- 1 mxm1368 wsb36  189 Mar 24 09:42 TensorQTL.trans._column_info.txt
-rw-r--r-- 1 mxm1368 wsb36  323 Mar 24 09:42 TensorQTL.trans._recipe.tsv
</code></pre><h3 id="paths-to-fine-mapping-with-susie-model">
  Path(s) to fine-mapping with SuSiE model
  <a class="anchor" href="#paths-to-fine-mapping-with-susie-model">#</a>
</h3>
<h3 id="paths-to-fine-mapping-with-susie-rss-model">
  Path(s) to fine-mapping with SuSiE RSS model
  <a class="anchor" href="#paths-to-fine-mapping-with-susie-rss-model">#</a>
</h3>
<h3 id="paths-to-colocalization-with-susie-coloc">
  Path(s) to colocalization with SuSiE-coloc
  <a class="anchor" href="#paths-to-colocalization-with-susie-coloc">#</a>
</h3>
<h2 id="links-to-qtl-analysis-notebooks">
  Links to QTL analysis notebooks
  <a class="anchor" href="#links-to-qtl-analysis-notebooks">#</a>
</h2>
<p>Please list the links to analysis notebooks in the order you executed our pipeline:</p>
<ol>
<li>Genotype processing: 
  <a href="https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_genotype_processing.ipynb">https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_genotype_processing.ipynb</a></li>
<li>Phenotype processing: 
  <a href="https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_phenotype_processing.ipynb">https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_phenotype_processing.ipynb</a></li>
<li>Covariate processing: 
  <a href="https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_covariate_processing.ipynb">https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_covariate_processing.ipynb</a></li>
<li>TensorQTL analysis:  
  <a href="https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_tensorqtl.ipynb">https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_tensorqtl.ipynb</a></li>
<li>Complete analysis pipeline: 
  <a href="https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_complete_analysis.ipynb">https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Bush_CWRU/magenta_nhw_eqtl_complete_analysis.ipynb</a></li>
</ol>
</article>
 
      

      <footer class="book-footer">
        
  <div class="flex flex-wrap justify-between">





</div>



  <script>(function(){function a(c){const a=window.getSelection(),b=document.createRange();b.selectNodeContents(c),a.removeAllRanges(),a.addRange(b)}document.querySelectorAll("pre code").forEach(b=>{b.addEventListener("click",function(c){if(window.getSelection().toString())return;a(b.parentElement),navigator.clipboard&&navigator.clipboard.writeText(b.parentElement.textContent)})})})()</script>


 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      <div class="book-toc-content">
        
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#analysis-status">Analysis Status</a></li>
    <li><a href="#dataset-details">Dataset Details</a>
      <ul>
        <li><a href="#paths-to-genotype-matrix">Path(s) to genotype matrix</a></li>
        <li><a href="#paths-to-omics-data-matrix">Path(s) to omics-data matrix</a></li>
        <li><a href="#paths-to-covariate-data-matrix">Path(s) to covariate data matrix</a></li>
        <li><a href="#paths-to-qtl-results">Path(s) to QTL results</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-model">Path(s) to fine-mapping with SuSiE model</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-rss-model">Path(s) to fine-mapping with SuSiE RSS model</a></li>
        <li><a href="#paths-to-colocalization-with-susie-coloc">Path(s) to colocalization with SuSiE-coloc</a></li>
      </ul>
    </li>
    <li><a href="#links-to-qtl-analysis-notebooks">Links to QTL analysis notebooks</a></li>
  </ul>
</nav>


 
      </div>
    </aside>
    
  </main>

  
</body>
</html>












