<!DOCTYPE html>
<html lang="en-us" dir="ltr">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="ROSMAP WGS data #  Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) whole-genome sequence data
Contact #  Hao Sun and Xuanhe Chen
Data Descriptions #  This data is part of WGS done for three AMP-AD supported studies: The ROSMAP study (the data provided here), the MayoRNAseq study, and the MSBB study. Provided are VCF files, consisting of all samples from that study divided by chromosome, for the following analysis.">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ffffff">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#343a40">
<meta name="color-scheme" content="light dark"><meta property="og:title" content="ROSMAP WGS data" />
<meta property="og:description" content="ROSMAP WGS data #  Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) whole-genome sequence data
Contact #  Hao Sun and Xuanhe Chen
Data Descriptions #  This data is part of WGS done for three AMP-AD supported studies: The ROSMAP study (the data provided here), the MayoRNAseq study, and the MSBB study. Provided are VCF files, consisting of all samples from that study divided by chromosome, for the following analysis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://cumc.github.io/xqtl-data/r/rosmap-wgs-data/" /><meta property="article:section" content="r" />



<title>ROSMAP WGS data | FunGen-xQTL Data Repository</title>
<link rel="manifest" href="/xqtl-data/manifest.json">
<link rel="icon" href="/xqtl-data/favicon.png" >
<link rel="canonical" href="https://cumc.github.io/xqtl-data/r/rosmap-wgs-data/">
<link rel="stylesheet" href="/xqtl-data/book.min.33a48f5432973b8ff9a82679d9e45d67f2c15d4399bd2829269455cfe390b5e8.css" integrity="sha256-M6SPVDKXO4/5qCZ52eRdZ/LBXUOZvSgpJpRVz&#43;OQteg=" crossorigin="anonymous">
  <script defer src="/xqtl-data/flexsearch.min.js"></script>
  <script defer src="/xqtl-data/en.search.min.7d8c7087392e9b20178e66069a2711358324dd1c2231aa1919452888d5fa6faa.js" integrity="sha256-fYxwhzkumyAXjmYGmicRNYMk3RwiMaoZGUUoiNX6b6o=" crossorigin="anonymous"></script>

  <script defer src="/xqtl-data/sw.min.5a8d9ccf5ea73dcbbb1f34b183236a28dd36cc6d6540bb5b46431b4e00517252.js" integrity="sha256-Wo2cz16nPcu7HzSxgyNqKN02zG1lQLtbRkMbTgBRclI=" crossorigin="anonymous"></script>
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->
  
</head>
<body dir="ltr">
  <input type="checkbox" class="hidden toggle" id="menu-control" />
  <input type="checkbox" class="hidden toggle" id="toc-control" />
  <main class="container flex">
    <aside class="book-menu">
      <div class="book-menu-content">
        
  <nav>
<h2 class="book-brand">
  <a class="flex align-center" href="/xqtl-data/"><span>FunGen-xQTL Data Repository</span>
  </a>
</h2>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>












  <ul>
<li>Alphabetic List of Data Available
<ul>
<li>
  <a href="/xqtl-data/a/">A</a></li>
<li>
  <a href="/xqtl-data/d/">D</a></li>
<li>
  <a href="/xqtl-data/e/">E</a></li>
<li>
  <a href="/xqtl-data/f/">F</a></li>
<li>
  <a href="/xqtl-data/h/">H</a></li>
<li>
  <a href="/xqtl-data/k/">K</a></li>
<li>
  <a href="/xqtl-data/m/">M</a></li>
<li>
  <a href="/xqtl-data/n/">N</a></li>
<li>
  <a href="/xqtl-data/p/">P</a></li>
<li>
  <a href="/xqtl-data/r/">R</a></li>
<li>
  <a href="/xqtl-data/s/">S</a></li>
<li>
  <a href="/xqtl-data/t/">T</a></li>
<li>
  <a href="/xqtl-data/w/">W</a></li>
</ul>
</li>
</ul>










</nav>




  <script>(function(){var a=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(b){localStorage.setItem("menu.scrollTop",a.scrollTop)}),a.scrollTop=localStorage.getItem("menu.scrollTop")})()</script>


 
      </div>
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/xqtl-data/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>ROSMAP WGS data</strong>

  <label for="toc-control">
    
    <img src="/xqtl-data/svg/toc.svg" class="book-icon" alt="Table of Contents" />
    
  </label>
</div>


  
  <aside class="hidden clearfix">
    
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#data-descriptions">Data Descriptions</a></li>
  </ul>
</nav>



  </aside>
  
 
      </header>

      
      
  <article class="markdown book-article"><h1 id="rosmap-wgs-data">
  ROSMAP WGS data
  <a class="anchor" href="#rosmap-wgs-data">#</a>
</h1>
<p>Religious Orders Study (ROS) or the Rush Memory and Aging Project (MAP) whole-genome sequence data</p>
<h2 id="contact">
  Contact
  <a class="anchor" href="#contact">#</a>
</h2>
<p>Hao Sun and Xuanhe Chen</p>
<h2 id="data-descriptions">
  Data Descriptions
  <a class="anchor" href="#data-descriptions">#</a>
</h2>
<p>This data is part of WGS done for three AMP-AD supported studies: The ROSMAP study (the data provided here), the MayoRNAseq study, and the MSBB study. Provided are VCF files, consisting of all samples from that study divided by chromosome, for the following analysis.
Information about overlaps of ROSMAP samples and other database can be find 
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/ROSMAP/genotype/WGS_sample_id_map.ipynb">here</a>
Information of the preprocessing: 
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/ROSMAP/genotype/genotype_preprocessing.ipynb">https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/ROSMAP/genotype/genotype_preprocessing.ipynb</a></p>
<ul>
<li>Data Location on CU cluster:
The original VCF files of joint-call ROS/MAP WGS is here (N =  1162/4669 of them are ROS/MAP; GRCh38):
<code>/mnt/vast/hpc/bvardarajan_lab/data/Family_WGS/vcfs/vcf_b38_with_rosmap_2022/joint_vcf</code>
The intermediate VCF files for QCing the vcf can be found here:
<code>/mnt/mfs/ctcn/team/lu/project_h3k9ac/QC/output</code>
After processing through our pipeline, the processed plink file is here (N =  1160, MAC = 0, HWE = 1E-8, sample/variants missingness filter = 0.1 ; GRCh38)
<code>/mnt/vast/hpc/csg/FunGen_xQTL/ROSMAP/Genotype/ROSMAP_NIA_WGS.leftnorm.bcftools_qc.plink_qc.bed</code>
The LD matrixes calculated based on <strong>old version of this genotype</strong> (<code>/mnt/vast/hpc/csg/FunGen_xQTL/ROSMAP/genotype/ROSMAP_NIA_WGS.leftnorm.filtered.filtered.bed</code>) are here:
<code>/mnt/vast/hpc/csg/molecular_phenotype_calling/LD/output/1300_hg38_EUR_LD_blocks_LD</code></li>
<li>Data Location on MSSM cluster:
The ROSMAP monocyte WGS data was downloaded in VCF format, with one file per chromosome (plus X, Y, MT). (Path on MSSM to be added)</li>
</ul>
</article>
 
      

      <footer class="book-footer">
        
  <div class="flex flex-wrap justify-between">





</div>



  <script>(function(){function a(c){const a=window.getSelection(),b=document.createRange();b.selectNodeContents(c),a.removeAllRanges(),a.addRange(b)}document.querySelectorAll("pre code").forEach(b=>{b.addEventListener("click",function(c){if(window.getSelection().toString())return;a(b.parentElement),navigator.clipboard&&navigator.clipboard.writeText(b.parentElement.textContent)})})})()</script>


 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      <div class="book-toc-content">
        
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#data-descriptions">Data Descriptions</a></li>
  </ul>
</nav>


 
      </div>
    </aside>
    
  </main>

  
</body>
</html>












