<!DOCTYPE html>
<html lang="en-us" dir="ltr">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="xQTL protocol data #  A toy data-set consisting of 49 de-identified samples from ROSMAP project, used to illustrates the computational protocols we developed for the detection and analysis of molecular QTLs (xQTLs).
Contact #  Hao Sun
Location #  The input data and some of the intermediate output data can be download from this Synapse folder. To download the files, a synapse account and synapse clients are both needed.">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ffffff">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#343a40">
<meta name="color-scheme" content="light dark"><meta property="og:title" content="xQTL protocol data" />
<meta property="og:description" content="xQTL protocol data #  A toy data-set consisting of 49 de-identified samples from ROSMAP project, used to illustrates the computational protocols we developed for the detection and analysis of molecular QTLs (xQTLs).
Contact #  Hao Sun
Location #  The input data and some of the intermediate output data can be download from this Synapse folder. To download the files, a synapse account and synapse clients are both needed." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://cumc.github.io/xqtl-data/x/xqtl-protocol-data/" /><meta property="article:section" content="x" />



<title>xQTL protocol data | FunGen-xQTL Data Repository</title>
<link rel="manifest" href="/xqtl-data/manifest.json">
<link rel="icon" href="/xqtl-data/favicon.png" >
<link rel="canonical" href="https://cumc.github.io/xqtl-data/x/xqtl-protocol-data/">
<link rel="stylesheet" href="/xqtl-data/book.min.33a48f5432973b8ff9a82679d9e45d67f2c15d4399bd2829269455cfe390b5e8.css" integrity="sha256-M6SPVDKXO4/5qCZ52eRdZ/LBXUOZvSgpJpRVz&#43;OQteg=" crossorigin="anonymous">
  <script defer src="/xqtl-data/flexsearch.min.js"></script>
  <script defer src="/xqtl-data/en.search.min.5b634dda2013129f82f516c9314802c46ca2390a1bc1f8243ac473206dcd2f05.js" integrity="sha256-W2NN2iATEp&#43;C9RbJMUgCxGyiOQobwfgkOsRzIG3NLwU=" crossorigin="anonymous"></script>

  <script defer src="/xqtl-data/sw.min.5a8d9ccf5ea73dcbbb1f34b183236a28dd36cc6d6540bb5b46431b4e00517252.js" integrity="sha256-Wo2cz16nPcu7HzSxgyNqKN02zG1lQLtbRkMbTgBRclI=" crossorigin="anonymous"></script>
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->
  
</head>
<body dir="ltr">
  <input type="checkbox" class="hidden toggle" id="menu-control" />
  <input type="checkbox" class="hidden toggle" id="toc-control" />
  <main class="container flex">
    <aside class="book-menu">
      <div class="book-menu-content">
        
  <nav>
<h2 class="book-brand">
  <a class="flex align-center" href="/xqtl-data/"><span>FunGen-xQTL Data Repository</span>
  </a>
</h2>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>












  <ul>
<li>Alphabetic List of Data Available
<ul>
<li>
  <a href="/xqtl-data/a/">A</a></li>
<li>
  <a href="/xqtl-data/d/">D</a></li>
<li>
  <a href="/xqtl-data/e/">E</a></li>
<li>
  <a href="/xqtl-data/f/">F</a></li>
<li>
  <a href="/xqtl-data/h/">H</a></li>
<li>
  <a href="/xqtl-data/i/">I</a></li>
<li>
  <a href="/xqtl-data/k/">K</a></li>
<li>
  <a href="/xqtl-data/m/">M</a></li>
<li>
  <a href="/xqtl-data/n/">N</a></li>
<li>
  <a href="/xqtl-data/p/">P</a></li>
<li>
  <a href="/xqtl-data/r/">R</a></li>
<li>
  <a href="/xqtl-data/s/">S</a></li>
<li>
  <a href="/xqtl-data/t/">T</a></li>
<li>
  <a href="/xqtl-data/w/">W</a></li>
</ul>
</li>
</ul>










</nav>




  <script>(function(){var a=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(b){localStorage.setItem("menu.scrollTop",a.scrollTop)}),a.scrollTop=localStorage.getItem("menu.scrollTop")})()</script>


 
      </div>
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/xqtl-data/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>xQTL protocol data</strong>

  <label for="toc-control">
    
    <img src="/xqtl-data/svg/toc.svg" class="book-icon" alt="Table of Contents" />
    
  </label>
</div>


  
  <aside class="hidden clearfix">
    
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#location">Location</a></li>
    <li><a href="#how-the-data-was-prepared">How the data was prepared</a>
      <ul>
        <li><a href="#source-files">Source files</a></li>
        <li><a href="#generating-the-input-phenotype-data">Generating the input phenotype data</a></li>
        <li><a href="#creation-of-sample-name-mapper-and-masks">Creation of sample name mapper and masks</a></li>
        <li><a href="#de-identifing-the-input-phenotype-data">De-identifing the input phenotype data</a></li>
        <li><a href="#generating-the-input-fastq-list">Generating the input fastq list</a></li>
        <li><a href="#subsetting-and-indexing-the-genotypes">Subsetting and Indexing the genotypes</a></li>
      </ul>
    </li>
  </ul>
</nav>



  </aside>
  
 
      </header>

      
      
  <article class="markdown book-article"><h1 id="xqtl-protocol-data">
  xQTL protocol data
  <a class="anchor" href="#xqtl-protocol-data">#</a>
</h1>
<p>A toy data-set consisting of 49 de-identified samples from ROSMAP project, used to illustrates the computational protocols we developed for the detection and analysis of molecular QTLs (xQTLs).</p>
<h2 id="contact">
  Contact
  <a class="anchor" href="#contact">#</a>
</h2>
<p>Hao Sun</p>
<h2 id="location">
  Location
  <a class="anchor" href="#location">#</a>
</h2>
<p>The input data and some of the intermediate output data can be download from 
  <a href="https://www.synapse.org/#!Synapse:syn36416601">this Synapse folder</a>. To download the files, a synapse account and synapse clients are both needed. To setup a synapse client, please follow 
  <a href="https://help.synapse.org/docs/Installing-Synapse-API-Clients.1985249668.html">this post</a>.</p>
<h2 id="how-the-data-was-prepared">
  How the data was prepared
  <a class="anchor" href="#how-the-data-was-prepared">#</a>
</h2>
<h3 id="source-files">
  Source files
  <a class="anchor" href="#source-files">#</a>
</h3>
<p>The samples that we use are 49 samples of 
  <a href="https://www.synapse.org/#!Synapse:syn4164376">ROSMAP dataset</a>. The data used in this protocol paper after we processed and de-identified can be found at 
  <a href="/xqtl-data/">here</a></p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">cd /mnt/vast/hpc/csg/xqtl_workflow_testing/finalizing/ROSMAP_data/bam
</code></pre></div><div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash"><span style="color:#66d9ef">for</span> i in <span style="color:#e6db74">`</span>cat 50_samples_synapse_id<span style="color:#e6db74">`</span>; <span style="color:#66d9ef">do</span>
synapse get $i;
<span style="color:#66d9ef">done</span>
</code></pre></div><div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">The Genotype data are downloaded using:
</code></pre></div><div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">wget https://www.ebi.ac.uk/arrayexpress/files/E-GEUV-1/GEUVADIS.chr21.PH1PH2_465.IMPFRQFILT_BIALLELIC_PH.annotv2.genotypes.vcf.gz <span style="color:#ae81ff">\
</span><span style="color:#ae81ff"></span>https://www.ebi.ac.uk/arrayexpress/files/E-GEUV-1/GEUVADIS.chr22.PH1PH2_465.IMPFRQFILT_BIALLELIC_PH.annotv2.genotypes.vcf.gz
</code></pre></div><div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">cd ../../
</code></pre></div><p>Since we are using the fastq files as starting point of our RNASeq calling pipeline, the phenotype of xqtl protocol data required some preprocessing .</p>
<h3 id="generating-the-input-phenotype-data">
  Generating the input phenotype data
  <a class="anchor" href="#generating-the-input-phenotype-data">#</a>
</h3>
<p>Command 1 take only the chromosome 21 and 22 from each of the bam file in the desinated diretory, then command 2 changes them into fastq file. Doing so keeps our xqtl protocol data into a managable size</p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">sos run pipeline/phenotype_formatting.ipynb bam_subsetting  <span style="color:#ae81ff">\
</span><span style="color:#ae81ff"></span>--phenoFile <span style="color:#e6db74">`</span>ls ROSMAP_data/RNASeq/*.bam<span style="color:#e6db74">`</span> <span style="color:#ae81ff">\
</span><span style="color:#ae81ff"></span>--cwd ROSMAP_data/RNASeq/subsetted  <span style="color:#ae81ff">\
</span><span style="color:#ae81ff"></span>--container containers/rna_quantification.sif
</code></pre></div><div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">sos run pipeline/phenotype_formatting.ipynb bam_to_fastq  <span style="color:#ae81ff">\
</span><span style="color:#ae81ff"></span>--phenoFile <span style="color:#e6db74">`</span>ls ROSMAP_data/RNASeq/subsetted/*.bam<span style="color:#e6db74">`</span> <span style="color:#ae81ff">\
</span><span style="color:#ae81ff"></span>--cwd ROSMAP_data/RNASeq/fastq  <span style="color:#ae81ff">\
</span><span style="color:#ae81ff"></span>--container containers/rna_quantification.sif
</code></pre></div><h3 id="creation-of-sample-name-mapper-and-masks">
  Creation of sample name mapper and masks
  <a class="anchor" href="#creation-of-sample-name-mapper-and-masks">#</a>
</h3>
<p>To match and de-identified the samples in both Genotype/phenotype, a index file was created with the following codes</p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">echo -e <span style="color:#e6db74">&#34;fq1\tfq2&#34;</span> &gt; xqtl_protocol_data_sample_list
paste &lt;<span style="color:#f92672">(</span>ls *.1.fastq<span style="color:#f92672">)</span> &lt;<span style="color:#f92672">(</span>ls *.2.fastq<span style="color:#f92672">)</span> &gt;&gt; xqtl_protocol_data_sample_list
</code></pre></div><p><em><strong>Following codes are ran in python.</strong></em></p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">import pandas as pd
a <span style="color:#f92672">=</span> pd.read_csv<span style="color:#f92672">(</span><span style="color:#e6db74">&#34;xqtl_protocol_data_sample_list&#34;</span>,<span style="color:#e6db74">&#34;\t&#34;</span><span style="color:#f92672">)</span>
sample_id <span style="color:#f92672">=</span> <span style="color:#f92672">[</span>x.split<span style="color:#f92672">(</span><span style="color:#e6db74">&#34;.&#34;</span><span style="color:#f92672">)[</span>0<span style="color:#f92672">]</span> <span style="color:#66d9ef">for</span> x in a.fq1 <span style="color:#f92672">]</span>
b <span style="color:#f92672">=</span> pd.read_csv<span style="color:#f92672">(</span><span style="color:#e6db74">&#34;filtered_sample_index&#34;</span>,<span style="color:#e6db74">&#34;\t&#34;</span><span style="color:#f92672">)</span>
c <span style="color:#f92672">=</span> pd.read_csv<span style="color:#f92672">(</span><span style="color:#e6db74">&#34;ROSMAP_assay_rnaSeq_metadata.csv&#34;</span>,<span style="color:#e6db74">&#34;,&#34;</span><span style="color:#f92672">)</span>
a<span style="color:#f92672">[</span><span style="color:#e6db74">&#34;rnaseq_id&#34;</span><span style="color:#f92672">]</span> <span style="color:#f92672">=</span> sample_id
a.merge<span style="color:#f92672">(</span>b, on <span style="color:#f92672">=</span> <span style="color:#e6db74">&#34;rnaseq_id&#34;</span><span style="color:#f92672">)</span>
abc <span style="color:#f92672">=</span> ab.merge<span style="color:#f92672">(</span>c, left_on <span style="color:#f92672">=</span> <span style="color:#e6db74">&#34;rnaseq_id&#34;</span>, right_on <span style="color:#f92672">=</span> <span style="color:#e6db74">&#34;specimenID&#34;</span><span style="color:#f92672">)</span>
abc.to_csv<span style="color:#f92672">(</span><span style="color:#e6db74">&#34;../../comprehensive_xqtl_protocol_sample_index.tsv&#34;</span>,<span style="color:#e6db74">&#34;\t&#34;</span>,index <span style="color:#f92672">=</span> False<span style="color:#f92672">)</span>
</code></pre></div><p><code>ROSMAP_assay_rnaSeq_metadata.csv</code> can be downloaded from 
  <a href="https://www.synapse.org/#!Synapse:syn21088596">ROSMAP metadata</a> wherease <code>filtered_sample_index</code> is an internal file we used to determined which samples to used. For the purpose of deidentifying this file will not be released to the public.</p>
<h3 id="de-identifing-the-input-phenotype-data">
  De-identifing the input phenotype data
  <a class="anchor" href="#de-identifing-the-input-phenotype-data">#</a>
</h3>
<p>In compliance to HIPAA and the regulation on ROSMAP, we need to de-identify the data before releasing them to publics</p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">readarray -t array1 &lt;  &lt;<span style="color:#f92672">(</span>tail -49 ../../comprehensive_xqtl_protocol_sample_index.tsv | cut -f5<span style="color:#f92672">)</span>
readarray -t array2 &lt;  &lt;<span style="color:#f92672">(</span>tail -49 ../../comprehensive_xqtl_protocol_sample_index.tsv | cut -f3<span style="color:#f92672">)</span>
</code></pre></div><div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash"><span style="color:#66d9ef">for</span> i in <span style="color:#e6db74">${</span>!array1[*]<span style="color:#e6db74">}</span> ; <span style="color:#66d9ef">do</span> mv <span style="color:#e6db74">${</span>array1[$i]<span style="color:#e6db74">}</span>.subsetted.1.fastq Sample_<span style="color:#e6db74">${</span>array2[$i]<span style="color:#e6db74">}</span>.subsetted.1.fastq   ;<span style="color:#66d9ef">done</span>
<span style="color:#66d9ef">for</span> i in <span style="color:#e6db74">${</span>!array1[*]<span style="color:#e6db74">}</span> ; <span style="color:#66d9ef">do</span> mv <span style="color:#e6db74">${</span>array1[$i]<span style="color:#e6db74">}</span>.subsetted.2.fastq Sample_<span style="color:#e6db74">${</span>array2[$i]<span style="color:#e6db74">}</span>.subsetted.2.fastq   ;<span style="color:#66d9ef">done</span>
<span style="color:#66d9ef">for</span> i in <span style="color:#e6db74">${</span>!array1[*]<span style="color:#e6db74">}</span> ; <span style="color:#66d9ef">do</span> mv <span style="color:#e6db74">${</span>array1[$i]<span style="color:#e6db74">}</span>.subsetted.1.stderr Sample_<span style="color:#e6db74">${</span>array2[$i]<span style="color:#e6db74">}</span>.subsetted.1.stderr   ;<span style="color:#66d9ef">done</span>
<span style="color:#66d9ef">for</span> i in <span style="color:#e6db74">${</span>!array1[*]<span style="color:#e6db74">}</span> ; <span style="color:#66d9ef">do</span> mv <span style="color:#e6db74">${</span>array1[$i]<span style="color:#e6db74">}</span>.subsetted.1.stdout Sample_<span style="color:#e6db74">${</span>array2[$i]<span style="color:#e6db74">}</span>.subsetted.1.stdout   ;<span style="color:#66d9ef">done</span>
</code></pre></div><h3 id="generating-the-input-fastq-list">
  Generating the input fastq list
  <a class="anchor" href="#generating-the-input-fastq-list">#</a>
</h3>
<p>The input of our RNA calling section requirs a list of following format, it was generated manually. We allows 2 optional columns: strand and read_length so that user can specify different stand and read length for each of the samples. However, it is not necessary to include them. Our pipeline can detect the strand based on the output of STAR Alignment.
<em><strong>Following codes are ran in python.</strong></em></p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">import pandas as pd
abc <span style="color:#f92672">=</span> pd.read_csv<span style="color:#f92672">(</span><span style="color:#e6db74">&#34;comprehensive_xqtl_protocol_sample_index.tsv&#34;</span>,<span style="color:#e6db74">&#34;\t&#34;</span>,index <span style="color:#f92672">=</span> False<span style="color:#f92672">)</span>
abc <span style="color:#f92672">=</span> abc<span style="color:#f92672">[[</span><span style="color:#e6db74">&#34;sample_id&#34;</span>,<span style="color:#e6db74">&#34;fq1&#34;</span>,<span style="color:#e6db74">&#34;fq2&#34;</span>,<span style="color:#e6db74">&#34;strand&#34;</span>,<span style="color:#e6db74">&#34;readLength&#34;</span><span style="color:#f92672">]]</span>
abc<span style="color:#f92672">[</span><span style="color:#e6db74">&#34;fq1&#34;</span><span style="color:#f92672">]</span> <span style="color:#f92672">=</span>  <span style="color:#f92672">[</span><span style="color:#e6db74">&#34;.&#34;</span>.join<span style="color:#f92672">([</span>x<span style="color:#f92672">]</span> + y.split<span style="color:#f92672">(</span><span style="color:#e6db74">&#34;.&#34;</span><span style="color:#f92672">)[</span>1:<span style="color:#f92672">])</span>  <span style="color:#66d9ef">for</span> x,y in  zip<span style="color:#f92672">(</span> abc.sample_id, abc.fq1<span style="color:#f92672">)</span> <span style="color:#f92672">]</span>
abc<span style="color:#f92672">[</span><span style="color:#e6db74">&#34;fq2&#34;</span><span style="color:#f92672">]</span> <span style="color:#f92672">=</span>  <span style="color:#f92672">[</span><span style="color:#e6db74">&#34;.&#34;</span>.join<span style="color:#f92672">([</span>x<span style="color:#f92672">]</span> + y.split<span style="color:#f92672">(</span><span style="color:#e6db74">&#34;.&#34;</span><span style="color:#f92672">)[</span>1:<span style="color:#f92672">])</span>  <span style="color:#66d9ef">for</span> x,y in  zip<span style="color:#f92672">(</span> abc.sample_id, abc.fq2<span style="color:#f92672">)</span> <span style="color:#f92672">]</span>
abc.colums <span style="color:#f92672">=</span> <span style="color:#f92672">[</span><span style="color:#e6db74">&#34;ID&#34;</span>,<span style="color:#e6db74">&#34;fq1&#34;</span>,<span style="color:#e6db74">&#34;fq2&#34;</span>,<span style="color:#e6db74">&#34;strand&#34;</span>,<span style="color:#e6db74">&#34;read_length&#34;</span><span style="color:#f92672">]</span>
abc.to_csv<span style="color:#f92672">(</span><span style="color:#e6db74">&#34;xqtl_protocol_data.fastqlist&#34;</span>,<span style="color:#e6db74">&#34;\t&#34;</span>,index <span style="color:#f92672">=</span> False<span style="color:#f92672">)</span>
</code></pre></div><h3 id="subsetting-and-indexing-the-genotypes">
  Subsetting and Indexing the genotypes
  <a class="anchor" href="#subsetting-and-indexing-the-genotypes">#</a>
</h3>
<p>Since we only use 49 samples, we extract 49 samples from the genotype data to save memory and time</p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">cd ../
echo -e <span style="color:#e6db74">&#34;old_name\tnew_name&#34;</span> &gt; xqtl_protocol_data_sample_list
paste &lt;<span style="color:#f92672">(</span>cut -f6 ../comprehensive_xqtl_protocol_data_sample_index.tsv <span style="color:#f92672">)</span> &lt;<span style="color:#f92672">(</span>cut -f1 ../comprehensive_xqtl_protocol_data_sample_index.tsv  <span style="color:#f92672">)</span> &gt;&gt; xqtl_protocol_data_sample_mask
</code></pre></div><div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-bash" data-lang="bash">bcftools view DEJ_11898_B01_GRM_WGS_2017-05-15_21.recalibrated_variants.vcf.gz -S &lt;<span style="color:#f92672">(</span>cat ../comprehensive_xqtl_protocol_data_sample_index.tsv | cut -f6 | tail -49 <span style="color:#f92672">)</span> | <span style="color:#ae81ff">\
</span><span style="color:#ae81ff"></span>bcftools reheader --samples xqtl_protocol_data_sample_mask  -Oz -o DEJ_11898_B01_GRM_WGS_2017-05-15_21.recalibrated_variants.xqtl_protocol_data.vcf
bcftools view DEJ_11898_B01_GRM_WGS_2017-05-15_22.recalibrated_variants.vcf.gz -S &lt;<span style="color:#f92672">(</span>cat ../comprehensive_xqtl_protocol_data_sample_index.tsv | cut -f6 | tail -49 <span style="color:#f92672">)</span> | <span style="color:#ae81ff">\
</span><span style="color:#ae81ff"></span>bcftools reheader --samples xqtl_protocol_data_sample_mask  -Oz -o  DEJ_11898_B01_GRM_WGS_2017-05-15_22.recalibrated_variants.xqtl_protocol_data.vcf
bgzip DEJ_11898_B01_GRM_WGS_2017-05-15_21.recalibrated_variants.xqtl_protocol_data.vcf
bgzip DEJ_11898_B01_GRM_WGS_2017-05-15_22.recalibrated_variants.xqtl_protocol_data.vcf
tabix DEJ_11898_B01_GRM_WGS_2017-05-15_21.recalibrated_variants.xqtl_protocol_data.vcf.gz
tabix DEJ_11898_B01_GRM_WGS_2017-05-15_22.recalibrated_variants.xqtl_protocol_data.vcf.gz
</code></pre></div></article>
 
      

      <footer class="book-footer">
        
  <div class="flex flex-wrap justify-between">





</div>



  <script>(function(){function a(c){const a=window.getSelection(),b=document.createRange();b.selectNodeContents(c),a.removeAllRanges(),a.addRange(b)}document.querySelectorAll("pre code").forEach(b=>{b.addEventListener("click",function(c){if(window.getSelection().toString())return;a(b.parentElement),navigator.clipboard&&navigator.clipboard.writeText(b.parentElement.textContent)})})})()</script>


 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      <div class="book-toc-content">
        
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#location">Location</a></li>
    <li><a href="#how-the-data-was-prepared">How the data was prepared</a>
      <ul>
        <li><a href="#source-files">Source files</a></li>
        <li><a href="#generating-the-input-phenotype-data">Generating the input phenotype data</a></li>
        <li><a href="#creation-of-sample-name-mapper-and-masks">Creation of sample name mapper and masks</a></li>
        <li><a href="#de-identifing-the-input-phenotype-data">De-identifing the input phenotype data</a></li>
        <li><a href="#generating-the-input-fastq-list">Generating the input fastq list</a></li>
        <li><a href="#subsetting-and-indexing-the-genotypes">Subsetting and Indexing the genotypes</a></li>
      </ul>
    </li>
  </ul>
</nav>


 
      </div>
    </aside>
    
  </main>

  
</body>
</html>












