<!DOCTYPE html>
<html lang="en-us" dir="ltr">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="MAGENTA African American blood gene expression #  Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P. Hussman Institute for Human Genomics (HIHG) at the University of Miami Miller School of Medicine (Miami, FL), North Carolina A&amp;T State University (Greensboro, NC), and Case Western Reserve University (Cleveland, OH).">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ffffff">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#343a40">
<meta name="color-scheme" content="light dark"><meta property="og:title" content="MAGENTA African American blood gene expression" />
<meta property="og:description" content="MAGENTA African American blood gene expression #  Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P. Hussman Institute for Human Genomics (HIHG) at the University of Miami Miller School of Medicine (Miami, FL), North Carolina A&amp;T State University (Greensboro, NC), and Case Western Reserve University (Cleveland, OH)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://cumc.github.io/xqtl-data/m/magenta-african-american-blood-gene-expression/" /><meta property="article:section" content="m" />



<title>MAGENTA African American blood gene expression | FunGen-xQTL Data Repository</title>
<link rel="manifest" href="/xqtl-data/manifest.json">
<link rel="icon" href="/xqtl-data/favicon.png" >
<link rel="canonical" href="https://cumc.github.io/xqtl-data/m/magenta-african-american-blood-gene-expression/">
<link rel="stylesheet" href="/xqtl-data/book.min.33a48f5432973b8ff9a82679d9e45d67f2c15d4399bd2829269455cfe390b5e8.css" integrity="sha256-M6SPVDKXO4/5qCZ52eRdZ/LBXUOZvSgpJpRVz&#43;OQteg=" crossorigin="anonymous">
  <script defer src="/xqtl-data/flexsearch.min.js"></script>
  <script defer src="/xqtl-data/en.search.min.695034637e55c1c0c18c1e6a8cc3c29aa17415904372c71ee93ee5337657d4fc.js" integrity="sha256-aVA0Y35VwcDBjB5qjMPCmqF0FZBDcsce6T7lM3ZX1Pw=" crossorigin="anonymous"></script>

  <script defer src="/xqtl-data/sw.min.5a8d9ccf5ea73dcbbb1f34b183236a28dd36cc6d6540bb5b46431b4e00517252.js" integrity="sha256-Wo2cz16nPcu7HzSxgyNqKN02zG1lQLtbRkMbTgBRclI=" crossorigin="anonymous"></script>
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->
  
</head>
<body dir="ltr">
  <input type="checkbox" class="hidden toggle" id="menu-control" />
  <input type="checkbox" class="hidden toggle" id="toc-control" />
  <main class="container flex">
    <aside class="book-menu">
      <div class="book-menu-content">
        
  <nav>
<h2 class="book-brand">
  <a class="flex align-center" href="/xqtl-data/"><span>FunGen-xQTL Data Repository</span>
  </a>
</h2>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>












  <ul>
<li>Alphabetic List of Data Available
<ul>
<li>
  <a href="/xqtl-data/a/">A</a></li>
<li>
  <a href="/xqtl-data/d/">D</a></li>
<li>
  <a href="/xqtl-data/e/">E</a></li>
<li>
  <a href="/xqtl-data/f/">F</a></li>
<li>
  <a href="/xqtl-data/h/">H</a></li>
<li>
  <a href="/xqtl-data/i/">I</a></li>
<li>
  <a href="/xqtl-data/k/">K</a></li>
<li>
  <a href="/xqtl-data/m/">M</a></li>
<li>
  <a href="/xqtl-data/n/">N</a></li>
<li>
  <a href="/xqtl-data/p/">P</a></li>
<li>
  <a href="/xqtl-data/r/">R</a></li>
<li>
  <a href="/xqtl-data/s/">S</a></li>
<li>
  <a href="/xqtl-data/t/">T</a></li>
<li>
  <a href="/xqtl-data/w/">W</a></li>
</ul>
</li>
</ul>










</nav>




  <script>(function(){var a=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(b){localStorage.setItem("menu.scrollTop",a.scrollTop)}),a.scrollTop=localStorage.getItem("menu.scrollTop")})()</script>


 
      </div>
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/xqtl-data/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>MAGENTA African American blood gene expression</strong>

  <label for="toc-control">
    
    <img src="/xqtl-data/svg/toc.svg" class="book-icon" alt="Table of Contents" />
    
  </label>
</div>


  
  <aside class="hidden clearfix">
    
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#dataset-details">Dataset Details</a>
      <ul>
        <li><a href="#raw-data">Raw data</a></li>
        <li><a href="#molecular-phenotype-matrices">Molecular phenotype matrices</a></li>
        <li><a href="#other-key-data-files">Other key data files</a></li>
      </ul>
    </li>
    <li><a href="#links-to-omics-data-analysis-notebooks">Links to omics data analysis notebooks</a></li>
  </ul>
</nav>



  </aside>
  
 
      </header>

      
      
  <article class="markdown book-article"><h1 id="magenta-african-american-blood-gene-expression">
  MAGENTA African American blood gene expression
  <a class="anchor" href="#magenta-african-american-blood-gene-expression">#</a>
</h1>
<p>Multi-Ancestry Genomics, Epigenomics, and Transcriptomics of Alzheimer’s (MAGENTA) Project: Participants include 465 individuals (AA – 113 with AD, 118 cognitively intact controls; NHW – 116 with AD, 118 controls) ascertained by the John P. Hussman Institute for Human Genomics (HIHG) at the University of Miami Miller School of Medicine (Miami, FL), North Carolina A&amp;T State University (Greensboro, NC), and Case Western Reserve University (Cleveland, OH).  Participants were ascertained as part of the ADSP Follow-up Study and included both cases (&gt;65 years of age of onset) and controls (&gt;65 years of age at age of exam).  All participants were adjudicated by a clinical panel with expertise in AD related disorders and classified as AD according to standard criteria developed by the National Institute of Aging and the Alzheimer’s Association.</p>
<h2 id="contact">
  Contact
  <a class="anchor" href="#contact">#</a>
</h2>
<p>Makaela Mews (analyst); Dr. William S. Bush (
  <a href="mailto:wsb36@case.edu">wsb36@case.edu</a>);  Dr. Anthony J. Griswold (Griswold, Anthony J 
  <a href="mailto:agriswold@miami.edu">agriswold@miami.edu</a>)</p>
<h2 id="study-overview">
  Study Overview
  <a class="anchor" href="#study-overview">#</a>
</h2>
<ul>
<li>Study information: <code>study_info/MAGENTA.md</code>.</li>
</ul>
<h2 id="dataset-details">
  Dataset Details
  <a class="anchor" href="#dataset-details">#</a>
</h2>
<h3 id="raw-data">
  Raw data
  <a class="anchor" href="#raw-data">#</a>
</h3>
<p>URL to original data, if applicable:</p>
<ul>
<li>TO ADD: 
  <a href="https://dss.niagads.org/datasets/">https://dss.niagads.org/datasets/</a>&hellip;
Path(s) on HPC:</li>
<li>Bush Lab - Raw Genotypes, server: aneris-dev.case.edu <code>/storage2/BushLab/transfer/miami/AfricanAmerican</code></li>
<li>Bush Lab - Imputed Genotypes, server: aneris-dev.case.edu <code>/storage2/BushLab/transfer/miami/TPMs/revised_01_12_23/aa/imputed_geno</code></li>
<li>Bush Lab - Analysis, server: rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/aa_233</code>
Summary of Raw Genotypes:</li>
</ul>
<pre tabindex="0"><code>$  ls -lh
total 53M
-rw-r--r-- 1 wsb36 users  35M Oct 19 07:39 aa_all.bed
-rw-r--r-- 1 wsb36 users  19M Oct 19 07:39 aa_all.bim
-rw-r--r-- 1 wsb36 users 3.8K Oct 19 07:39 aa_all.fam
-rw-r--r-- 1 wsb36 users 2.0K Oct 19 07:39 all_AA_plink.txt
-rw-r--r-- 1 wsb36 users  924 Oct 19 07:39 case_AA_plink.txt
-rw-r--r-- 1 wsb36 users 1.1K Oct 19 07:39 con_AA_plink.txt
</code></pre><p>Summary of Imputed Genotypes:</p>
<pre tabindex="0"><code>$ ls -lh
total 26G
-rwxrwxr-x 1 mxm1368 mxm1368 1.3G Feb  6 10:14 chr10.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  21M Feb  6 10:14 chr10.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 1.4K Feb  6 10:14 chr_10.log
-rwxrwxr-x 1 mxm1368 mxm1368 1.2G Feb  6 10:14 chr11.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  20M Feb  6 10:14 chr11.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 1.4K Feb  6 10:14 chr_11.log
-rwxrwxr-x 1 mxm1368 mxm1368 1.2G Feb  6 10:15 chr12.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  20M Feb  6 10:15 chr12.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 1.4K Feb  6 10:15 chr_12.log
-rwxrwxr-x 1 mxm1368 mxm1368 888M Feb  6 10:15 chr13.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  15M Feb  6 10:15 chr13.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 1.2K Feb  6 10:15 chr_13.log
-rwxrwxr-x 1 mxm1368 mxm1368 826M Feb  6 10:15 chr14.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  14M Feb  6 10:15 chr14.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368  997 Feb  6 10:15 chr_14.log
-rwxrwxr-x 1 mxm1368 mxm1368 723M Feb  6 10:15 chr15.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  12M Feb  6 10:15 chr15.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368  997 Feb  6 10:15 chr_15.log
-rwxrwxr-x 1 mxm1368 mxm1368 801M Feb  6 10:15 chr16.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  14M Feb  6 10:15 chr16.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368  998 Feb  6 10:15 chr_16.log
-rwxrwxr-x 1 mxm1368 mxm1368 739M Feb  6 10:15 chr17.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  12M Feb  6 10:15 chr17.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368  995 Feb  6 10:15 chr_17.log
-rwxrwxr-x 1 mxm1368 mxm1368 743M Feb  6 10:15 chr18.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  12M Feb  6 10:15 chr18.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368  798 Feb  6 10:15 chr_18.log
-rwxrwxr-x 1 mxm1368 mxm1368 640M Feb  6 10:15 chr19.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368 9.1M Feb  6 10:15 chr19.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368  598 Feb  6 10:15 chr_19.log
-rwxrwxr-x 1 mxm1368 mxm1368 2.0G Feb  6 10:15 chr1.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  33M Feb  6 10:15 chr1.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 2.5K Feb  6 10:15 chr_1.log
-rwxrwxr-x 1 mxm1368 mxm1368 582M Feb  6 10:16 chr20.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368 9.5M Feb  6 10:16 chr20.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368  796 Feb  6 10:16 chr_20.log
-rwxrwxr-x 1 mxm1368 mxm1368 379M Feb  6 10:16 chr21.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368 5.8M Feb  6 10:16 chr21.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368  597 Feb  6 10:16 chr_21.log
-rwxrwxr-x 1 mxm1368 mxm1368 354M Feb  6 10:16 chr22.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368 5.4M Feb  6 10:16 chr22.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368  596 Feb  6 10:16 chr_22.log
-rwxrwxr-x 1 mxm1368 mxm1368 2.2G Feb  6 10:16 chr2.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  36M Feb  6 10:16 chr2.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 2.5K Feb  6 10:16 chr_2.log
-rwxrwxr-x 1 mxm1368 mxm1368 1.9G Feb  6 10:16 chr3.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  30M Feb  6 10:16 chr3.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 2.0K Feb  6 10:16 chr_3.log
-rwxrwxr-x 1 mxm1368 mxm1368 2.0G Feb  6 10:16 chr4.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  30M Feb  6 10:16 chr4.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 2.0K Feb  6 10:16 chr_4.log
-rwxrwxr-x 1 mxm1368 mxm1368 1.8G Feb  6 10:17 chr5.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  28M Feb  6 10:17 chr5.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 2.0K Feb  6 10:17 chr_5.log
-rwxrwxr-x 1 mxm1368 mxm1368 1.6G Feb  6 10:17 chr6.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  27M Feb  6 10:17 chr6.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 1.8K Feb  6 10:17 chr_6.log
-rwxrwxr-x 1 mxm1368 mxm1368 1.5G Feb  6 10:17 chr7.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  24M Feb  6 10:17 chr7.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 1.6K Feb  6 10:17 chr_7.log
-rwxrwxr-x 1 mxm1368 mxm1368 1.5G Feb  6 10:17 chr8.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  24M Feb  6 10:17 chr8.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 1.6K Feb  6 10:17 chr_8.log
-rwxrwxr-x 1 mxm1368 mxm1368 1.1G Feb  6 10:17 chr9.dose.vcf.gz
-rwxrwxr-x 1 mxm1368 mxm1368  18M Feb  6 10:17 chr9.info.gz
-rwxrwxr-x 1 mxm1368 mxm1368 1.4K Feb  6 10:17 chr_9.log
-rwxrwxr-x 1 mxm1368 mxm1368 808K Feb  6 10:17 qcreport.html
</code></pre><h3 id="molecular-phenotype-matrices">
  Molecular phenotype matrices
  <a class="anchor" href="#molecular-phenotype-matrices">#</a>
</h3>
<p>Path(s) on HPC:</p>
<ul>
<li>Bush Lab - Raw Gene Reads/TPMs, server: aneris-dev.case.edu <code>/storage2/BushLab/transfer/miami/TPMs/revised_01_12_23/aa</code></li>
<li>Bush Lab - Analysis, server: rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/aa_233</code>
A description of these matrices:</li>
<li>Raw Gene Reads/TPMs: 1st row is version (e.g., #1.2), 2nd row dimensions of file in terms of # genes x # samples (e.g., 60668   233), 3rd row Column Names (e.g., Name  Description   SAMPLE&hellip;), 4th+ row gene values (e.g., ENSG00000223972 DDX11L1 0 &hellip;)</li>
<li>Analysis Reads/TPMs: see <code>aa_miami_eQTL_phenotype.ipynb</code> for processing of raw gene reads/TPMs into analysis ready reads/TPMs; format: columns (e.g., gene_ID, SAMPLE&hellip;), rows (e.g., ENSG00000223972 0&hellip;)</li>
</ul>
<pre tabindex="0"><code>$ cd /mnt/pan/Data14/metabrain_lasso/rna_seq_norm/aa_233
# Number of rows:
$ wc -l aa_233_gene_reads_prep_v2.gct
60669 aa_233_gene_reads_prep_v2.gct
$ wc -l aa_233_gene_tpm_prep_v2.gct
60669 aa_233_gene_tpm_prep_v2.gct
# Number of columns:
$ awk '{print NF}' aa_233_gene_reads_prep_v2.gct | sort -n | uniq
234
$ awk '{print NF}' aa_233_gene_tpm_prep_v2.gct | sort -n | uniq
234
</code></pre><h3 id="other-key-data-files">
  Other key data files
  <a class="anchor" href="#other-key-data-files">#</a>
</h3>
<p>Path(s) on HPC:</p>
<ul>
<li>Bush Lab - Covariate data, server: rider.case.edu <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/AANHW_updated_472_RNA.csv</code></li>
</ul>
<pre tabindex="0"><code>$ head -1 AANHW_updated_472_RNA.csv
ID,cohort,Sex,APOE,AD_Control,age_of_onset,age_of_exam,SAMPLE
$ wc -l AANHW_updated_472_RNA.csv
475 AANHW_updated_472_RNA.csv
</code></pre><ul>
<li>Path(s) to the data on your local cluster where you analyzed the data. For example:</li>
<li>Bush Lab, <code>/mnt/pan/Data14/metabrain_lasso/rna_seq_norm/aa_233</code></li>
</ul>
<h2 id="links-to-omics-data-analysis-notebooks">
  Links to omics data analysis notebooks
  <a class="anchor" href="#links-to-omics-data-analysis-notebooks">#</a>
</h2>
<p>Please list links to the analysis in the order you performed them:</p>
<ol>
<li>Molecular Phenotype Processing: <code>magenta_aa_eqtl_phenotype_processing.ipynb</code></li>
<li>Covariate Processing: <code>magenta_aa_eqtl_covariate_processing.ipynb</code></li>
<li>Genotype Processing: <code>magenta_aa_eqtl_genotype_processing.ipynb</code></li>
<li>TensorQTL - cis &amp; trans: <code>magenta_aa_eqtl_tensorqtl.ipynb</code></li>
<li>Complete Analysis Protocol: <code>magenta_aa_eqtl_complete_analysis.ipynb</code></li>
</ol>
</article>
 
      

      <footer class="book-footer">
        
  <div class="flex flex-wrap justify-between">





</div>



  <script>(function(){function a(c){const a=window.getSelection(),b=document.createRange();b.selectNodeContents(c),a.removeAllRanges(),a.addRange(b)}document.querySelectorAll("pre code").forEach(b=>{b.addEventListener("click",function(c){if(window.getSelection().toString())return;a(b.parentElement),navigator.clipboard&&navigator.clipboard.writeText(b.parentElement.textContent)})})})()</script>


 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      <div class="book-toc-content">
        
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#dataset-details">Dataset Details</a>
      <ul>
        <li><a href="#raw-data">Raw data</a></li>
        <li><a href="#molecular-phenotype-matrices">Molecular phenotype matrices</a></li>
        <li><a href="#other-key-data-files">Other key data files</a></li>
      </ul>
    </li>
    <li><a href="#links-to-omics-data-analysis-notebooks">Links to omics data analysis notebooks</a></li>
  </ul>
</nav>


 
      </div>
    </aside>
    
  </main>

  
</body>
</html>












