<!DOCTYPE html>
<html lang="en-us" dir="ltr">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="Knight ADRC brain proteomics QTL #  Charles F. And Joanne Knight Alzheimer&rsquo;s Disease Research Center (Knight-ADRC) Please refer to this document for an overview of the Knight project.
Contact #  Zining Qi( zq2209@cumc.columbia.edu)
Study Overview #   Study name : Knight ADRC brain proteomics QTL Study Description : Charles F. And Joanne Knight Alzheimer&rsquo;s Disease Research Center (Knight-ADRC) brain proteomics QTL analysis summary statistics using the FGC xQTL pipeline.">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ffffff">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#343a40">
<meta name="color-scheme" content="light dark"><meta property="og:title" content="Knight ADRC brain proteomics QTL" />
<meta property="og:description" content="Knight ADRC brain proteomics QTL #  Charles F. And Joanne Knight Alzheimer&rsquo;s Disease Research Center (Knight-ADRC) Please refer to this document for an overview of the Knight project.
Contact #  Zining Qi( zq2209@cumc.columbia.edu)
Study Overview #   Study name : Knight ADRC brain proteomics QTL Study Description : Charles F. And Joanne Knight Alzheimer&rsquo;s Disease Research Center (Knight-ADRC) brain proteomics QTL analysis summary statistics using the FGC xQTL pipeline." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://cumc.github.io/xqtl-data/k/knight-adrc-brain-proteomics-qtl/" /><meta property="article:section" content="k" />



<title>Knight ADRC brain proteomics QTL | FunGen-xQTL Data Repository</title>
<link rel="manifest" href="/xqtl-data/manifest.json">
<link rel="icon" href="/xqtl-data/favicon.png" >
<link rel="canonical" href="https://cumc.github.io/xqtl-data/k/knight-adrc-brain-proteomics-qtl/">
<link rel="stylesheet" href="/xqtl-data/book.min.33a48f5432973b8ff9a82679d9e45d67f2c15d4399bd2829269455cfe390b5e8.css" integrity="sha256-M6SPVDKXO4/5qCZ52eRdZ/LBXUOZvSgpJpRVz&#43;OQteg=" crossorigin="anonymous">
  <script defer src="/xqtl-data/flexsearch.min.js"></script>
  <script defer src="/xqtl-data/en.search.min.fd7011d93cabe050c70980ac5fda5d47a1920abc6a0f225733faa3ae96df1be4.js" integrity="sha256-/XAR2Tyr4FDHCYCsX9pdR6GSCrxqDyJXM/qjrpbfG&#43;Q=" crossorigin="anonymous"></script>

  <script defer src="/xqtl-data/sw.min.5a8d9ccf5ea73dcbbb1f34b183236a28dd36cc6d6540bb5b46431b4e00517252.js" integrity="sha256-Wo2cz16nPcu7HzSxgyNqKN02zG1lQLtbRkMbTgBRclI=" crossorigin="anonymous"></script>
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->
  
</head>
<body dir="ltr">
  <input type="checkbox" class="hidden toggle" id="menu-control" />
  <input type="checkbox" class="hidden toggle" id="toc-control" />
  <main class="container flex">
    <aside class="book-menu">
      <div class="book-menu-content">
        
  <nav>
<h2 class="book-brand">
  <a class="flex align-center" href="/xqtl-data/"><span>FunGen-xQTL Data Repository</span>
  </a>
</h2>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>












  <ul>
<li>Alphabetic List of Data Available
<ul>
<li>
  <a href="/xqtl-data/a/">A</a></li>
<li>
  <a href="/xqtl-data/d/">D</a></li>
<li>
  <a href="/xqtl-data/e/">E</a></li>
<li>
  <a href="/xqtl-data/f/">F</a></li>
<li>
  <a href="/xqtl-data/h/">H</a></li>
<li>
  <a href="/xqtl-data/i/">I</a></li>
<li>
  <a href="/xqtl-data/k/">K</a></li>
<li>
  <a href="/xqtl-data/m/">M</a></li>
<li>
  <a href="/xqtl-data/n/">N</a></li>
<li>
  <a href="/xqtl-data/p/">P</a></li>
<li>
  <a href="/xqtl-data/r/">R</a></li>
<li>
  <a href="/xqtl-data/s/">S</a></li>
<li>
  <a href="/xqtl-data/t/">T</a></li>
<li>
  <a href="/xqtl-data/w/">W</a></li>
</ul>
</li>
</ul>










</nav>




  <script>(function(){var a=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(b){localStorage.setItem("menu.scrollTop",a.scrollTop)}),a.scrollTop=localStorage.getItem("menu.scrollTop")})()</script>


 
      </div>
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/xqtl-data/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>Knight ADRC brain proteomics QTL</strong>

  <label for="toc-control">
    
    <img src="/xqtl-data/svg/toc.svg" class="book-icon" alt="Table of Contents" />
    
  </label>
</div>


  
  <aside class="hidden clearfix">
    
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#analysis-status">Analysis Status</a></li>
    <li><a href="#analysis-details">Analysis Details</a>
      <ul>
        <li><a href="#sample-size-tracking-through-the-analysis-workflow">Sample size tracking through the analysis workflow</a></li>
        <li><a href="#qc-and-normalization-details-for-phenotype-data">QC and Normalization Details for Phenotype Data</a></li>
        <li><a href="#genotype-parameters">Genotype parameters</a></li>
        <li><a href="#covariate-parameters">Covariate parameters</a></li>
        <li><a href="#tensorqtl-association-scan-parameters">TensorQTL association scan parameters</a></li>
      </ul>
    </li>
    <li><a href="#links-to-qtl-analysis-notebooks">Links to QTL analysis notebooks</a>
      <ul>
        <li><a href="#association-data-preprocessing">Association data preprocessing</a></li>
        <li><a href="#association-scan-using-tensorqtl-and-summary-statistics-standardization">Association scan using TensorQTL and summary statistics standardization</a></li>
      </ul>
    </li>
    <li><a href="#dataset-description">Dataset Description</a>
      <ul>
        <li><a href="#paths-to-genotype-matrix">Path(s) to genotype matrix</a></li>
        <li><a href="#paths-to-omics-data-matrix">Path(s) to omics-data matrix</a></li>
        <li><a href="#paths-to-covariate-data-matrix">Path(s) to covariate data matrix</a></li>
        <li><a href="#paths-to-qtl-results">Path(s) to QTL results</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-model">Path(s) to fine-mapping with SuSiE model</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-rss-model">Path(s) to fine-mapping with SuSiE RSS model</a></li>
      </ul>
    </li>
  </ul>
</nav>



  </aside>
  
 
      </header>

      
      
  <article class="markdown book-article"><h1 id="knight-adrc-brain-proteomics-qtl">
  Knight ADRC brain proteomics QTL
  <a class="anchor" href="#knight-adrc-brain-proteomics-qtl">#</a>
</h1>
<p>Charles F. And Joanne Knight Alzheimer&rsquo;s Disease Research Center (Knight-ADRC)
Please refer to 
  <a href="../study_info/KnightADRC.ipynb">this document</a> for an overview of the Knight project.</p>
<h2 id="contact">
  Contact
  <a class="anchor" href="#contact">#</a>
</h2>
<p>Zining Qi(
  <a href="mailto:zq2209@cumc.columbia.edu">zq2209@cumc.columbia.edu</a>)</p>
<h2 id="study-overview">
  Study Overview
  <a class="anchor" href="#study-overview">#</a>
</h2>
<ul>
<li>Study name : Knight ADRC brain proteomics QTL</li>
<li>Study Description : Charles F. And Joanne Knight Alzheimer&rsquo;s Disease Research Center (Knight-ADRC) brain proteomics QTL analysis summary statistics using the FGC xQTL pipeline.
Samples' phenotype information (sex, age, race etc.) can be found in Knight metadata, see &ldquo;Other information&rdquo; section in 
  <a href="../study_info/KnightADRC.md">this document</a></li>
</ul>
<h2 id="analysis-status">
  Analysis Status
  <a class="anchor" href="#analysis-status">#</a>
</h2>
<p>TransQTL association: Need to be updated with new covariate files.</p>
<h2 id="analysis-details">
  Analysis Details
  <a class="anchor" href="#analysis-details">#</a>
</h2>
<h3 id="sample-size-tracking-through-the-analysis-workflow">
  Sample size tracking through the analysis workflow
  <a class="anchor" href="#sample-size-tracking-through-the-analysis-workflow">#</a>
</h3>
<table>
<thead>
<tr>
<th>Step</th>
<th>Sample Size</th>
</tr>
</thead>
<tbody>
<tr>
<td>Raw data</td>
<td>442</td>
</tr>
<tr>
<td>After overlapping with Knight covariates</td>
<td>418</td>
</tr>
<tr>
<td>After overlapping with Knight genotype</td>
<td>412</td>
</tr>
</tbody>
</table>
<h3 id="qc-and-normalization-details-for-phenotype-data">
  QC and Normalization Details for Phenotype Data
  <a class="anchor" href="#qc-and-normalization-details-for-phenotype-data">#</a>
</h3>
<p>For QCs performed on data please refer to <code>descriptor/omics/Knight_ADRC_brain_proteomics.md</code></p>
<h3 id="genotype-parameters">
  Genotype parameters
  <a class="anchor" href="#genotype-parameters">#</a>
</h3>
<ul>
<li>Kinship coefficient for related individuals: 0.0625</li>
<li>Minor allele count cutoff for single variant analysis: 5</li>
<li>Variant level missingness threshold: 0.1</li>
<li>Sample level missingness threshold: 0.1</li>
</ul>
<h3 id="covariate-parameters">
  Covariate parameters
  <a class="anchor" href="#covariate-parameters">#</a>
</h3>
<p>PCA analysis:</p>
<ul>
<li>MAF for PCA: 0.05</li>
<li>LD pruning via PLINK for PCA analysis: window 50, shift 10, r2 0.1</li>
<li>Tolerance of missingness in covariates: 0.3</li>
<li>number of pc that captured more than 70% PVE were kept
Hidden Factor analysis - PCA</li>
<li>Use Marchenko-Pastur limit to choose PCs
Final covariates: Sex, Age at death, PMI, PCs(12), hidden factors(18)</li>
</ul>
<h3 id="tensorqtl-association-scan-parameters">
  TensorQTL association scan parameters
  <a class="anchor" href="#tensorqtl-association-scan-parameters">#</a>
</h3>
<ul>
<li>cis/trans window for the up and downstream radius to analyze around the region of interest, in units of bp: 1000000</li>
<li>Minor allele count cutoff: 5</li>
<li>Minor allele frequency cutoff: 5/(2.0*592) = 0.00422297</li>
</ul>
<h2 id="links-to-qtl-analysis-notebooks">
  Links to QTL analysis notebooks
  <a class="anchor" href="#links-to-qtl-analysis-notebooks">#</a>
</h2>
<p>See notebooks in:</p>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Wang_Columbia/knight/pqtl">https://github.com/cumc/fungen-xqtl-analysis/tree/main/analysis/Wang_Columbia/knight/pqtl</a>
The notebooks in this folder contain the commands and data wrangling codes for analysis of the proteomics data in ROSMAP. (data wrangling exist because not all data are processed using the xqtl-pipeline from the begnning and need to be reformated to fit one intermediate step of the pipeline).</li>
</ul>
<h3 id="association-data-preprocessing">
  Association data preprocessing
  <a class="anchor" href="#association-data-preprocessing">#</a>
</h3>
<p>phenotype preprocessing can be run in parallel but covariate preprocessing need to be done last since it requires both phenotype data and PCs from PCA in genotype preprocessing.</p>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/knight/pqtl/genotype_preprocessing.ipynb">genotype_preprocessing.ipynb</a> shows the commands used for genotype processing and preparation steps.</li>
<li>Includes QC, PCA, and PCA result figures.</li>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/knight/pqtl/phenotype_preprocessing.ipynb">phenotype_preprocessing.ipynb</a> shows the commands used for the phenotype data processing and preparation steps.</li>
<li>This notebook strated from some already cleaned data and covert them to bed format inputs taken by tensorQTL.</li>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/knight/pqtl/covariates_preprocessing.ipynb">covariate_preprocessing.ipynb</a> shows the commands used for the covariate data processing and preparation steps.</li>
<li>This includes PCA hidden factor analysis steps and merging the PCA hidden factors to covariates and PCA.</li>
</ul>
<h3 id="association-scan-using-tensorqtl-and-summary-statistics-standardization">
  Association scan using TensorQTL and summary statistics standardization
  <a class="anchor" href="#association-scan-using-tensorqtl-and-summary-statistics-standardization">#</a>
</h3>
<ul>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/knight/pqtl/cis_haqtl_association.ipynb">cis_pqtl_association.ipynb</a> provides information and result of cis-QTL analysis.</li>
<li>
  <a href="https://github.com/cumc/fungen-xqtl-analysis/blob/main/analysis/Wang_Columbia/knight/pqtl/trans_pqtl_association.ipynb">trans_pqtl_association.ipynb</a> provides information and result of AD genes trans-QTL analysis.</li>
</ul>
<h2 id="dataset-description">
  Dataset Description
  <a class="anchor" href="#dataset-description">#</a>
</h2>
<h3 id="paths-to-genotype-matrix">
  Path(s) to genotype matrix
  <a class="anchor" href="#paths-to-genotype-matrix">#</a>
</h3>
<ul>
<li>Wang Lab, <code>/mnt/vast/hpc/csg/molecular_phenotype_calling/WashU_genotype/genotype_qc</code></li>
</ul>
<pre tabindex="0"><code>-rw-r--r-- 1 zq2209 zq2209  48M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.10.bed
-rw-r--r-- 1 zq2209 zq2209  17M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.10.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.10.fam
-rw-r--r-- 1 zq2209 zq2209  47M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.11.bed
-rw-r--r-- 1 zq2209 zq2209  16M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.11.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.11.fam
-rw-r--r-- 1 zq2209 zq2209  45M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.12.bed
-rw-r--r-- 1 zq2209 zq2209  16M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.12.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.12.fam
-rw-r--r-- 1 zq2209 zq2209  35M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.13.bed
-rw-r--r-- 1 zq2209 zq2209  12M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.13.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.13.fam
-rw-r--r-- 1 zq2209 zq2209  30M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.14.bed
-rw-r--r-- 1 zq2209 zq2209  11M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.14.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.14.fam
-rw-r--r-- 1 zq2209 zq2209  26M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.15.bed
-rw-r--r-- 1 zq2209 zq2209 8.9M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.15.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.15.fam
-rw-r--r-- 1 zq2209 zq2209  29M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.16.bed
-rw-r--r-- 1 zq2209 zq2209 9.6M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.16.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.16.fam
-rw-r--r-- 1 zq2209 zq2209  25M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.17.bed
-rw-r--r-- 1 zq2209 zq2209 8.3M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.17.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.17.fam
-rw-r--r-- 1 zq2209 zq2209  27M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.18.bed
-rw-r--r-- 1 zq2209 zq2209 9.1M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.18.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.18.fam
-rw-r--r-- 1 zq2209 zq2209  21M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.19.bed
-rw-r--r-- 1 zq2209 zq2209 7.0M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.19.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.19.fam
-rw-r--r-- 1 zq2209 zq2209  72M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.1.bed
-rw-r--r-- 1 zq2209 zq2209  24M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.1.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.1.fam
-rw-r--r-- 1 zq2209 zq2209  21M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.20.bed
-rw-r--r-- 1 zq2209 zq2209 7.0M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.20.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.20.fam
-rw-r--r-- 1 zq2209 zq2209  13M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.21.bed
-rw-r--r-- 1 zq2209 zq2209 4.3M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.21.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.21.fam
-rw-r--r-- 1 zq2209 zq2209  13M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.22.bed
-rw-r--r-- 1 zq2209 zq2209 4.3M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.22.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.22.fam
-rw-r--r-- 1 zq2209 zq2209  77M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.2.bed
-rw-r--r-- 1 zq2209 zq2209  26M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.2.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.2.fam
-rw-r--r-- 1 zq2209 zq2209  66M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.3.bed
-rw-r--r-- 1 zq2209 zq2209  22M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.3.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.3.fam
-rw-r--r-- 1 zq2209 zq2209  68M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.4.bed
-rw-r--r-- 1 zq2209 zq2209  23M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.4.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.4.fam
-rw-r--r-- 1 zq2209 zq2209  61M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.5.bed
-rw-r--r-- 1 zq2209 zq2209  20M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.5.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.5.fam
-rw-r--r-- 1 zq2209 zq2209  62M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.6.bed
-rw-r--r-- 1 zq2209 zq2209  21M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.6.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.6.fam
-rw-r--r-- 1 zq2209 zq2209  55M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.7.bed
-rw-r--r-- 1 zq2209 zq2209  18M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.7.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.7.fam
-rw-r--r-- 1 zq2209 zq2209  51M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.8.bed
-rw-r--r-- 1 zq2209 zq2209  17M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.8.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.8.fam
-rw-r--r-- 1 zq2209 zq2209  41M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.9.bed
-rw-r--r-- 1 zq2209 zq2209  14M Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.9.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.9.fam
-rw-r--r-- 1 zq2209 zq2209 922M Feb 14 10:37 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.bed
-rw-r--r-- 1 zq2209 zq2209 309M Feb 14 10:37 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:30 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.fam
-rw-r--r-- 1 zq2209 zq2209  19K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.X.bed
-rw-r--r-- 1 zq2209 zq2209 6.2K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.X.bim
-rw-r--r-- 1 zq2209 zq2209 9.4K Feb 14 17:36 MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.X.fam
</code></pre><h3 id="paths-to-omics-data-matrix">
  Path(s) to omics-data matrix
  <a class="anchor" href="#paths-to-omics-data-matrix">#</a>
</h3>
<ul>
<li>Wang Lab, <code>/mnt/vast/hpc/csg/zq2209/data_production/proteomics/knight/pheno</code></li>
</ul>
<pre tabindex="0"><code>$ ls -lh *.{txt,gz}
-rw-r--r-- 1 zq2209 zq2209 4.2M Jun  1 13:18  WashU_pheno.bed.gz
-rw-r--r-- 1 zq2209 zq2209 133K Jun  1 13:18  WashU_pheno_chr10.bed.gz
-rw-r--r-- 1 zq2209 zq2209 243K Jun  1 13:18  WashU_pheno_chr11.bed.gz
-rw-r--r-- 1 zq2209 zq2209 284K Jun  1 13:18  WashU_pheno_chr12.bed.gz
-rw-r--r-- 1 zq2209 zq2209  83K Jun  1 13:18  WashU_pheno_chr13.bed.gz
-rw-r--r-- 1 zq2209 zq2209 122K Jun  1 13:18  WashU_pheno_chr14.bed.gz
-rw-r--r-- 1 zq2209 zq2209 111K Jun  1 13:18  WashU_pheno_chr15.bed.gz
-rw-r--r-- 1 zq2209 zq2209 144K Jun  1 13:18  WashU_pheno_chr16.bed.gz
-rw-r--r-- 1 zq2209 zq2209 291K Jun  1 13:18  WashU_pheno_chr17.bed.gz
-rw-r--r-- 1 zq2209 zq2209  57K Jun  1 13:18  WashU_pheno_chr18.bed.gz
-rw-r--r-- 1 zq2209 zq2209 348K Jun  1 13:18  WashU_pheno_chr19.bed.gz
-rw-r--r-- 1 zq2209 zq2209 453K Jun  1 13:18  WashU_pheno_chr1.bed.gz
-rw-r--r-- 1 zq2209 zq2209 151K Jun  1 13:18  WashU_pheno_chr20.bed.gz
-rw-r--r-- 1 zq2209 zq2209  91K Jun  1 13:18  WashU_pheno_chr21.bed.gz
-rw-r--r-- 1 zq2209 zq2209 103K Jun  1 13:18  WashU_pheno_chr22.bed.gz
-rw-r--r-- 1 zq2209 zq2209 238K Jun  1 13:18  WashU_pheno_chr2.bed.gz
-rw-r--r-- 1 zq2209 zq2209 220K Jun  1 13:18  WashU_pheno_chr3.bed.gz
-rw-r--r-- 1 zq2209 zq2209 235K Jun  1 13:18  WashU_pheno_chr4.bed.gz
-rw-r--r-- 1 zq2209 zq2209 222K Jun  1 13:18  WashU_pheno_chr5.bed.gz
-rw-r--r-- 1 zq2209 zq2209 177K Jun  1 13:18  WashU_pheno_chr6.bed.gz
-rw-r--r-- 1 zq2209 zq2209 182K Jun  1 13:18  WashU_pheno_chr7.bed.gz
-rw-r--r-- 1 zq2209 zq2209 129K Jun  1 13:18  WashU_pheno_chr8.bed.gz
-rw-r--r-- 1 zq2209 zq2209 138K Jun  1 13:18  WashU_pheno_chr9.bed.gz
-rw-r--r-- 1 zq2209 zq2209 115K Jun  1 13:18  WashU_pheno_chrX.bed.gz
</code></pre><h3 id="paths-to-covariate-data-matrix">
  Path(s) to covariate data matrix
  <a class="anchor" href="#paths-to-covariate-data-matrix">#</a>
</h3>
<ul>
<li>Wang Lab, <code>/mnt/vast/hpc/csg/zq2209/data_production/proteomics/knight/cov</code></li>
</ul>
<pre tabindex="0"><code>$ ls -lh *.gz
-rw-r--r-- 1 zq2209 zq2209 113K Jun  1 13:22 WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.gz
</code></pre><h3 id="paths-to-qtl-results">
  Path(s) to QTL results
  <a class="anchor" href="#paths-to-qtl-results">#</a>
</h3>
<ul>
<li>Wang lab: <code>/mnt/vast/hpc/csg/molecular_phenotype_calling/pQTL_cis/WashU</code></li>
</ul>
<pre tabindex="0"><code>ls -lh *.txt
-rw-r--r-- 1 zq2209 zq2209 8.9K May 19 14:33 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.10.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  44M May 19 14:32 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.10.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  17K May 19 14:34 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.11.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  76M May 19 14:34 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.11.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  19K May 19 14:36 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.12.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  81M May 19 14:35 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.12.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 5.5K May 19 14:35 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.13.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  26M May 19 14:35 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.13.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 8.6K May 19 14:35 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.14.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  41M May 19 14:34 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.14.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 7.2K May 19 14:35 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.15.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  29M May 19 14:35 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.15.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 9.8K May 19 14:35 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.16.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  42M May 19 14:35 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.16.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  20K May 19 14:37 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.17.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  81M May 19 14:36 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.17.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 3.8K May 19 14:36 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.18.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  18M May 19 14:36 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.18.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  24K May 19 14:36 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.19.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 122M May 19 14:36 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.19.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  31K May 19 14:33 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.1.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209 118M May 19 14:32 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.1.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  11K May 19 14:37 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.20.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  46M May 19 14:37 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.20.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 6.2K May 19 14:36 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.21.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  31M May 19 14:36 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.21.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 7.2K May 19 14:37 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.22.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  29M May 19 14:37 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.22.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  17K May 19 14:31 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.2.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  69M May 19 14:31 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.2.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  15K May 19 14:32 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.3.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  60M May 19 14:31 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.3.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  16K May 19 14:32 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.4.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  70M May 19 14:31 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.4.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  15K May 19 14:32 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.5.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  64M May 19 14:31 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.5.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  12K May 19 14:34 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.6.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  83M May 19 14:33 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.6.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209  13K May 19 14:33 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.7.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  51M May 19 14:33 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.7.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 9.0K May 19 14:34 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.8.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  40M May 19 14:34 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.8.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 9.5K May 19 14:34 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.9.emprical.cis_sumstats.txt
-rw-r--r-- 1 zq2209 zq2209  41M May 19 14:33 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.9.norminal.cis_long_table.txt
-rw-r--r-- 1 zq2209 zq2209 363K May 19 14:38 pheno_recipe_WashU_pheno.WashU_cov.MAP_Brain-xQTL_Gwas_geno_0.1_maf_0.0005.filtered.pQTL.related.filtered.extracted.pca.projected.resid.Marchenko_pc.emprical.cis_sumstats.txt
</code></pre><ul>
<li>FTP:</li>
</ul>
<h3 id="paths-to-fine-mapping-with-susie-model">
  Path(s) to fine-mapping with SuSiE model
  <a class="anchor" href="#paths-to-fine-mapping-with-susie-model">#</a>
</h3>
<h3 id="paths-to-fine-mapping-with-susie-rss-model">
  Path(s) to fine-mapping with SuSiE RSS model
  <a class="anchor" href="#paths-to-fine-mapping-with-susie-rss-model">#</a>
</h3>
</article>
 
      

      <footer class="book-footer">
        
  <div class="flex flex-wrap justify-between">





</div>



  <script>(function(){function a(c){const a=window.getSelection(),b=document.createRange();b.selectNodeContents(c),a.removeAllRanges(),a.addRange(b)}document.querySelectorAll("pre code").forEach(b=>{b.addEventListener("click",function(c){if(window.getSelection().toString())return;a(b.parentElement),navigator.clipboard&&navigator.clipboard.writeText(b.parentElement.textContent)})})})()</script>


 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      <div class="book-toc-content">
        
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#study-overview">Study Overview</a></li>
    <li><a href="#analysis-status">Analysis Status</a></li>
    <li><a href="#analysis-details">Analysis Details</a>
      <ul>
        <li><a href="#sample-size-tracking-through-the-analysis-workflow">Sample size tracking through the analysis workflow</a></li>
        <li><a href="#qc-and-normalization-details-for-phenotype-data">QC and Normalization Details for Phenotype Data</a></li>
        <li><a href="#genotype-parameters">Genotype parameters</a></li>
        <li><a href="#covariate-parameters">Covariate parameters</a></li>
        <li><a href="#tensorqtl-association-scan-parameters">TensorQTL association scan parameters</a></li>
      </ul>
    </li>
    <li><a href="#links-to-qtl-analysis-notebooks">Links to QTL analysis notebooks</a>
      <ul>
        <li><a href="#association-data-preprocessing">Association data preprocessing</a></li>
        <li><a href="#association-scan-using-tensorqtl-and-summary-statistics-standardization">Association scan using TensorQTL and summary statistics standardization</a></li>
      </ul>
    </li>
    <li><a href="#dataset-description">Dataset Description</a>
      <ul>
        <li><a href="#paths-to-genotype-matrix">Path(s) to genotype matrix</a></li>
        <li><a href="#paths-to-omics-data-matrix">Path(s) to omics-data matrix</a></li>
        <li><a href="#paths-to-covariate-data-matrix">Path(s) to covariate data matrix</a></li>
        <li><a href="#paths-to-qtl-results">Path(s) to QTL results</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-model">Path(s) to fine-mapping with SuSiE model</a></li>
        <li><a href="#paths-to-fine-mapping-with-susie-rss-model">Path(s) to fine-mapping with SuSiE RSS model</a></li>
      </ul>
    </li>
  </ul>
</nav>


 
      </div>
    </aside>
    
  </main>

  
</body>
</html>












