<!DOCTYPE html>
<html lang="en-us" dir="ltr">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="Alzheimer&rsquo;s Disease GWAS Summary Data (Kunkle) #  The SNP-level association testing summary statistics for Alzheimer&rsquo;s disease from Kunkle et al 2019 Nature Genetics. Position values were converted from hg19 to hg38 using liftOver.
Contact #  Oluwatosin Olayinka
Path(s) to summary statistics #   NIAGADS FTP, /ftp_fgc_xqtl/projects/ADGWAS_Kunkle_2019_hg38_liftover/kunkle_sumstat_hg38_qc.chr* CU original data (in GRCh37) stage 1: /mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/&#39;Kunkle_etal_Stage1_results.txt?file=1&#39; stage 2: /mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/&#39;Kunkle_etal_Stage2_results.txt?file=1&#39; liftover data (in GRCh38): stage 1: /mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/Kunkle_etal_Stage1_results.txt_file_1_hg38.txt stage 2: /mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/Kunkle_etal_Stage2_results.txt_file_1_hg38.txt  Path to SuSiE RSS Fine-mapping Objects #   Li-San Wang FTP: /ftp_fgc_xqtl/projects/GWAS_Finemapping_Results/Kunkle/  Download source #  This data is derived from summary statistics from the Kunkle et al.">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ffffff">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#343a40">
<meta name="color-scheme" content="light dark"><meta property="og:title" content="Alzheimer&#39;s Disease GWAS Summary Data (Kunkle)" />
<meta property="og:description" content="Alzheimer&rsquo;s Disease GWAS Summary Data (Kunkle) #  The SNP-level association testing summary statistics for Alzheimer&rsquo;s disease from Kunkle et al 2019 Nature Genetics. Position values were converted from hg19 to hg38 using liftOver.
Contact #  Oluwatosin Olayinka
Path(s) to summary statistics #   NIAGADS FTP, /ftp_fgc_xqtl/projects/ADGWAS_Kunkle_2019_hg38_liftover/kunkle_sumstat_hg38_qc.chr* CU original data (in GRCh37) stage 1: /mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/&#39;Kunkle_etal_Stage1_results.txt?file=1&#39; stage 2: /mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/&#39;Kunkle_etal_Stage2_results.txt?file=1&#39; liftover data (in GRCh38): stage 1: /mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/Kunkle_etal_Stage1_results.txt_file_1_hg38.txt stage 2: /mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/Kunkle_etal_Stage2_results.txt_file_1_hg38.txt  Path to SuSiE RSS Fine-mapping Objects #   Li-San Wang FTP: /ftp_fgc_xqtl/projects/GWAS_Finemapping_Results/Kunkle/  Download source #  This data is derived from summary statistics from the Kunkle et al." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://cumc.github.io/xqtl-data/a/alzheimers-disease-gwas-summary-data-kunkle/" /><meta property="article:section" content="a" />



<title>Alzheimer&#39;s Disease GWAS Summary Data (Kunkle) | FunGen-xQTL Data Repository</title>
<link rel="manifest" href="/xqtl-data/manifest.json">
<link rel="icon" href="/xqtl-data/favicon.png" >
<link rel="canonical" href="https://cumc.github.io/xqtl-data/a/alzheimers-disease-gwas-summary-data-kunkle/">
<link rel="stylesheet" href="/xqtl-data/book.min.33a48f5432973b8ff9a82679d9e45d67f2c15d4399bd2829269455cfe390b5e8.css" integrity="sha256-M6SPVDKXO4/5qCZ52eRdZ/LBXUOZvSgpJpRVz&#43;OQteg=" crossorigin="anonymous">
  <script defer src="/xqtl-data/flexsearch.min.js"></script>
  <script defer src="/xqtl-data/en.search.min.7d8c7087392e9b20178e66069a2711358324dd1c2231aa1919452888d5fa6faa.js" integrity="sha256-fYxwhzkumyAXjmYGmicRNYMk3RwiMaoZGUUoiNX6b6o=" crossorigin="anonymous"></script>

  <script defer src="/xqtl-data/sw.min.5a8d9ccf5ea73dcbbb1f34b183236a28dd36cc6d6540bb5b46431b4e00517252.js" integrity="sha256-Wo2cz16nPcu7HzSxgyNqKN02zG1lQLtbRkMbTgBRclI=" crossorigin="anonymous"></script>
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->
  
</head>
<body dir="ltr">
  <input type="checkbox" class="hidden toggle" id="menu-control" />
  <input type="checkbox" class="hidden toggle" id="toc-control" />
  <main class="container flex">
    <aside class="book-menu">
      <div class="book-menu-content">
        
  <nav>
<h2 class="book-brand">
  <a class="flex align-center" href="/xqtl-data/"><span>FunGen-xQTL Data Repository</span>
  </a>
</h2>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>












  <ul>
<li>Alphabetic List of Data Available
<ul>
<li>
  <a href="/xqtl-data/a/">A</a></li>
<li>
  <a href="/xqtl-data/d/">D</a></li>
<li>
  <a href="/xqtl-data/e/">E</a></li>
<li>
  <a href="/xqtl-data/f/">F</a></li>
<li>
  <a href="/xqtl-data/h/">H</a></li>
<li>
  <a href="/xqtl-data/k/">K</a></li>
<li>
  <a href="/xqtl-data/m/">M</a></li>
<li>
  <a href="/xqtl-data/n/">N</a></li>
<li>
  <a href="/xqtl-data/p/">P</a></li>
<li>
  <a href="/xqtl-data/r/">R</a></li>
<li>
  <a href="/xqtl-data/s/">S</a></li>
<li>
  <a href="/xqtl-data/t/">T</a></li>
<li>
  <a href="/xqtl-data/w/">W</a></li>
</ul>
</li>
</ul>










</nav>




  <script>(function(){var a=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(b){localStorage.setItem("menu.scrollTop",a.scrollTop)}),a.scrollTop=localStorage.getItem("menu.scrollTop")})()</script>


 
      </div>
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/xqtl-data/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>Alzheimer&#39;s Disease GWAS Summary Data (Kunkle)</strong>

  <label for="toc-control">
    
    <img src="/xqtl-data/svg/toc.svg" class="book-icon" alt="Table of Contents" />
    
  </label>
</div>


  
  <aside class="hidden clearfix">
    
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#paths-to-summary-statistics">Path(s) to summary statistics</a></li>
    <li><a href="#path-to-susie-rss-fine-mapping-objects">Path to SuSiE RSS Fine-mapping Objects</a></li>
    <li><a href="#download-source">Download source</a></li>
    <li><a href="#file-schema">File Schema</a></li>
    <li><a href="#links-to-gwas-data-analysis-notebooks">Links to GWAS data analysis notebooks</a></li>
    <li><a href="#cohorts-included-in-this-study">Cohorts included in this study</a></li>
  </ul>
</nav>



  </aside>
  
 
      </header>

      
      
  <article class="markdown book-article"><h1 id="alzheimers-disease-gwas-summary-data-kunkle">
  Alzheimer&rsquo;s Disease GWAS Summary Data (Kunkle)
  <a class="anchor" href="#alzheimers-disease-gwas-summary-data-kunkle">#</a>
</h1>
<p>The SNP-level association testing summary statistics for Alzheimer&rsquo;s disease from Kunkle et al 2019 Nature Genetics. Position values were converted from hg19 to hg38 using liftOver.</p>
<h2 id="contact">
  Contact
  <a class="anchor" href="#contact">#</a>
</h2>
<p>Oluwatosin Olayinka</p>
<h2 id="paths-to-summary-statistics">
  Path(s) to summary statistics
  <a class="anchor" href="#paths-to-summary-statistics">#</a>
</h2>
<ul>
<li>NIAGADS FTP, <code>/ftp_fgc_xqtl/projects/ADGWAS_Kunkle_2019_hg38_liftover/kunkle_sumstat_hg38_qc.chr*</code></li>
<li>CU</li>
<li>original data (in GRCh37)</li>
<li>stage 1: <code>/mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/'Kunkle_etal_Stage1_results.txt?file=1'</code></li>
<li>stage 2: <code>/mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/'Kunkle_etal_Stage2_results.txt?file=1'</code></li>
<li>liftover data (in GRCh38):</li>
<li>stage 1: <code>/mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/Kunkle_etal_Stage1_results.txt_file_1_hg38.txt</code></li>
<li>stage 2: <code>/mnt/vast/hpc/csg/data_public/GWAS_sumstats/20231011_Kunkle/Kunkle_etal_Stage2_results.txt_file_1_hg38.txt</code></li>
</ul>
<h2 id="path-to-susie-rss-fine-mapping-objects">
  Path to SuSiE RSS Fine-mapping Objects
  <a class="anchor" href="#path-to-susie-rss-fine-mapping-objects">#</a>
</h2>
<ul>
<li>Li-San Wang FTP: <code>/ftp_fgc_xqtl/projects/GWAS_Finemapping_Results/Kunkle/</code></li>
</ul>
<h2 id="download-source">
  Download source
  <a class="anchor" href="#download-source">#</a>
</h2>
<p>This data is derived from summary statistics from the 
  <a href="http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007511/Kunkle_etal_Stage1_results.txt">Kunkle et al.</a> Nature Genetics paper.</p>
<h2 id="file-schema">
  File Schema
  <a class="anchor" href="#file-schema">#</a>
</h2>
<ul>
<li><code>chromosome</code>: chromosome ID</li>
<li><code>position</code>: hg38 position (converted from hg19 with liftOver)</li>
<li><code>variant</code>: variant ID in the form <code>chr${chromosome}_${position}_${ref}_${alt}</code></li>
<li><code>alt</code>: alternative effect allele</li>
<li><code>ref</code>: reference allele</li>
<li><code>beta</code>: SNP effect size</li>
<li><code>se</code>: standard error of <code>beta</code></li>
<li><code>pvalue</code>: p-value of <code>beta</code></li>
</ul>
<h2 id="links-to-gwas-data-analysis-notebooks">
  Links to GWAS data analysis notebooks
  <a class="anchor" href="#links-to-gwas-data-analysis-notebooks">#</a>
</h2>
<ol>
<li>Summary statistics preprocessing: 
  <a href="https://github.com/floutt/brain-xqtl-analysis/blob/main/analysis/Zhang_BU/susie_rss/summary_stat_qc_all.nb.html">https://github.com/floutt/brain-xqtl-analysis/blob/main/analysis/Zhang_BU/susie_rss/summary_stat_qc_all.nb.html</a></li>
</ol>
<h2 id="cohorts-included-in-this-study">
  Cohorts included in this study
  <a class="anchor" href="#cohorts-included-in-this-study">#</a>
</h2>
<ul>
<li>46 case-control studies: 21,982 cases + 41,944 controls</li>
<li>final sample:
94,437 individuals = 35,274 clinical and autopsy-documented AD cases + 59,163 controls</li>
<li>4 consortia (46 case-control studies in Suppl. Table 2) non-Hispanic Whites (NHW):</li>
<li>ADGC: Alzheimer Disease Genetics Consortium</li>
<li>CHARGE: Cohorts for Heart and Aging Research in Genomic Epidemiology Consortium</li>
<li>EADI: The European Alzheimer&rsquo;s Disease Initiative</li>
<li>GERAD/PERADES: Genetic and Environmental Risk in AD/Defining Genetic, Polygenic and Environmental Risk for Alzheimer&rsquo;s Disease Consortium&quot;
<em>Supplementary Table 1. Description of the consortium data sets used for Stage 1 discovery, Stage 2 and Stage 3.</em>
<strong>Discovery</strong>
<table>
<thead>
<tr>
<th></th>
<th></th>
<th>Alzheimer’s disease cases</th>
<th></th>
<th>Controls</th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Consortium</td>
<td>N</td>
<td>Percent female</td>
<td>Mean AAO (s.d)</td>
<td>N</td>
<td>Percent female</td>
<td>Mean AAE (s.d)</td>
</tr>
<tr>
<td>ADGC</td>
<td>14,428</td>
<td>59.3</td>
<td>71.1 (17.3)</td>
<td>14,562</td>
<td>59.3</td>
<td>76.2 (9.9)</td>
</tr>
<tr>
<td>CHARGE</td>
<td>2,137</td>
<td>67.3</td>
<td>82.6 (12)</td>
<td>13,474</td>
<td>55.8</td>
<td>76.7 (8.2)</td>
</tr>
<tr>
<td>EADI</td>
<td>2,240</td>
<td>65</td>
<td>75.4 (9.1)</td>
<td>6,631</td>
<td>60.6</td>
<td>78.9 (7.0)</td>
</tr>
<tr>
<td>GERAD</td>
<td>3,177</td>
<td>64</td>
<td>73.0 (0.2)</td>
<td>7,277</td>
<td>51.8</td>
<td>51.0 (0.1)</td>
</tr>
<tr>
<td>N</td>
<td>21,982</td>
<td></td>
<td></td>
<td>41,944</td>
<td></td>
<td></td>
</tr>
<tr>
<td><strong>Stage 2*</strong></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Country</td>
<td>N</td>
<td>Percent female</td>
<td>Mean AAO (s.d)</td>
<td>N</td>
<td>Percent female</td>
<td>Mean AAE (s.d.)</td>
</tr>
<tr>
<td>&mdash;&mdash;&mdash;&mdash;&ndash;</td>
<td>&mdash;&mdash;-</td>
<td>&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;-</td>
<td>&mdash;&mdash;&mdash;&mdash;&ndash;</td>
<td>&ndash;</td>
<td>&mdash;&mdash;</td>
<td>&mdash;&mdash;</td>
</tr>
<tr>
<td>Belgium</td>
<td>878</td>
<td>66.1</td>
<td>78.8 (8.2)</td>
<td>661</td>
<td>59.5</td>
<td>65.7 (14.3)</td>
</tr>
<tr>
<td>Finland</td>
<td>422</td>
<td>68</td>
<td>71.4 (6.9)</td>
<td>562</td>
<td>59.3</td>
<td>69.1 (6.2)</td>
</tr>
<tr>
<td>Germany</td>
<td>972</td>
<td>63.9</td>
<td>73.0 (8.6)</td>
<td>2,378</td>
<td>53.1</td>
<td>69.5 (10.1)</td>
</tr>
<tr>
<td>Greece</td>
<td>256</td>
<td>63.3</td>
<td>73.1 (7.9)</td>
<td>229</td>
<td>34.1</td>
<td>49.3 (16.4)</td>
</tr>
<tr>
<td>Hungary</td>
<td>125</td>
<td>68</td>
<td>78.9 (7.3)</td>
<td>100</td>
<td>69</td>
<td>74.4 (6.5)</td>
</tr>
<tr>
<td>Italy</td>
<td>1,729</td>
<td>66.5</td>
<td>71.5 (8.7)</td>
<td>720</td>
<td>55.7</td>
<td>70.0 (10.4)</td>
</tr>
<tr>
<td>Spain</td>
<td>2,121</td>
<td>66.3</td>
<td>75.0 (8.3)</td>
<td>1,921</td>
<td>55.3</td>
<td>70.2 (10.8)</td>
</tr>
<tr>
<td>Sweden</td>
<td>797</td>
<td>61.7</td>
<td>76.8 (8.1)</td>
<td>1,506</td>
<td>62.8</td>
<td>70.6 (8.7)</td>
</tr>
<tr>
<td>UK</td>
<td>490</td>
<td>67.6</td>
<td>74.6 (8.7)</td>
<td>1,066</td>
<td>29.2</td>
<td>73.8 (6.5)</td>
</tr>
<tr>
<td>USA</td>
<td>572</td>
<td>61.9</td>
<td>83.5 (7.6)</td>
<td>1,340</td>
<td>54</td>
<td>79.3 (6.8)</td>
</tr>
<tr>
<td>N</td>
<td>8,362</td>
<td></td>
<td></td>
<td>10,483</td>
<td></td>
<td></td>
</tr>
<tr>
<td><strong>Stage 3*</strong></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Country</td>
<td>N</td>
<td>Percent female</td>
<td>Mean AAO (s.d)</td>
<td>N</td>
<td>Percent female</td>
<td>Mean AAE (s.d.)</td>
</tr>
<tr>
<td>&mdash;&mdash;</td>
<td>&mdash;&mdash;</td>
<td>&mdash;&mdash;</td>
<td>&mdash;&mdash;</td>
<td>&mdash;&mdash;</td>
<td>&mdash;&mdash;</td>
<td>&mdash;&mdash;</td>
</tr>
<tr>
<td>Spain (ACE)</td>
<td>932</td>
<td>71</td>
<td>77.7 (7.9)</td>
<td>1,813</td>
<td>68.4</td>
<td>54.7 (12.1)</td>
</tr>
<tr>
<td>UK (Cardiff)</td>
<td>1,902</td>
<td>64.8</td>
<td>77.1 (7.9)</td>
<td>1,047</td>
<td>57.8</td>
<td>73.9 (12.9)</td>
</tr>
<tr>
<td>USA (ADC7)</td>
<td>514</td>
<td>51.3</td>
<td>73.9 (8.3)</td>
<td>790</td>
<td>63.6</td>
<td>72.3 (7.7)</td>
</tr>
<tr>
<td>Spain (GR@ACE)</td>
<td>1,582</td>
<td>74.6</td>
<td>78.5 (8.0)</td>
<td>3,086</td>
<td>48</td>
<td>54.0 (14.0)</td>
</tr>
<tr>
<td>N</td>
<td>4,930</td>
<td></td>
<td></td>
<td>6,736</td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
</li>
<li>AAO, age at onset; AAE, age at examination.</li>
<li>*Stage 2 and Stage 3 datasets were combined for genotyping of 33 replication variants not present on the 2013 custom chip
<em>Supplementary Table 2. Demographic description of datasets within each consortium.</em>
<table>
<thead>
<tr>
<th></th>
<th>Consortium</th>
<th>AD cases</th>
<th></th>
<th></th>
<th>Controls</th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
<td>N</td>
<td>% female</td>
<td>Mean AA (SD)</td>
<td>N</td>
<td>% female</td>
<td>Mean AAE (SD)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ACT</td>
<td>532</td>
<td>62.6</td>
<td>78.8 (12.7)</td>
<td>1,571</td>
<td>55.6</td>
<td>81.7 (5.9)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ADC1</td>
<td>1,549</td>
<td>54.3</td>
<td>71.6 (11.0)</td>
<td>512</td>
<td>59.2</td>
<td>76.8 (8.8)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ADC2</td>
<td>727</td>
<td>50.8</td>
<td>61.4 (29.5)</td>
<td>156</td>
<td>67.9</td>
<td>75.8 (7.9)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ADC3</td>
<td>894</td>
<td>54.7</td>
<td>58.6 (32.8)</td>
<td>586</td>
<td>63</td>
<td>72.8 (17.5)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ADC4</td>
<td>304</td>
<td>55.3</td>
<td>73.4 (7.0)</td>
<td>377</td>
<td>63.9</td>
<td>75.7 (8.1)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ADC5</td>
<td>286</td>
<td>53.1</td>
<td>73.7 (7.0)</td>
<td>505</td>
<td>65.5</td>
<td>77.6 (9.0)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ADC6</td>
<td>213</td>
<td>58.2</td>
<td>73.9 (7.6)</td>
<td>338</td>
<td>66.6</td>
<td>74.6 (9.0)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ADNI</td>
<td>268</td>
<td>42.2</td>
<td>75.3 (7.1)</td>
<td>173</td>
<td>40.5</td>
<td>78.6 (5.5)</td>
</tr>
<tr>
<td>ADGC</td>
<td>BIOCARD</td>
<td>6</td>
<td>33.3</td>
<td>73.8 (6.1)</td>
<td>112</td>
<td>62.5</td>
<td>68.0 (5.5)</td>
</tr>
<tr>
<td>ADGC</td>
<td>CHAP</td>
<td>27</td>
<td>63</td>
<td>84.8 (7.6)</td>
<td>144</td>
<td>52.8</td>
<td>81.8 (6.6)</td>
</tr>
<tr>
<td>ADGC</td>
<td>EAS</td>
<td>9</td>
<td>44.4</td>
<td>85.2 (4.9)</td>
<td>141</td>
<td>41.1</td>
<td>84.4 (5.2)</td>
</tr>
<tr>
<td>ADGC</td>
<td>GenADA</td>
<td>666</td>
<td>56.9</td>
<td>72.8 (13.5)</td>
<td>712</td>
<td>63.9</td>
<td>74.2 (7.0)</td>
</tr>
<tr>
<td>ADGC</td>
<td>MAYO</td>
<td>658</td>
<td>57.4</td>
<td>73.6 (4.8)</td>
<td>1,046</td>
<td>51.1</td>
<td>72.9 (4.4)</td>
</tr>
<tr>
<td>ADGC</td>
<td>MIRAGE</td>
<td>491</td>
<td>63.3</td>
<td>69.9 (11.5)</td>
<td>738</td>
<td>58.8</td>
<td>70.8 (12.1)</td>
</tr>
<tr>
<td>ADGC</td>
<td>MTC</td>
<td>256</td>
<td>57</td>
<td>73.6 (11.8)</td>
<td>189</td>
<td>61.4</td>
<td>70.9 (9.7)</td>
</tr>
<tr>
<td>ADGC</td>
<td>NIALOAD</td>
<td>1,798</td>
<td>65</td>
<td>73.1 (9.3)</td>
<td>1,568</td>
<td>60.2</td>
<td>73.8 (9.3)</td>
</tr>
<tr>
<td>ADGC</td>
<td>NBB</td>
<td>80</td>
<td>71.3</td>
<td>74.5 (7.5)</td>
<td>48</td>
<td>56.3</td>
<td>81.5 (9.4)</td>
</tr>
<tr>
<td>ADGC</td>
<td>OHSU</td>
<td>132</td>
<td>62.1</td>
<td>85.9 (5.7)</td>
<td>153</td>
<td>54.9</td>
<td>83.9 (7.6)</td>
</tr>
<tr>
<td>ADGC</td>
<td>PFIZER</td>
<td>696</td>
<td>53.7</td>
<td>73.7 (5.0)</td>
<td>762</td>
<td>54.1</td>
<td>77.2 (4.9)</td>
</tr>
<tr>
<td>ADGC</td>
<td>RMAYO</td>
<td>13</td>
<td>23.1</td>
<td>78.5 (9.0)</td>
<td>233</td>
<td>41.6</td>
<td>79.2 (5.8)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ROSMAP</td>
<td>295</td>
<td>70.5</td>
<td>85.6 (6.2)</td>
<td>769</td>
<td>72</td>
<td>82.2 (7.1)</td>
</tr>
<tr>
<td>ADGC</td>
<td>ROSMAP2</td>
<td>59</td>
<td>78</td>
<td>81.9 (6.9)</td>
<td>217</td>
<td>76</td>
<td>80.8 (7.2)</td>
</tr>
<tr>
<td>ADGC</td>
<td>TARC1</td>
<td>323</td>
<td>61.6</td>
<td>74.0 (7.1)</td>
<td>181</td>
<td>65.2</td>
<td>73.9 (8.2)</td>
</tr>
<tr>
<td>ADGC</td>
<td>TGEN2</td>
<td>668</td>
<td>64.8</td>
<td>67.2 (22.9)</td>
<td>365</td>
<td>48.5</td>
<td>80.0 (8.7)</td>
</tr>
<tr>
<td>ADGC</td>
<td>UKS</td>
<td>596</td>
<td>57.4</td>
<td>72.2 (6.6)</td>
<td>170</td>
<td>51.2</td>
<td>64.1 (3.0)</td>
</tr>
<tr>
<td>ADGC</td>
<td>UMCWRMSSM</td>
<td>1,177</td>
<td>64.5</td>
<td>71.1 (17.4)</td>
<td>1,126</td>
<td>61.3</td>
<td>73.5 (10.6)</td>
</tr>
<tr>
<td>ADGC</td>
<td>UPITT</td>
<td>1,255</td>
<td>62.9</td>
<td>66.8 (22.4)</td>
<td>829</td>
<td>63.3</td>
<td>75.5 (6.0)</td>
</tr>
<tr>
<td>ADGC</td>
<td>WASHU</td>
<td>339</td>
<td>57.2</td>
<td>69.1 (21.5)</td>
<td>187</td>
<td>60.4</td>
<td>76.9 (8.4)</td>
</tr>
<tr>
<td>ADGC</td>
<td>WASHU2</td>
<td>38</td>
<td>57.9</td>
<td>73.4 (7.3)</td>
<td>94</td>
<td>46.8</td>
<td>51.7 (35.2)</td>
</tr>
<tr>
<td>ADGC</td>
<td>WHICAP</td>
<td>73</td>
<td>72.6</td>
<td>83.9 (7.8)</td>
<td>560</td>
<td>60.4</td>
<td>81.7 (6.7)</td>
</tr>
<tr>
<td>CHARGE</td>
<td>AGES</td>
<td>95</td>
<td>51.6</td>
<td>81.5 (0.1)</td>
<td>2,708</td>
<td>59.2</td>
<td>75.7 (0.1)</td>
</tr>
<tr>
<td>CHARGE</td>
<td>ASPS</td>
<td>277</td>
<td>57.8</td>
<td>76.4 (8.3)</td>
<td>169</td>
<td>58</td>
<td>66.4 (10.8)</td>
</tr>
<tr>
<td>CHARGE</td>
<td>CHS</td>
<td>450</td>
<td>66</td>
<td>81.9 (5.2)</td>
<td>1,702</td>
<td>60.3</td>
<td>81.1 (5.2)</td>
</tr>
<tr>
<td>CHARGE</td>
<td>FHS</td>
<td>330</td>
<td>64</td>
<td>86.1 (7.2)</td>
<td>3,910</td>
<td>49</td>
<td>74.0 (9.5)</td>
</tr>
<tr>
<td>CHARGE</td>
<td>ROTTERDAM</td>
<td>985</td>
<td>73.2</td>
<td>83.5 (6.6)</td>
<td>4,985</td>
<td>57.6</td>
<td>78.0 (7.6)</td>
</tr>
<tr>
<td>EADI</td>
<td></td>
<td>2,240</td>
<td>65</td>
<td>75.4 (9.1)</td>
<td>6,631</td>
<td>60.6</td>
<td>78.9 (7.0)</td>
</tr>
<tr>
<td>GERAD</td>
<td>  MRC</td>
<td>1,008</td>
<td>70.3</td>
<td>80.9 (6.5)</td>
<td>873</td>
<td>61.6</td>
<td>75.9 (6.3)</td>
</tr>
<tr>
<td>GERAD</td>
<td>  ARUK</td>
<td>939</td>
<td>61</td>
<td>76.6 (9.6)</td>
<td>82</td>
<td>59.8</td>
<td>77.9 (7.6)</td>
</tr>
<tr>
<td>GERAD</td>
<td>  BONN</td>
<td>551</td>
<td>63.7</td>
<td>72.9 (8.3)</td>
<td>37</td>
<td>64.9</td>
<td>79.5 (3.6)</td>
</tr>
<tr>
<td>GERAD</td>
<td>  WASHU</td>
<td>423</td>
<td>56</td>
<td>82.1 (9.0)</td>
<td>156</td>
<td>65.4</td>
<td>78.5 (9.7)</td>
</tr>
<tr>
<td>GERAD</td>
<td>  NIMH</td>
<td>127</td>
<td>63</td>
<td>80.1 (6.1)</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>  GERAD</td>
<td> UCL:PRION</td>
<td>82</td>
<td>59.8</td>
<td>63.6 (9.9)</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>  GERAD</td>
<td> UCL:LASER</td>
<td>47</td>
<td>74.5</td>
<td>80.6(7.9)</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>  GERAD</td>
<td> 1958BC</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>5,342</td>
<td>49.8</td>
<td>45.0 (0.0)</td>
</tr>
<tr>
<td>GERAD</td>
<td>  KORA</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>434</td>
<td>49.1</td>
<td>56.0 (7.2)</td>
</tr>
<tr>
<td>GERAD</td>
<td>  HNR</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>353</td>
<td>52.9</td>
<td>54.6 (5.3)</td>
</tr>
<tr>
<td>GERAD</td>
<td>  MAYO<!-- raw HTML omitted -->1<!-- raw HTML omitted --></td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>TOTALS</td>
<td></td>
<td>21,982</td>
<td></td>
<td></td>
<td>41,944</td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
</li>
</ul>
</article>
 
      

      <footer class="book-footer">
        
  <div class="flex flex-wrap justify-between">





</div>



  <script>(function(){function a(c){const a=window.getSelection(),b=document.createRange();b.selectNodeContents(c),a.removeAllRanges(),a.addRange(b)}document.querySelectorAll("pre code").forEach(b=>{b.addEventListener("click",function(c){if(window.getSelection().toString())return;a(b.parentElement),navigator.clipboard&&navigator.clipboard.writeText(b.parentElement.textContent)})})})()</script>


 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      <div class="book-toc-content">
        
  
<nav id="TableOfContents">
  <ul>
    <li><a href="#contact">Contact</a></li>
    <li><a href="#paths-to-summary-statistics">Path(s) to summary statistics</a></li>
    <li><a href="#path-to-susie-rss-fine-mapping-objects">Path to SuSiE RSS Fine-mapping Objects</a></li>
    <li><a href="#download-source">Download source</a></li>
    <li><a href="#file-schema">File Schema</a></li>
    <li><a href="#links-to-gwas-data-analysis-notebooks">Links to GWAS data analysis notebooks</a></li>
    <li><a href="#cohorts-included-in-this-study">Cohorts included in this study</a></li>
  </ul>
</nav>


 
      </div>
    </aside>
    
  </main>

  
</body>
</html>












